---
title: Papers 
papers: True
---

### A novel Drosophila injury model reveals severed axons are cleared through a Draper/MMP-1 signaling cascade 

Authors: Maria D Purice, Arpita Ray, Eva Jolanda Münzel, Bernard J Pope, Daniel J Park, Sean D Speese, Mary A Logan 

Journal: [eLIFE, 2017](https://elifesciences.org/articles/23611)

#### Abstract 

Neural injury triggers swift responses from glia, including glial migration and phagocytic clearance of damaged neurons. The transcriptional programs governing these complex innate glial immune responses are still unclear. Here, we describe a novel injury assay in adult Drosophila that elicits widespread glial responses in the ventral nerve cord (VNC). We profiled injury-induced changes in VNC gene expression by RNA sequencing (RNA-seq) and found that responsive genes fall into diverse signaling classes. One factor, matrix metalloproteinase-1 (MMP-1), is induced in Drosophila ensheathing glia responding to severed axons. Interestingly, glial induction of MMP-1 requires the highly conserved engulfment receptor Draper, as well as AP-1 and STAT92E. In MMP-1 depleted flies, glia do not properly infiltrate neuropil regions after axotomy and, as a consequence, fail to clear degenerating axonal debris. This work identifies Draper-dependent activation of MMP-1 as a novel cascade required for proper glial clearance of severed axons.

****

### Somatic mutations of the coding microsatellites within the beta-2-microglobulin gene in mismatch repair-deficient colorectal cancers and adenomas  

Authors: Mark Clendenning, Alvin Huang, Harindra Jayasekara, Marie Lorans, Susan Preston, Neil O’Callaghan, Bernard J. Pope, Finlay A. Macrae, Ingrid M. Winship, Roger L. Milne, Graham G. Giles, Dallas R. English, John L. Hopper, Aung K. Win, Mark A. Jenkins, Melissa C. Southey, Christophe Rosty, Daniel D. Buchanan  

Journal: [Familial Cancer, 2017](https://link.springer.com/article/10.1007/s10689-017-0013-y)

Topics: beta-2-microglobulin (B2M), colorectal cancer, mismatch repair deficiency, microsatellite instability, lynch syndrome,  MLH1 methylation   

#### Abstract 

In colorectal cancers (CRCs) with tumour mismatch repair (MMR) deficiency, genes involved in the host immune response that contain microsatellites in their coding regions, including beta-2-microglobulin (B2M), can acquire mutations that may alter the immune response, tumour progression and prognosis. We screened the coding microsatellites within B2M for somatic mutations in MMR-deficient CRCs and adenomas to determine associations with tumour subtypes, clinicopathological features and survival. Incident MMR-deficient CRCs from Australasian Colorectal Cancer Family Registry (ACCFR) and the Melbourne Collaborative Cohort Study participants (n = 144) and 63 adenomas from 41 MMR gene mutation carriers from the ACCFR were screened for somatic mutations within five coding microsatellites of B2M. Hazard ratios (HR) and 95% confidence intervals (CI) for overall survival by B2M mutation status were estimated using Cox regression, adjusting for age at CRC diagnosis, sex, AJCC stage and grade. B2M mutations occurred in 30 (20.8%) of the 144 MMR-deficient CRCs (29% of the MLH1-methylated, 17% of the Lynch syndrome and 9% of the suspected Lynch CRCs). No B2M mutations were identified in the 63 adenomas tested. B2M mutations differed by site, stage, grade and lymphocytic infiltration although none reached statistical significance (p > 0.05). The HR for overall survival for B2M mutated CRC was 0.65 (95% CI 0.29–1.48) compared with B2M wild-type. We observed differences in B2M mutation status in MMR-deficient CRC by tumour subtypes, site, stage, grade, immune infiltrate and for overall survival that warrant further investigation in larger studies before B2M mutation status can be considered to have clinical utility.

****

### Four simple recommendations to encourage best practices in research software 

Authors: Rafael C. Jiménez, Mateusz Kuzak, Monther Alhamdoosh, Michelle Barker, Bérénice Batut, Mikael Borg, Salvador Capella-Gutierrez, Neil Chue Hong, Martin Cook, Manuel Corpas, Madison Flannery, Leyla Garcia, Josep Ll. Gelpí, Simon Gladman, Carole Goble, Montserrat González Ferreiro, Alejandra Gonzalez-Beltran, Philippa C. Griffin, Björn Grüning, Jonas Hagberg, Petr Holub, Rob Hooft, Jon Ison, Daniel S. Katz, Brane Leskošek, Federico López Gómez, Luis J. Oliveira, David Mellor, Rowland Mosbergen, Nicola Mulder, Yasset Perez-Riverol, Robert Pergl, Horst Pichler, Bernard Pope, Ferran Sanz, Maria V. Schneider, Victoria Stodden, Radosław Suchecki, Radka Svobodová Vařeková, Harry-Anton Talvik, Ilian Todorov, Andrew Treloar, Sonika Tyagi, Maarten van Gompel, Daniel Vaughan, Allegra Via, Xiaochuan Wang, Nathan S. Watson-Haigh, Steve Crouch

Journal: [F1000 Research, 2017](https://f1000research.com/articles/6-876/v1)

Topics: Research software, best practices

#### Abstract 

Scientific research relies on computer software, yet software is not always developed following practices that ensure its quality and sustainability. This manuscript does not aim to propose new software development best practices, but rather to provide simple recommendations that encourage the adoption of existing best practices. Software development best practices promote better quality software, and better quality software improves the reproducibility and reusability of research. These recommendations are designed around Open Source values, and provide practical suggestions that contribute to making research software and its source code more discoverable, reusable and transparent. This manuscript is aimed at developers, but also at organisations, projects, journals and funders that can increase the quality and sustainability of research software by encouraging the adoption of these recommendations.

****

### Variant effect prediction tools assessed using independent, functional assay-based datasets: implications for discovery and diagnostics  

Authors: Khalid Mahmood, Chol-hee Jung, Gayle Philip, Peter Georgeson, Jessica Chung, Bernard J. Pope and Daniel J. Park  

Journal: [Human Genomics, 2017](https://humgenomics.biomedcentral.com/articles/10.1186/s40246-017-0104-8)

Topics: Variant effect prediction, functional datasets, benchmarking, mutation assessment, pathogenicity prediction, protein function, functional assays, genomic screening  

#### Abstract 

Background

Genetic variant effect prediction algorithms are used extensively in clinical genomics and research to determine the likely consequences of amino acid substitutions on protein function. It is vital that we better understand their accuracies and limitations because published performance metrics are confounded by serious problems of circularity and error propagation. Here, we derive three independent, functionally determined human mutation datasets, UniFun, BRCA1-DMS and TP53-TA, and employ them, alongside previously described datasets, to assess the pre-eminent variant effect prediction tools.

Results

Apparent accuracies of variant effect prediction tools were influenced significantly by the benchmarking dataset. Benchmarking with the assay-determined datasets UniFun and BRCA1-DMS yielded areas under the receiver operating characteristic curves in the modest ranges of 0.52 to 0.63 and 0.54 to 0.75, respectively, considerably lower than observed for other, potentially more conflicted datasets.

Conclusions

These results raise concerns about how such algorithms should be employed, particularly in a clinical setting. Contemporary variant effect prediction tools are unlikely to be as accurate at the general prediction of functional impacts on proteins as reported prior. Use of functional assay-based datasets that avoid prior dependencies promises to be valuable for the ongoing development and accurate benchmarking of such tools.

****

### Mutation screening of ACKR3 and COPS8 in kidney cancer cases from the CONFIRM study 

Authors: Maryam Mahmoodi, Tu Nguyen-Dumont, Fleur Hammet, Bernard J. Pope, Daniel J. Park, Melissa C. Southey, John M. Darlow, Fiona Bruinsma, Ingrid Winship

Journal: [Familial Cancer, 2017](http://link.springer.com/article/10.1007/s10689-016-9961-x)

Topics: Kidney cancer, ACKR3, COPS8, Mutation screening, Massively parallel sequencing, Hi-Plex  

#### Abstract 

An apparently balanced t(2;3)(q37.3;q13.2) translocation that appears to segregate with renal cell carcinoma (RCC) has indicated potential areas to search for the elusive genetic basis of clear cell RCC. We applied Hi-Plex targeted sequencing to analyse germline DNA from 479 individuals affected with clear cell RCC for this breakpoint translocation and genetic variants in neighbouring genes on chromosome 2, ACKR3 and COPS8. While only synonymous variants were found in COPS8, one of the missense variants in ACKR3:c.892C>T, observed in 4/479 individuals screened (0.8%), was predicted likely to damage ACKR3 function. Identification of causal genes for RCC has potential clinical utility, where risk assessment and risk management can offer better outcomes, with surveillance for at-risk relatives and nephron sparing surgery through earlier intervention.

****

### Single nucleotide-level mapping of DNA double-strand breaks in human HEK293T cells 

Authors: Bernard J. Pope, Khalid Mahmood, Chol-hee Jung, Peter Georgeson, Daniel J. Park 

Journal: [Genomics Data, 2016](http://www.sciencedirect.com/science/article/pii/S2213596016301428)

Topics: Double-strand breaks; Fragile sites; Human genome; Forum domains; HEK293T  

#### Abstract 

Constitutional biological processes involve the generation of DNA double-strand breaks (DSBs). The production of such breaks and their subsequent resolution are also highly relevant to neurodegenerative diseases and cancer, in which extensive DNA fragmentation has been described Stephens et al. (2011), Blondet et al. (2001) . Tchurikov et al. Tchurikov et al. (2011, 2013) have reported previously that frequent sites of DSBs occur in chromosomal domains involved in the co-ordinated expression of genes. This group report that hot spots of DSBs in human HEK293T cells often coincide with H3K4me3 marks, associated with active transcription Kravatsky et al. (2015) and that frequent sites of DNA double-strand breakage are likely to be relevant to cancer genomics Tchurikov et al. (2013, 2016) . Recently, they applied a RAFT (rapid amplification of forum termini) protocol that selects for blunt-ended DSB sites and mapped these to the human genome within defined co-ordinate ‘windows’. In this paper, we re-analyse public RAFT data to derive sites of DSBs at the single-nucleotide level across the built genome for human HEK293T cells (https://figshare.com/s/35220b2b79eaaaf64ed8). This refined mapping, combined with accessory ENCODE data tracks and ribosomal DNA-related sequence annotations, will likely be of value for the design of clinically relevant targeted assays such as those for cancer susceptibility, diagnosis, treatment-matching and prognostication.

****

### Fine resolution mapping of double-strand break sites for human ribosomal DNA units 

Authors: Bernard J. Pope, Khalid Mahmood, Chol-hee Jung, Daniel J. Park 

Journal: [Genomics Data, 2016](http://www.sciencedirect.com/science/article/pii/S221359601630109X)

Topics: Double-strand breaks; Fragile sites; rDNA; Forum domains; HEK293T  

#### Abstract 

DNA breakage arises during a variety of biological processes, including transcription, replication and genome rearrangements. In the context of disease, extensive fragmentation of DNA has been described in cancer cells and during early stages of neurodegeneration. Stults et al. reported that human rDNA gene clusters are hotspots for recombination and that rDNA restructuring is among the most common chromosomal alterations in adult solid tumours. As such, analysis of rDNA regions is likely to have significant prognostic and predictive value, clinically. Tchurikov et al. have made major advances in this direction, reporting that sites of human genome double-strand breaks (DSBs) occur frequently at sites in rDNA that are tightly linked with active transcription - the authors used a RAFT (rapid amplification of forum termini) protocol that selects for blunt-ended sites. They reported the relative frequency of these rDNA DSBs within defined co-ordinate 'windows' of varying size and made these data (as well as the relevant 'raw' sequencing information) available to the public. Assay designs targeting rDNA DSB hotspots will benefit greatly from the publication of break sites at greater resolution. Here, we re-analyse public RAFT data and make available rDNA DSB co-ordinates to the single-nucleotide level.

****

### UNDR ROVER - a fast and accurate variant caller for targeted DNA sequencing

Authors: Daniel J. Park, Roger Li, Edmund Lau, Peter Georgeson, Tú Nguyen-Dumont and Bernard J. Pope 

Journal: [BMC Bioinformatics, 2016](http://www.biomedcentral.com/1471-2105/17/165)

Topics: PCR-MPS; Hi-Plex; ROVER; Targeted sequencing; Massively parallel sequencing; Variant calling  

#### Abstract 

Background

Previously, we described ROVER, a DNA variant caller which identifies genetic variants from PCR-targeted massively parallel sequencing (MPS) datasets generated by the Hi-Plex protocol. ROVER permits stringent filtering of sequencing chemistry-induced errors by requiring reported variants to appear in both reads of overlapping pairs above certain thresholds of occurrence. ROVER was developed in tandem with Hi-Plex and has been used successfully to screen for genetic mutations in the breast cancer predisposition gene PALB2.

ROVER is applied to MPS data in BAM format and, therefore, relies on sequence reads being mapped to a reference genome. In this paper, we describe an improvement to ROVER, called UNDR ROVER (Unmapped primer-Directed ROVER), which accepts MPS data in FASTQ format, avoiding the need for a computationally expensive mapping stage. It does so by taking advantage of the location-specific nature of PCR-targeted MPS data.

Results

The UNDR ROVER algorithm achieves the same stringent variant calling as its predecessor with a significant runtime performance improvement. In one indicative sequencing experiment, UNDR ROVER (in its fastest mode) required 8-fold less sequential computation time than the ROVER pipeline and 13-fold less sequential computation time than a variant calling pipeline based on the popular GATK tool.

UNDR ROVER is implemented in Python and runs on all popular POSIX-like operating systems (Linux, OS X). It requires as input a tab-delimited format file containing primer sequence information, a FASTA format file containing the reference genome sequence, and paired FASTQ files containing sequence reads. Primer sequences at the 5′ end of reads associate read-pairs with their targeted amplicon and, thus, their expected corresponding coordinates in the reference genome. The primer-intervening sequence of each read is compared against the reference sequence from the same location and variants are identified using the same algorithm as ROVER. Specifically, for a variant to be ‘called’ it must appear at the same location in both of the overlapping reads above user-defined thresholds of minimum number of reads and proportion of reads.

Conclusions

UNDR ROVER provides the same rapid and accurate genetic variant calling as its predecessor with greatly reduced computational costs.

****

### MethPat: a tool for the analysis and visualisation of complex methylation patterns obtained by massively parallel sequencing 

Authors: N Wong, B Pope, I Candiloro, D Korbie, M Trau, S Wong, T Mikeska, X Zhang, M Pitman, S Eggers, S Doyle and A Dobrovic

Journal: [BMC Bioinformatics, 2016](http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-0950-8)

Topics: DNA Methylation, Visualization

#### Abstract 

Background

DNA methylation at a gene promoter region has the potential to regulate gene transcription. Patterns of methylation over multiple CpG sites in a region are often complex and cell type specific, with the region showing multiple allelic patterns in a sample. This complexity is commonly obscured when DNA methylation data is summarised as an average percentage value for each CpG site (or aggregated across CpG sites). True representation of methylation patterns can only be fully characterised by clonal analysis. Deep sequencing provides the ability to investigate clonal DNA methylation patterns in unprecedented detail and scale, enabling the proper characterisation of the heterogeneity of methylation patterns. However, the sheer amount and complexity of sequencing data requires new synoptic approaches to visualise the distribution of allelic patterns.

Results

We have developed a new analysis and visualisation software tool “Methpat”, that extracts and displays clonal DNA methylation patterns from massively parallel sequencing data aligned using Bismark. Methpat was used to analyse multiplex bisulfite amplicon sequencing on a range of CpG island targets across a panel of human cell lines and primary tissues. Methpat was able to represent the clonal diversity of epialleles analysed at specific gene promoter regions. We also used Methpat to describe epiallelic DNA methylation within the mitochondrial genome.

Conclusions

Methpat can summarise and visualise epiallelic DNA methylation results from targeted amplicon, massively parallel sequencing of bisulfite converted DNA in a compact and interpretable format. Unlike currently available tools, Methpat can visualise the diversity of epiallelic DNA methylation patterns in a sample.

****

### Exemplary multiplex bisulfite amplicon data used to demonstrate the utility of Methpat  

Authors: Wong NC, Pope BJ, Candiloro I, Korbie D, Trau M, Wong SQ, Mikeska T, van Denderen BJ, Thompson EW, Eggers S, Doyle SR, Dobrovic A  

Journal: [Gigascience, 2015](http://www.gigasciencejournal.com/content/4/1/55)

Topics: DNA methylation, Bisulfite sequencing, PCR, Visualization, Epigenetics, Cancer, Epialleles 

#### Abstract 

Background

DNA methylation is a complex epigenetic marker that can be analyzed using a wide variety of methods. Interpretation and visualization of DNA methylation data can mask complexity in terms of methylation status at each CpG site, cellular heterogeneity of samples and allelic DNA methylation patterns within a given DNA strand. Bisulfite sequencing is considered the gold standard, but visualization of massively parallel sequencing results remains a significant challenge.

Findings

We created a program called Methpat that facilitates visualization and interpretation of bisulfite sequencing data generated by massively parallel sequencing. To demonstrate this, we performed multiplex PCR that targeted 48 regions of interest across 86 human samples. The regions selected included known gene promoters associated with cancer, repetitive elements, known imprinted regions and mitochondrial genomic sequences. We interrogated a range of samples including human cell lines, primary tumours and primary tissue samples. Methpat generates two forms of output: a tab-delimited text file for each sample that summarizes DNA methylation patterns and their read counts for each amplicon, and a HTML file that summarizes this data visually. Methpat can be used with publicly available whole genome bisulfite sequencing and reduced representation bisulfite sequencing datasets with sufficient read depths.

Conclusions

Using Methpat, complex DNA methylation data derived from massively parallel sequencing can be summarized and visualized for biological interpretation. By accounting for allelic DNA methylation states and their abundance in a sample, Methpat can unmask the complexity of DNA methylation and yield further biological insight in existing datasets.


****

### High-Resolution Twin-Ion Metabolite Extraction (HiTIME) Mass Spectrometry: Nontargeted Detection of Unknown Drug Metabolites by Isotope Labeling, Liquid Chromatography Mass Spectrometry, and Automated High-Performance Computing 

Authors: Michael G. Leeming, Andrew P. Isaac, Bernard J. Pope, Noel Cranswick, Christine E. Wright, James Ziogas, Richard A. J. O’Hair, and William A. Donald.

Journal: [Analytical Chemistry, 2015](http://pubs.acs.org/doi/abs/10.1021/ac504767d)

Topics: Mass Spectrometry, Bioinformatics

#### Abstract 

The metabolic fate of a compound can often determine the success of a new drug lead. Thus, significant effort is directed toward identifying the metabolites formed from a given molecule. Here, an automated and nontargeted procedure is introduced for detecting drug metabolites without authentic metabolite standards via the use of stable isotope labeling, liquid chromatography mass spectrometry (LC/MS), and high-performance computing. LC/MS of blood plasma extracts from rats that were administered a 1:1 mixture of acetaminophen (APAP) and 13C6-APAP resulted in mass spectra that contained “twin” ions for drug metabolites that were not detected in control spectra (i.e., no APAP administered). Because of the development of a program (high-resolution twin-ion metabolite extraction; HiTIME) that can identify twin-ions in high-resolution mass spectra without centroiding (i.e., reduction of mass spectral peaks to single data points), 9 doublets corresponding to APAP metabolites were identified. This is nearly twice that obtained by use of existing programs that make use of centroiding to reduce computational cost under these conditions with a quadrupole time-of-flight mass spectrometer. By a manual search for all reported APAP metabolite ions, no additional twin-ion signals were assigned. These data indicate that all the major metabolites of APAP and multiple low-abundance metabolites (e.g., acetaminophen hydroxy- and methoxysulfate) that are rarely reported were detected. This methodology can be used to detect drug metabolites without prior knowledge of their identity. HiTIME is freely available from [https://github.com/bjpop/HiTIME](https://github.com/bjpop/HiTIME).

****

### Mutation screening of PALB2 in clinically ascertained families from the Breast Cancer Family Registry

Authors: Tú Nguyen-Dumont, Fleur Hammet, Maryam Mahmoodi, Helen Tsimiklis, Zhi L. Teo, Roger Li, Bernard J. Pope, Mary Beth Terry, Saundra S. Buys, Mary Daly, John L. Hopper, Ingrid Winship, David E. Goldgar, Daniel J. Park, Melissa C. Southey 

Journal: [Breast Cancer Research and Treatment, 2015](http://link.springer.com/article/10.1007/s10549-014-3260-8)

Topics: Breast Cancer, Bioinformatics, Genomics

#### Abstract 

Loss-of-function mutations in PALB2 are associated with an increased risk of breast cancer, with recent data showing that female breast cancer risks for PALB2 mutation carriers are comparable in magnitude to those for BRCA2 mutation carriers. This study applied targeted massively parallel sequencing to characterize the mutation spectrum of PALB2 in probands attending breast cancer genetics clinics in the USA. The coding regions and proximal intron–exon junctions of PALB2 were screened in probands not known to carry a mutation in BRCA1 or BCRA2 from 1,250 families enrolled through familial cancer clinics by the Breast Cancer Family Registry. Mutation screening was performed using Hi-Plex, an amplicon-based targeted massively parallel sequencing platform. Screening of PALB2 was successful in 1,240/1,250 probands and identified nine women with protein-truncating mutations (three nonsense mutations and five frameshift mutations). Four of the 33 missense variants were predicted to be deleterious to protein function by in silico analysis using two different programs. Analysis of tumors from carriers of truncating mutations revealed that the majority were high histological grade, invasive ductal carcinomas. Young onset was apparent in most families, with 19 breast cancers under 50 years of age, including eight under the age of 40 years. Our data demonstrate the utility of Hi-Plex in the context of high-throughput testing for rare genetic mutations and provide additional timely information about the nature and prevalence of PALB2 mutations, to enhance risk assessment and risk management of women at high risk of cancer attending clinical genetic services.


****

### Abridged adapter primers increase the target scope of Hi-Plex 

Authors: Tú Nguyen-Dumont, Fleur Hammet, Maryam Mahmoodi, Bernard J. Pope, Graham G. Giles, John L. Hopper, Melissa C. Southey, and Daniel J. Park  

Journal: [BioTechniques, 2015](http://www.biotechniques.com/BiotechniquesJournal/2015/January/Abridged-adapter-primers-increase-the-target-scope-of-Hi-Plex/biotechniques-356249.html)

Topics: Multiplex PCR, DNA Sequencing

#### Abstract 

Previously, we reported Hi-Plex, an amplicon-based method for targeted massively parallel sequencing capable of generating 60 amplicons simultaneously. In further experiments, however, we found our approach did not scale to higher amplicon numbers. Here, we report a modification to the original Hi-Plex protocol that includes the use of abridged adapter oligonucleotides as universal primers (bridge primers) in the initial PCR mixture. Full-length adapter primers (indexing primers) are included only during latter stages of thermal cycling with concomitant application of elevated annealing temperatures. Using this approach, we demonstrate the application of Hi-Plex across a broad range of amplicon numbers (16-plex, 62-plex, 250-plex, and 1003-plex) while preserving the low amount (25 ng) of input DNA required.

****

### SRST2: Rapid genomic surveillance for public health and hospital microbiology labs

Authors: Michael Inouye, Harriet Dashnow, Lesley-Ann Raven, Mark B Schultz, Bernard J Pope, Takehiro Tomita, Justin Zobel and Kathryn E Holt 

Journal: [Genome Medicine, 2014](http://genomemedicine.com/content/6/11/90)

Topics: Bioinformatics, Microbiology

#### Abstract 

Rapid molecular typing of bacterial pathogens is critical for public health epidemiology, surveillance and infection control, yet routine use of whole genome sequencing (WGS) for these purposes poses significant challenges. Here we present SRST2, a read mapping-based tool for fast and accurate detection of genes, alleles and multi-locus sequence types (MLST) from WGS data. Using >900 genomes from common pathogens, we show SRST2 is highly accurate and outperforms assembly-based methods in terms of both gene detection and allele assignment. We include validation of SRST2 within a public health laboratory, and demonstrate its use for microbial genome surveillance in the hospital setting. In the face of rising threats of antimicrobial resistance and emerging virulence among bacterial pathogens, SRST2 represents a powerful tool for rapidly extracting clinically useful information from raw WGS data.

****

### Annokey: an annotation tool based on key term search of the NCBI Entrez Gene database

Authors: Daniel J Park, Tú Nguyen-Dumont, Sori Kang, Karin Verspoor and Bernard J Pope 

Journal: [Source Code for Biology and Medicine, 2014](http://www.scfbm.org/content/9/1/15/abstract)

Topics: Bioinformatics, Gene Annotation

#### Abstract 

The NCBI Entrez Gene and PubMed databases contain a wealth of high-quality information about genes for many different organisms. The NCBI Entrez online web-search interface is convenient for simple manual search for a small number of genes but impractical for the kinds of outputs seen in typical genomics projects.
We have developed an efficient open source tool implemented in Python called Annokey, which annotates gene lists with the results of a keyword search of the NCBI Entrez Gene database and linked Pubmed article information. The user steers the search by specifying a ranked list of keywords (including multi-word phrases and regular expressions) that are correlated with their topic of interest. Rank information of matched terms allows the user to guide further investigation.
We applied Annokey to the entire human Entrez Gene database using the key-term "DNA repair" and assessed its performance in identifying the 176 members of a published "gold standard" list of genes established to be involved in this pathway. For this test case we observed a sensitivity and specificity of 97 % and 96 %, respectively.
Annokey facilitates the identification of genes related to an area of interest, a task which can be onerous if performed manually on a large number of genes. Annokey provides a way to capitalize on the high quality information provided by the Entrez Gene database allowing both scalability and compatibility with automated analysis pipelines, thus offering the potential to significantly enhance research productivity.

****

### Rare mutations in RINT1 predispose carriers to breast and Lynch Syndrome-spectrum cancers

Authors: Daniel J Park, Kayoko Tao, Florence Le Calvez-Kelm, Tu Nguyen-Dumont, Nivonirina Robinot, Fleur Hammet, Fabrice Odefrey, Helen Tsimiklis, Zhi L Teo, Louise B Thingholm, Erin L Young, Catherine Voegele, Andrew Lonie, Bernard J Pope, Terrell C Roane, Russell Bell, Hao Hu, Shankaracharya, Chad D Huff, Jonathan Ellis, Jun Li, Igor V Makunin, Esther M. John, Irene L. Andrulis, Mary Beth Terry, Mary Daly, Saundra S. Buys, Carrie Snyder, Henry T Lynch, Peter Devilee, Graham G. Giles, John L. Hopper, Bing-Jian Feng, Fabienne Lesueur, Sean Tavtigian, Melissa C. Southey and David E Goldgar.

Journal: [Cancer Discovery, 2014](http://cancerdiscovery.aacrjournals.org/content/early/2014/05/02/2159-8290.CD-14-0212.abstract)

Topics: Bioinformatics, Genomics, DNA Sequencing, Breast Cancer

#### Abstract 

Approximately half of the familial aggregation of breast cancer remains unexplained. A multiple-case breast cancer family exome sequencing study identified three likely pathogenic mutations in RINT1 (NM_021930.4) not present in public sequencing databases: RINT1 c.343C>T (p.Q115X), c.1132_1134del (p.M378del) and c.1207G>T (p.D403Y). Based on this finding, a population-based case-control mutation-screening study was conducted and identified 29 carriers of rare (MAF < 0.5%), likely pathogenic variants: 23 in 1,313 early-onset breast cancer cases and 6 in 1,123 frequency-matched controls (OR=3.24, 95%CI 1.29-8.17; p=0.013). RINT1 mutation screening of probands from 798 multiple-case breast cancer families identified 4 additional carriers of rare genetic variants. Analysis of the incidence of first primary cancers in families of women in RINT1-mutation carrying families estimated that carriers were at increased risks of Lynch syndrome-spectrum cancers (SIR 3.35, 95% CI 1.7-6.0; P=0.005), particularly for relatives diagnosed with cancer under age 60 years (SIR 10.9, 95%CI 4.7-21; P=0.0003).

****

### ROVER variant caller: read-pair overlap considerate variant-calling software applied to PCR-based massively parallel sequencing datasets

Authors: B. Pope, T. Nguyen-Dumont, F. Hammet and D. Park.

Journal: [Source Code for Biology and Medicine, 2014](http://www.scfbm.org/content/9/1/3)

Topics: Bioinformatics, Genomics, DNA Sequencing, Variant Calling


#### Abstract 

We recently described Hi-Plex, a highly multiplexed PCR-based target-enrichment system for massively parallel sequencing (MPS), which allows the uniform definition of library size so that subsequent paired-end sequencing can achieve complete overlap of read pairs. Variant calling from Hi-Plex-derived datasets can thus rely on the identification of variants appearing in both reads of read-pairs, permitting stringent filtering of sequencing chemistry-induced errors. These principles underly ROVER software (derived from Read Overlap PCR-MPS variant caller), which we have recently used to report the screening for genetic mutations in the breast cancer predisposition gene PALB2. Here, we describe the algorithms underlying ROVER and its usage.

****

### Hi-Plex for high-throughput mutation screening: application to the breast cancer susceptibility gene PALB2

Authors: T. Nguyen-Dumont, Z.L. Teo, B.J. Pope, F. Hammet, M. Mahmoodi, H. Tsimiklis, N. Sabbaghian, M. Tischkowitz, W.D. Foulkes, Kathleen Cuningham Foundation Consortium for research into Familial Breast cancer (kConFab), G.G. Giles, J.L. Hopper, Australian Breast Cancer Family Registry, M.C. Southey, D.J. Park.

Journal: [BMC Medical Genomics, 2013](http://www.biomedcentral.com/1755-8794/6/48)

Topics: Bioinformatics, Genomics, DNA Sequencing, Breast Cancer, Multiplex PCR

#### Abstract 

Massively parallel sequencing (MPS) has revolutionised biomedical research and offers enormous capacity for clinical application. We previously reported Hi-Plex, a streamlined highly-multiplexed PCR-MPS approach, allowing a given library to be sequenced with both the Ion Torrent and TruSeq chemistries. Comparable sequencing efficiency was achieved using material derived from lymphoblastoid cell lines and formalin-fixed paraffin-embedded tumour. Here, we report high-throughput application of Hi-Plex by performing blinded mutation screening of the coding regions of the breast cancer susceptibility gene PALB2 on a set of 95 blood-derived DNA samples that had previously been screened using Sanger sequencing and high-resolution melting curve analysis (n = 90), or genotyped by Taqman probe-based assays (n = 5). Hi-Plex libraries were prepared simultaneously using relatively inexpensive, readily available reagents in a simple half-day protocol followed by MPS on a single MiSeq run. We observed that 99.93% of amplicons were represented at ≥10X coverage. All 56 previously identified variant calls were detected and no false positive calls were assigned. Four additional variant calls were made and confirmed upon re-analysis of previous data or subsequent Sanger sequencing. These results support Hi-Plex as a powerful approach for rapid, cost-effective and accurate high-throughput mutation screening. They further demonstrate that Hi-Plex methods are suitable for and can meet the demands of high-throughput genetic testing in research and clinical settings. 

****

### MYRF Is a Membrane-Associated Transcription Factor That Autoproteolytically Cleaves to Directly Activate Myelin Genes

Authors: H. Bujalka, M. Koenning, S. Jackson, V.M. Perreau, B. Pope, C.M. Hay, S. Mitew, A.F. Hill, Q.R. Lu, M. Wegner, R. Srinivasan, J. Svaren, M. Willingham, B.A. Barres, B. Emery.

Journal: [PLOS Biology, 2013](http://www.plosbiology.org/article/info%3Adoi%2F10.1371%2Fjournal.pbio.1001625). 

Topics: Bioinformatics, DNA Sequencing, Neurobiology 

#### Abstract

The myelination of axons is a crucial step during vertebrate central nervous system (CNS) development, allowing for rapid and energy efficient saltatory conduction of nerve impulses. Accordingly, the differentiation of oligodendrocytes, the myelinating cells of the CNS, and their expression of myelin genes are under tight transcriptional control. We previously identified a putative transcription factor, Myelin Regulatory Factor (Myrf), as being vital for CNS myelination. Myrf is required for the generation of CNS myelination during development and also for its maintenance in the adult. It has been controversial, however, whether Myrf directly regulates transcription, with reports of a transmembrane domain and lack of nuclear localization. Here we show that Myrf is a membrane-associated transcription factor that undergoes an activating proteolytic cleavage to separate its transmembrane domain-containing C-terminal region from a nuclear-targeted N-terminal region. Unexpectedly, this cleavage event occurs via a protein domain related to the autoproteolytic intramolecular chaperone domain of the bacteriophage tail spike proteins, the first time this domain has been found to play a role in eukaryotic proteins. Using ChIP-Seq we show that the N-terminal cleavage product directly binds the enhancer regions of oligodendrocyte-specific and myelin genes. This binding occurs via a defined DNA-binding consensus sequence and strongly promotes the expression of target genes. These findings identify Myrf as a novel example of a membrane-associated transcription factor and provide a direct molecular mechanism for its regulation of oligodendrocyte differentiation and CNS myelination.

****

### Cross-platform compatibility of Hi-Plex, a streamlined approach for targeted massively parallel sequencing

Authors: T. Nguyen-Dumont, B.J. Pope, F. Hammet, M. Mahmoodi, H. Tsimiklis, M.C. Southey, D.J. Park.

Journal: [Analytical Biochemistry, 2013](http://www.ncbi.nlm.nih.gov/pubmed/23933242)

Topics: Bioinformatics, Genomics, DNA Sequencing, Multiplex PCR

#### Abstract

While per-base sequencing costs have decreased in recent years, library preparation for targeted massively parallel sequencing remains constrained by high reagent cost, limited design flexibility and protocol complexity. To address these limitations, we previously developed Hi-Plex, a PCR-massively parallel sequencing strategy for screening panels of genomic target regions. Here, we demonstrate that Hi-Plex applied with hybrid adapters can generate a library suitable for sequencing with both the Ion Torrent and the TruSeq chemistries, and that adjusting primer concentrations improves coverage uniformity. These results expand Hi-Plex capabilities as an accurate, affordable, flexible and rapid approach for various genetic screening applications.

****

### A high-plex PCR approach for massively parallel sequencing

Authors: T. Nguyen-Dumont, B.J. Pope, F Hammet, M.C. Southey, and D.J. Park.

Journal: [BioTechniques, Vol. 55, No. 2, August 2013, pp. 69–74](http://www.biotechniques.com/BiotechniquesJournal/2013/August/A-high-plex-PCR-approach-for-massively-parallel-sequencing/biotechniques-345734.html)

Topics: Bioinformatics, Genomics, DNA Sequencing, Multiplex PCR

#### Abstract 

Current methods for targeted massively parallel sequencing (MPS) have several drawbacks, including limited design flexibility, expense, and protocol complexity, which restrict their application to settings involving modest target size and requiring low cost and high throughput. To address this, we have developed Hi-Plex, a PCR-MPS strategy intended for high-throughput screening of multiple genomic target regions that integrates simple, automated primer design software to control product size. Featuring permissive thermocycling conditions and clamp bias reduction, our protocol is simple, cost- and time-effective, uses readily available reagents, does not require expensive instrumentation, and requires minimal optimization. In a 60-plex assay targeting the breast cancer predisposition genes PALB2 and XRCC2, we applied Hi-Plex to 100 ng LCL-derived DNA, and 100 ng and 25 ng FFPE tumor-derived DNA. Altogether, at least 86.94% of the human genome-mapped reads were on target, and 100% of targeted amplicons were represented within 25-fold of the mean. Using 25 ng FFPE-derived DNA, 95.14% of mapped reads were on-target and relative representation ranged from 10.1-fold lower to 5.8-fold higher than the mean. These results were obtained using only the initial automatically-designed primers present in equal concentration. Hi-Plex represents a powerful new approach for screening panels of genomic target regions.

****

### FAVR (Filtering and Annotation of Variants that are Rare): methods to facilitate the analysis of rare germline genetic variants from massively parallel sequencing datasets

Authors: B.J. Pope, T. Nguyen-Dumont, F. Odefrey, R. Bell, S.V. Tavtigian, D.E. Goldgar, A. Lonie, M.C. Southey and D.J. Park.

Journal: [BMC Bioinformatics, 25 February 2013, 14:65](http://www.biomedcentral.com/1471-2105/14/65/abstract).

Topics: Bioinformatics, Genomics, DNA Sequencing, Variant Detection

#### Abstract 

FAVR is a suite of new methods designed to work with commonly used MPS analysis pipelines to assist in the resolution of some of the issues related to the analysis of the vast amount of resulting data, with a focus on relatively rare genetic variants. To the best of our knowledge, no equivalent method has previously been described. The most important and novel aspect of FAVR is the use of signatures in comparator sequence alignment files during variant filtering, and annotation of variants potentially shared between individuals. The FAVR methods use these signatures to facilitate filtering of (i) platform and/or mapping- specific artefacts, (ii) common genetic variants, and, where relevant, (iii) artefacts derived from imbalanced paired-end sequencing, as well as annotation of genetic variants based on evidence of co-occurrence in individuals. We applied conventional variant calling applied to whole-exome sequencing datasets, produced using both SOLiD and TruSeq chemistries, with or without downstream processing by FAVR methods. We demonstrate a 3-fold smaller rare single nucleotide variant shortlist with no detected reduction in sensitivity. This analysis included Sanger sequencing of rare variant signals not evident in dbSNP131, assessment of known variant signal preservation, and comparison of observed and expected rare variant numbers across a range of first cousin pairs. The principles described herein were applied in our recent publication identifying XRCC2 as a new breast cancer risk gene and have been made publically available as a suite of software tools.

****

### Identification of new breast cancer predisposition genes via whole exome sequencing  

Authors: MC Southey, DJ Park, F Lesueur, F Odefrey, T Nguyen-Dumont, F Hammet, SL Neuhausen, EM John, IL Andrulis, G Chenevix-Trench, L Baglietto, F Le Calvez-Kelm, M Pertesi, A Lonie, B Pope, O Sinilnikova, H Tsimiklis, MCCS, BCFR, kConFab, GG Giles, JL Hopper, SV Tavtigian and DE Goldgar  

Journal: [Hereditary Cancer in Clinical Practice, 2012](http://hccpjournal.biomedcentral.com/articles/10.1186/1897-4287-10-S2-A40)

Topics: Breast Cancer, Exome Sequencing, Inherited risk

#### Abstract 

The application of massively parallel sequencing (MPS) platforms has begun to revolutionize our understanding of the immense variation in the human genome and the complexity that can underlie genetic susceptibility to disease. The utility of exome capture MPS through the identification of genes for rare Mendelian disorders based on analysis of only a few individuals has been eloquently demonstrated. Common diseases such as breast cancer present substantially increased complexity in terms of locus, allelic and phenotypic heterogeneity, as well as complex relationships between genotype and phenotype (reduced penetrance, phenocopies etc.). With careful consideration of study design [1], thoughtful selection of families from our international resources (whole exome sequencing of two highly selected affected members of multiple-case breast cancer families), and a well-developed strategy (analytical pipeline) for distinguishing the few true breast cancer susceptibility genes from the many genes that have rare genetic variants that could plausibly alter protein function, we are advancing a large program of work aimed at identifying the majority of the “missing heritability” of breast cancer.


****

### Expanded genetic analysis of a PALB2 c.3113G>A mutation carrying multiple-case breast cancer family via exome sequencing  

Authors: ZL Teo, DJ Park, F Odefrey, F Hammet, T Nguyen-Dumont, H Tsimiklis, BJ Pope, A Lonie, I Winship, GG Giles, JL Hopper, and MC Southey1, ABCFS  

Journal: [Hereditary Cancer in Clinical Practice, 2012](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3327032/)

Topics: Breast Cancer, Exome Sequencing, PALB2

#### Abstract 

PALB2 is a breast cancer (BC) susceptibility gene. Its product is the binding partner of BRCA1 and BRCA2 and is involved in DNA repair. Studies of multiple-case BC families have reported that truncating mono-allelic PALB2 mutations, on average, increase BC risk two to six fold.

Whole exome capture followed by massively parallel sequencing was performed on four strategically selected affected females of this family. We have shortlisted 15 variants in this family predicted to be protein damaging by SIFT in genes with plausible relevance to cancer aetiology. Eight such variants are predicted to result in protein truncation or are variants predicted to have strong effects on splicing efficiency, one of which have been observed in key DNA repair genes. Six of the genes have roles in DNA repair, three in cell cycle checkpoint regulation, one controls telomerase activity and another has been reported to be a prostate cancer predisposition gene. Further analyses of these candidate BC predisposition genes will be presented.

****

### Bpipe: a tool for running and managing bioinformatics pipelines

Authors: Simon P. Sadedin, Bernard Pope and Alicia Oshlack.

Journal: [Bioinformatics, Volume 28, Issue 11, 2012 pages 1525-1526](http://bioinformatics.oxfordjournals.org/content/28/11/1525). 

Topics: Bioinformatics, Pipelines, Workflows, Automation

#### Abstract 

Bpipe is a simple, dedicated programming language for defining and executing bioinformatics pipelines. It specializes in enabling users to turn existing pipelines based on shell scripts or command line tools into highly flexible, adaptable and maintainable workflows with a minimum of effort. Bpipe ensures that pipelines execute in a controlled and repeatable fashion and keeps audit trails and logs to ensure that experimental results are reproducible. Requiring only Java as a dependency, Bpipe is fully self-contained and cross-platform, making it very easy to adopt and deploy into existing environments.

****

### Rare Mutations in XRCC2 Increase the Risk of Breast Cancer

Authors: D.J. Park, F. Lesueur, T. Nguyen-Dumont, M. Pertesi, F. Odefrey, F. Hammet, S.L. Neuhausen, E.M. John, I.L. Andrulis, M.B. Terry, M. Daly, S. Buys, F. Le Calvez-Kelm, A. Lonie, B.J. Pope, H. Tsimiklis, C. Voegele, F.M. Hilbers, N. Hoogerbrugge, A. Barroso, A. Osorio, the Breast Cancer Family Registry the Kathleen Cuningham Foundation Consortium for Research into Familial Breast Cancer G.G. Giles, P. Devilee, J. Benitez, J.L. Hopper, S.V. Tavtigian, D.E. Goldgar, M.C. Southey. 

Journal: [American Journal of Human Genetics, Volume 90, Issue 4, 6 April 2012, Pages 734–739](http://www.sciencedirect.com/science/article/pii/S0002929712001450). 

Video about this work: [Visions: Breast Cancer Gene Discovery](http://www.youtube.com/watch?v=3kGsTCUOQss)

Topics: Breast Cancer, Bioinformatics, DNA Sequencing

#### Abstract 

An exome-sequencing study of families with multiple breast-cancer-affected individuals identified two families with XRCC2 mutations, one with a protein-truncating mutation and one with a probably deleterious missense mutation. We performed a population-based case-control mutation-screening study that identified six probably pathogenic coding variants in 1,308 cases with early-onset breast cancer and no variants in 1,120 controls (the severity grading was p < 0.02). We also performed additional mutation screening in 689 multiple-case families. We identified ten breast-cancer-affected families with protein-truncating or probably deleterious rare missense variants in XRCC2. Our identification of XRCC2 as a breast cancer susceptibility gene thus increases the proportion of breast cancers that are associated with homologous recombination-DNA-repair dysfunction and Fanconi anemia and could therefore benefit from specific targeted treatments such as PARP (poly ADP ribose polymerase) inhibitors. This study demonstrates the power of massively parallel sequencing for discovering susceptibility genes for common, complex diseases.

****

### Petascale computation performance of lightweight multiscale cardiac models using Hybrid Programming Models

Authors: B.J. Pope, B.G. Fitch, M.C. Pitman, J.J. Rice, M. Reumann.  

Conference: [IEEE Engingeering in Medicine and Biology Society, 2011, pages 433-6](http://www.ncbi.nlm.nih.gov/pubmed/22254341). 

Topics: Cardiac Modelling, High Performance Computing, Hybrid Parallel Programming, Simulation

#### Abstract

Future multiscale and multiphysics models must use the power of high performance computing (HPC) systems to enable research into human disease, translational medical science, and treatment. Previously we showed that computationally efficient multiscale models will require the use of sophisticated hybrid programming models, mixing distributed message passing processes (e.g. the message passing interface (MPI)) with multithreading (e.g. OpenMP, POSIX pthreads). The objective of this work is to compare the performance of such hybrid programming models when applied to the simulation of a lightweight multiscale cardiac model. Our results show that the hybrid models do not perform favourably when compared to an implementation using only MPI which is in contrast to our results using complex physiological models. Thus, with regards to lightweight multiscale cardiac models, the user may not need to increase programming complexity by using a hybrid programming approach. However, considering that model complexity will increase as well as the HPC system size in both node count and number of cores per node, it is still foreseeable that we will achieve faster than real time multiscale cardiac simulations on these systems using hybrid programming models. 

****

### Performance of Hybrid Programming Models for Multiscale Cardiac Simulations: Preparing for Petascale Computation

Authors: B.J. Pope, B.G. Fitch, M.C. Pitman, J.J. Rice, M. Reumann.

Journal: [IEEE Transactions on Biomedical Engineering. Volume 58, Issue 10, October 2011](http://ieeexplore.ieee.org/xpls/icp.jsp?arnumber=5951744). 

Topics: Cardiac Modelling, High Performance Computing, Hybrid Parallel Programming, Simulation

#### Abstract

Future multiscale and multiphysics models that support research into human disease, translational medical science, and treatment can utilize the power of high-performance computing (HPC) systems. We anticipate that computationally efficient multiscale models will require the use of sophisticated hybrid programming models, mixing distributed message-passing processes [e.g., the message-passing interface (MPI)] with multithreading (e.g., OpenMP, Pthreads). The objective of this study is to compare the performance of such hybrid programming models when applied to the simulation of a realistic physiological multiscale model of the heart. Our results show that the hybrid models perform favorably when compared to an implementation using only the MPI and, furthermore, that OpenMP in combination with the MPI provides a satisfactory compromise between performance and code complexity. Having the ability to use threads within MPI processes enables the sophisticated use of all processor cores for both computation and communication phases. Considering that HPC systems in 2012 will have two orders of magnitude more cores than what was used in this study, we believe that faster than real-time multiscale cardiac simulations can be achieved on these systems.

****

### High Performance Haskell with MPI

Authors: Bernard Pope and Dmitry Astapov

Paper: [The Monad Reader, Issue 19, October 2011](/docs/haskell-mpi.monad.reader.pdf)

Topics: Distributed Parallel Programming, Functional Programming, High Performance Computing

#### Abstract

In this article, we give a brief overview of the Haskell-MPI library and show how it can be used to write distributed parallel programs. We use the trapezoid method for approximating definite integrals as a motivating example and compare the performance of an implementation using Haskell-MPI to three variations of the same algorithm: a sequential Haskell program, a multi-threaded Haskell program, and a C program also using MPI.

****

### Step inside the GHCi debugger

Authors: Bernard Pope

Paper: [The Monad Reader, Issue 10, April 2008](/docs/ghci-debug.monad.reader.pdf)

Topics: Debugging, Functional Programming, Haskell

#### Abstract

Major releases of GHC are highly anticipated events, especially because of all the exciting new features they bring. The 6.8 series was a particularly impressive example, which came with lots of goodies, including a shiny new debugger. In this article we take the debugger out for a test run, and see what it can do.

****

### A Lightweight Interactive Debugger for Haskell

Authors: Simon Marlow, Jose Iborra, Bernard Pope and Andy Gill. 

Conference: [The Haskell Workshop, June 2007](/docs/ghci-debug.pdf)

Topics: Debugging, Functional Programming, Haskell

#### Abstract 

This paper describes the design and construction of a Haskell source-level debugger built into the GHCi interactive environment. We have taken a pragmatic approach: the debugger is based on the traditional stop-examine-continue model of online debugging, which is simple and intuitive, but has traditionally been shunned in the context of Haskell because it exposes the lazy evaluation order. We argue that this drawback is not as severe as it may seem, and in some cases is an advantage. The design focuses on availability: our debugger is intended to work on all programs that can be compiled with GHC, and without requiring the programmer to jump through additional hoops to debug their program. The debugger has a novel approach for reconstructing the type of runtime values in a polymorphic context. Our implementation is light on complexity, and was integrated into GHC without significant upheaval.

****

### Getting a Fix from the Right Fold

Authors: Bernard Pope    

Paper: [The Monad Reader, Issue 6, January 2007](http://www.haskell.org/sitewiki/images/1/14/TMR-Issue6.pdf)


Topics: Fixed Points, Folds, Functional Programming

#### Abstract 

What can you do with foldr? This is a seemingly innocent question that will confront most functional programmers at some point in their life. I was recently posed a folding challenge by a teaching colleague. The challenge was to write dropWhile using foldr. We gave the challenge to our first-year students, and awarded a small prize to the author of the first working solution. I have since passed the challenge on to other functional friends, and the results have been illuminating. That prompted me to write this article.

****

### Declarative Debugging with buddha

Authors: Bernard Pope

Conference: [Advanced Functional Programming, 5th International School, AFP 2004, volume 3622 of Lecture Notes in Computer Science, pages 331-357. Springer Verlag, September 2005](/docs/Afp.pdf).

Topics: Debugging, Functional Programming, Program Transformation

#### Abstract 

Haskell is a very safe language, particularly because of its type system. However there will always be programs that do the wrong thing. Programmer fallibility, partial or incorrect specifications and typographic errors are but a few of the reasons that make bugs a fact of life. This paper is about the use and implementation of a debugger, called buddha, which helps Haskell programmers understand why their programs misbehave. Traditional debugging tools that examine the program execution step-by-step are not suitable for Haskell because of its unorthodox evaluation strategy. Instead, a different approach is taken which abstracts away the evaluation order of the program and focuses on its high-level logical meaning. This style of debugging is called Declarative Debugging, and it has its roots in the Logic Programming community. At the heart of the debugger is a tree which records information about the evaluation of the program in a manner which is easy to relate to the structure of the source code. It resembles a call graph annotated with the arguments and results of function applications, shown in their most evaluated form. Logical relationships between entities in the source are reflected in the links between nodes in the tree. An error diagnosis algorithm is applied to the tree in a top-down fashion in the search for causes of bugs.

Slides for the talk are available: [Set 1](/docs/Afp.slides1.pdf), [Set 2](/docs/Afp.slides2.pdf) and [Set 3](/docs/Afp.slides3.pdf).

****

### Practical Aspects of Declarative Debugging in Haskell 98

Authors: Bernard Pope and Lee Naish.

Conference: [Principles and Practice of Declarative Programming, 2003](/docs/PopeNaishPPDP.pdf)

Topics: Debugging, Functional Programming, Program Transformation

#### Abstract

Non-strict purely functional languages pose many challenges to the designers of debugging tools. Declarative debugging has long been considered a suitable candidate for the task due to its abstraction over the evaluation order of the program, although the provision of practical implementations has been lagging. In this paper we discuss the solutions used in our declarative debugger for Haskell to tackle the problems of printing values, memory usage and I/O. The debugger is based on program transformation, although much leverage is gained by interfacing with the runtime environment of the language implementation through a foreign function interface.

****

### A program transformation for debugging Haskell 98

Authors: Bernard Pope and Lee Naish.

Conference: [Australasian Computer Science Conference, 2003](/docs/acsc.pdf)

Topics: Debugging, Functional Programming, Program Transformation

We present a source-to-source transformation of Haskell 98 pro-grams for the purpose of debugging. The source code of a program is transformed into a new program which, when executed,computes the value of the original program and a high-level semantics for that computation. The semantics is given by a tree whose nodes represent function applications that were evaluated during execution. This tree is useful in situations where a high-level view of a computation is needed, such as declarative debugging. The main contribution of the paper is the treatment of higher-order functions, which have previously proven difficult to support in declarative debugging schemes.

****

### A program transformation for Declarative Debugging (draft)

Authors: Bernard Pope and Lee Naish

[Paper](/docs/pope.naish.apepm.pdf)

Topics: Debugging, Functional Programming, Program Transformation

#### Abstract 

We present a declarative debugger for lazy functional programs, based primarily on program transformation. The debugger is designed to assist the detection and location of errors in programs which produce incorrect results for some or all of their inputs. We define the transformation over a core functional language, and pay close attention to the treatment of curried function applications. We consider the space complexity of the debugger, and sketch a method for improvement. We use Haskell as the target of the transformation, and consider extending the transformation to support the entire language.

****

### Reification in Haskell (draft)

Authors: Bernard Pope and Lee Naish

Workshop: [Implementation of Functional Languages, 2001](/docs/reify.pdf)

Topics: Functional Programming, Meta Programming, Debugging

#### Abstract

In this paper we investigate a limited form of reification in Haskell. We incrementally develop a meta-representation with a view to expressing the lazy evaluation and cyclic sharing of first-order values. We implement most of our facility within Haskell for portability and make extensive use of Haskell's type classes. We show how this metarepresentation can be derived for algebraic data types, and illustrate its use through a generic printing facility and a re-evaluation scheme. We also briefly explore a means for converting meta-representations back into their object level values.

[Slides](/docs/reify.talk.pdf)

****

### Specialisation of Higher-Order Functions for Debugging

Authors: Bernard Pope and Lee Naish

Workshop: [Workshop on Functional and Logic Programming, Kiel, Germany 2001](/docs/specialise.pdf)

Topics: Programming Languages, Functional Programming, Debugging

#### Abstract

Because functions are abstract values without convenient print representations, implementing debuggers which support higher-order code is a challenge. We present an algorithm for statically specialising higher-order functions and encoding higher-order values to allow printing. We define our algorithm for a small functional language and discuss how it may be extended to support sophisticated features of modern functional programming languages. This research forms part of a project to build a declarative debugger for Haskell, based primarily on source-to-source transformation.

[Slides](/docs/spec.talk.pdf)

****

### Buddha: A Declarative Debugger for Haskell

Authors: Bernard Pope

[Honours Thesis](/docs/BerniePope.Hons.Thesis.pdf)

Topics: Debugging, Functional Programming, Program Transformation

#### Abstract

Due to their reliance on the execution order of programs, traditional debugging techniques are not well suited to locating the source of logical errors in programs written in lazy functional languages. We describe the implementation of a declarative debugger for the programming language Haskell, which assists the location of logical errors based on the declarative semantics of program definitions. The implementation is based on the Hugs interpreter, and both solidifies previous work in the field and extends it to incorporate features typical of many modern lazy functional languages.

****

### A tour of the Haskell Prelude

Authors: Bernard Pope

[Paper](/docs/haskell.tour.pdf)

Topics: Haskell

#### Abstract

A guide to the functions, operators and classes of the Haskell 98 Prelude.

This paper serves as a reference guide for the functions and types in the Haskell Prelude (the standard library which is implicitly imported into all Haskell modules). The intended audience is people who are starting to learn Haskell. It has proven particularly useful in teaching Haskell at university level, and a HTML version of the document is widely available on the Internet.

****

### Monadic Parsing: A Case Study

Authors: B. Pope, M. Wielaard and S. Taylor. 

[Paper](/docs/monad_parse.pdf)

Topics: Parsing, Monads, Functional Programming

#### Abstract

One of the selling points for functional languages is the ease with which parsers for simple languages can be expressed. In this report we give an introduction to monadic and operator precedence parsing in functional languages, using the parsing of propositional logic formulas as an example.

****

### Haskell for Miranda Programmers

Authors: Bernard Pope and Kevin Glynn. 

[Paper](/docs/mira2hask.pdf)

Topics: Functional Programming

#### Abstract

This document is designed to help programmers with a knowledge of Miranda move to Haskell as quickly and as painlessly as possible.
