<!doctype html>
<html lang="en">
  <head>
     
    <title>Bernie Pope: Presentations, Posters and Media</title>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css"
          integrity="sha384-PsH8R72JQ3SOdhVi3uxftmaW6Vc51MKb0q5P2rRUpPvrszuE4W1povHYgTpBfshb" crossorigin="anonymous">
   <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css" rel="stylesheet" />
    <link rel="stylesheet" href="assets/css/website.css">
      </head>

  <body>
      <div class="container"> 
      <!-- <div class="container-fluid"">  -->
         

      <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
            <a class="navbar-brand" href="index.html">Bernie Pope</a>
            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
              <span class="navbar-toggler-icon"></span>
            </button>
            <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
              <div class="navbar-nav">

                                  <a class="nav-item nav-link"href="index.html">Home</a>
                                  <a class="nav-item nav-link"href="publications.html">Publications</a>
                                  <a class="nav-item nav-link active"href="presentations.html">Presentations</a>
                                  <a class="nav-item nav-link"href="software.html">Software</a>
                                  <a class="nav-item nav-link"href="funding.html">Funding</a>
                                  <a class="nav-item nav-link"href="teaching.html">Teaching</a>
                                  <a class="nav-item nav-link"href="contact.html">Contact</a>
                
              </div>
            </div>
          </nav>
      

<h1 class="display-4 my-5">Presentations, Posters and Media</h1>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Colorectal cancer susceptibility genes: findings from whole genome, exome and targeted sequencing of early onset and multiple-case families, Familial Aspects of Cancer: Research and Practice, 1 September, 2017</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Bernard Pope, Khalid Mahmood, Mark Clendenning, Christophe Rosty, Marie Lorans, Harindra Jayasekara, Neil O’Callaghan, Susan Preston, Daniel J. Park, Fleur Hammet, Tu Nguyen-Dumont, Ashton Connor, Steven Gallinger, David Duggan, Graham Casey, Stephen N. Thibodeau, John L. Hopper, Melissa C. Southey, Aung K. Win, Finlay A. Macrae, Ingrid M. Winship, Mark A. Jenkins, Daniel D. Buchanan for the Australasian Colorectal Cancer Family Registry</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Oral presentation</dd>
              
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">Familial Aspects of Cancer: Research and Practice</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Kingscliff, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">1 September, 2017</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>The Genetics of Colonic Polyposis Study: identifying the genetic basis of unexplained colonic polyposis, Familial Aspects of Cancer: Research and Practice, 29 August - 1 September, 2017</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Sharelle Joseland, Mark Clendenning, Bernard J. Pope, Khalid Mahmood, Julie Arnold, Sonja Woodall, Marie Lorans, Neil O’Callaghan, Susan Preston, Chloe Wheatland, Kevin Sweet, Kara Semotiuk, Melissa C. Southey, Aung Ko Win, Mark A. Jenkins, Christophe Rosty, Finlay A. Macrae, Ingrid M. Winship, Susan Parry, Daniel D. Buchanan and FCCs</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Poster</dd>
              
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">Familial Aspects of Cancer: Research and Practice</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Kingscliff, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">29 August - 1 September, 2017</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Hi-Plex Origin for the high-throughput screening of disease genes, Familial Aspects of Cancer: Research and Practice, 29 August - 1 September, 2017</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Thomas Green, Fleur Hammet, Khalid Mahmood, Tu Nguyen-Dumont, Melissa Southey, Bernard Pope, Daniel Park</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Poster</dd>
              
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">Familial Aspects of Cancer: Research and Practice</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Kingscliff, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">29 August - 1 September, 2017</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>A registry of Australian Genomics bioinformatics pipelines incorporating a structured representation and interactive visualisation, Australian Genomics National Conference, 4 August 2017</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Mailie Gall, Bernard Pope, John Pearson, Natalie Thorne</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Poster</dd>
              
                              <dt class="col-sm-2">Media</dt>
		<dd class="col-sm-10">
                  		      <a href="assets/files/AGHA_2017_poster.pdf">PDF poster</a>
                                  </dd>
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">Australian Genomics National Conference</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Brisbane, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">4 August 2017</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Scientists encode GIF into DNA, ABC Radio National Interview, 13 July 2017</summary>
          <div class="card card-body my-3">
            <dl class="row">

              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Interview</dd>
              
                              <dt class="col-sm-2">Media</dt>
		<dd class="col-sm-10">
                  		      <a href="http://www.abc.net.au/radionational/programs/drive/dna-data-storage/8706456">Link to ABC Radio National interview recording</a>
                                  </dd>
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">ABC Radio National Interview</dd>
              
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">13 July 2017</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Characterisation of Mismatch Repair variants submitted to the International Mismatch Repair Consortium (IMRC), The International Society for Gastrointestinal Hereditary Tumours (InSiGHT), biennial meeting, 6 - 8 July, 2017</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Jeanette Reece, Daniel Buchanan, John-Paul Plazzer, Khalid Mahmood, Bernard Pope, Mark Clendenning, Grant Lee, Aung Win, Allyson Templeton, Robert Haile, Gabriela Moslein, Finlay Macrae, Mark Jenkins</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Poster</dd>
              
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">The International Society for Gastrointestinal Hereditary Tumours (InSiGHT), biennial meeting</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Florence, Italy</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">6 - 8 July, 2017</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Whole genome sequencing as a diagnostic tool for Lynch Syndrome, The International Society for Gastrointestinal Hereditary Tumours (InSiGHT), biennial meeting, 6 July, 2017</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">B Pope, M Clendenning, K Mahmood, C Rosty, R Walters, A Win, J Hopper, M Southey, F Macrae, I Winship, M Jenkins, A Spurdle, D Buchanan</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Oral presentation</dd>
              
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">The International Society for Gastrointestinal Hereditary Tumours (InSiGHT), biennial meeting</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Florence, Italy</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">6 July, 2017</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Fuzz testing with American Fuzzy Lop (AFL), VASET Group, The University of Melbourne, 18 August 2016</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Bernard Pope</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Oral presentation</dd>
              
                              <dt class="col-sm-2">Media</dt>
		<dd class="col-sm-10">
                  		      <a href="assets/files/afl_fuzz.pdf">PDF slides</a>
                                  </dd>
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">VASET Group, The University of Melbourne</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Melbourne, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">18 August 2016</dd>
              
                              <dt class="col-sm-2">Abstract </dt>
                <dd class="col-sm-10"><a href="http://lcamtuf.coredump.cx/afl/">American Fuzzy Lop (AFL)</a> is a fuzz testing tool that has helped detect an impressive set of bugs. It uses compile-time program instrumentation and simple run-time statistics to detect program inputs which trigger novel execution paths. An initial program input is supplied by the user, from which a stream of new cases are generated based on a carefully tuned "genetic" algorithm (author's terminology). In some cases AFL can synthesise complex input file structure from fairly distant starting points. A useful by-product of running AFL is a compact collection of synthetic test inputs which can be used for other testing regimes.  AFL's runtime overheads are very low, and, according to the author, real-word fuzzed programs can run at near-native speed.  In this talk I give an overview of AFL and attempt to explain some of its implementation details.
</dd>
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Clinical Genomics: a Computational Perspective, Linux Conference Australia, 1 February 2016</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Bernard Pope</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Oral presentation</dd>
              
                              <dt class="col-sm-2">Media</dt>
		<dd class="col-sm-10">
                  		    <ul>
		    		      <li><a href="assets/files/clinical_genomics_lca_2016.pdf">PDF slides</a></li>
                    		      <li><a href="https://www.youtube.com/watch?v=tGTVB-dqji4&feature=youtu.be">YouTube video</a></li>
                    		    </ul>
                                  </dd>
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">Linux Conference Australia</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Geelong, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">1 February 2016</dd>
              
                              <dt class="col-sm-2">Abstract </dt>
                <dd class="col-sm-10">Low cost highly accurate DNA sequencing is quickly becoming an option for clinical diagnosis and treatment. In this talk I discuss the computational aspects of Clinical Genomics, based on recent experiences with the Melbourne Genomics Health Alliance. This was an invited talk for the Open Source and Bioinformatics miniconf, as part of the Linux Conference in Geelong 2016.  
</dd>
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>OptimusPrime: Multiplex Primer Design Tool for Hi-Plex Sequencing, Australasian Genomic Technologies Association (AGTA), Annual Conference, 9 - 12 October, 2016</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Edmund Lau, Daniel Park, Bernard Pope</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Poster</dd>
              
                              <dt class="col-sm-2">Media</dt>
		<dd class="col-sm-10">
                  		      <a href="assets/files/hiplex_AGTA_poster_2016.pdf">PDF poster</a>
                                  </dd>
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">Australasian Genomic Technologies Association (AGTA), Annual Conference</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Auckland, New Zealand</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">9 - 12 October, 2016</dd>
              
                              <dt class="col-sm-2">Abstract </dt>
                <dd class="col-sm-10">The advent of Massively Parallel Sequencing (MPS) has dramatically reduced the cost and increased the throughput of DNA sequencing. A number of methods have been  developed that target specified genomic regions for MPS. However, these, variously, are compromised by issues of relative expense, accuracy, requirement for specialist  equipment and the cumbersome nature of protocols. In this work we present Hi-Plex, a novel MPS platform, and multiplex Polymerase Chain Reaction (PCR) primer design software to meet the challenges the platform presents.
</dd>
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Navigating the Research Data Life Cycle, The Australian Bioinformatics and Computational Biology Society (AB3ACBS) Conference, 31 October - 2 November, 2016</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">P Griffin, R Appels, D Bulach, K Dudley, G Keeble-Gagnere, A Pask, B Pope, U Roessner, T Seemann, D Bolser, J Khadake, S Lewis, S Orchard, S Tyagi, A Lonie, MV Schneider</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Poster</dd>
              
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">The Australian Bioinformatics and Computational Biology Society (AB3ACBS) Conference</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Brisbane, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">31 October - 2 November, 2016</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Whole-genome analysis of paediatric cancer from cases with a family cancer history: early insights from Victoria, Australia, American Society of Human Genetics (ASHG) annual meeting, 18 - 22 October, 2016</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">T Nguyen-Dumont, B Feng, H Tsimiklis, D Amor, E Algar, E Milne, L Spector, J Ng, A Lonie, D Park, B Pope, E Smibert, G Dite, M Southey, J Heath, J Hopper</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Poster</dd>
              
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">American Society of Human Genetics (ASHG) annual meeting</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Vancouver, Canada</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">18 - 22 October, 2016</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>RNASEL, a modifier of breast cancer risk in BRCA1 and BRCA2 carriers?, 6th International Symposium on Hereditary Breast and Ovarian Cancer, 10 – 13 May, 2016</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">T Nguyen-Dumont, A Roberge, D Park, M Kapuscinski, Z Teo, F Hammet, M Mahmoodi, H Tsimiklis, B Pope, A Lonie, K Mahmood, kConFab, ABCFR, J Hopper, G Giles, M Southey</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Poster</dd>
              
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">6th International Symposium on Hereditary Breast and Ovarian Cancer</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Montreal, Canada</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">10 – 13 May, 2016</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Another look at declarative debugging for Haskell, VASET Group, The University of Melbourne, 30 October 2015</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Bernard Pope</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Oral presentation</dd>
              
                              <dt class="col-sm-2">Media</dt>
		<dd class="col-sm-10">
                  		      <a href="assets/files/dd_vaset.pdf">PDF slides</a>
                                  </dd>
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">VASET Group, The University of Melbourne</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Melbourne, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">30 October 2015</dd>
              
                              <dt class="col-sm-2">Abstract </dt>
                <dd class="col-sm-10">Despite rising notoriety, Haskell still lacks effective debugging support. A breakpoint debugger exists, but, as expected, it is difficult to use in the presence of lazy evaluation and higher-order functions. We have previously considered declarative debugging as a more suitable approach for Haskell, but a number of technical challenges remain unsolved. In this talk I will revisit the main ideas behind declarative debugging and outline a few interesting areas of potential research in the context of non-strict purely functional languages, namely: generalising the structure of the debugging tree; interleaving debugging and program execution to reduce memory consumption.  
</dd>
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Mutation screening of RNASEL, a candidate breast cancer susceptibility gene identified via whole-exome sequencing, Familial Aspects of Cancer: Research and Practice, 25 - 28 August, 2015</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">A Roberge, T Nguyen-Dumont, D Park, M Kapuscinski, F Hammet, M Mahmoodi, H Tsimiklis, B Pope, A Lonie, K Mahmood, kConFab, ABCFS, J Hopper, M Southey</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Poster</dd>
              
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">Familial Aspects of Cancer: Research and Practice</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Kingscliff, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">25 - 28 August, 2015</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Lindenmayer Systems, Foundations of Computing Advanced Lecture, The University of Melbourne, 22 August, 2014</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Bernard Pope</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Lecture</dd>
              
                              <dt class="col-sm-2">Media</dt>
		<dd class="col-sm-10">
                  		      <a href="assets/files/lsystem.pdf">PDF slides</a>
                                  </dd>
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">Foundations of Computing Advanced Lecture, The University of Melbourne</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Melbourne, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">22 August, 2014</dd>
              
                              <dt class="col-sm-2">Abstract </dt>
                <dd class="col-sm-10">These slides introduce the idea of turtle graphics in Python and show how they can be used to implement Lindenmayer Systems (term rewriting languages with interesting interpretations as graphics).
</dd>
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Visualizing and analyzing heterogeneous DNA methylation patterns by massively parallel sequencing using methpat, Australian Bioinformatics Conference (ABiC), 4 June, 2014</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">N Wong, I Candiloro, B Pope, D Korbie, M Trau, S Wong, T Mikeska, X Zhang, M Pitman, A Dobrovic</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Poster</dd>
              
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">Australian Bioinformatics Conference (ABiC)</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Melbourne, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">4 June, 2014</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Introducing RedDog, a mapping-based genome comparison pipeline for bacterial high-throughput sequence data, Lorne Infection and Immunity Conference, 19 - 21 February, 2014</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">David Edwards, Bernard Pope, Kat Holt</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Poster</dd>
              
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">Lorne Infection and Immunity Conference</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Lorne, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">19 - 21 February, 2014</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Hi-Plex: a flexible, streamlined and cost-effective approach to targeted massively parallel sequencing, American Society of Human Genetics (ASHG) annual meeting, 18 - 22 October, 2014</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">T Nguyen-Dumont, B Pope, F Hammet, M Mahmoodi, H Tsimiklis, M Southey, D Park</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Poster</dd>
              
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">American Society of Human Genetics (ASHG) annual meeting</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">San Diego, USA</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">18 - 22 October, 2014</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Hi-Plex: a flexible, streamlined and cost-effective approach to targeted massively parallel sequencing, 10th Australasian Mutation Detection meeting, 1 - 4 September, 2014</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">T Nguyen-Dumont, B Pope, F Hammet, M Mahmoodi, H Tsimiklis, M Southey</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Poster</dd>
              
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">10th Australasian Mutation Detection meeting</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Daydream Island, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">1 - 4 September, 2014</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>ALLOCATE: sorting ovarian cancer patients into treatment arms based on molecular and genetic make up of their tumours, Lorne Cancer Conference, 13 - 15 February, 2014</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">O Kondrashova, S Lunke, L Melishkin, C Scott, B Tran, D Bowtell, O McNally, M Wakefield, C Sloggett, B Pope, C Anderson, G Taylor, P Waring</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Poster</dd>
              
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">Lorne Cancer Conference</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Lorne, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">13 - 15 February, 2014</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>How Python Works, Foundations of Computing Advanced Lecture, The University of Melbourne, 10 October, 2014</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Bernard Pope</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Lecture</dd>
              
                              <dt class="col-sm-2">Media</dt>
		<dd class="col-sm-10">
                  		      <a href="assets/files/how_python_works.pdf">PDF slides</a>
                                  </dd>
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">Foundations of Computing Advanced Lecture, The University of Melbourne</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Melbourne, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">10 October, 2014</dd>
              
                              <dt class="col-sm-2">Abstract </dt>
                <dd class="col-sm-10">An overview of how Python works (CPython specifically). Covers syntax analysis (lexing and parsing), compilation to bytecode and program exeuction. The slides conclude with an example bytecode execution of a simple Python program.
</dd>
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Computer games in Python, Foundations of Computing Advanced Lecture, The University of Melbourne, 19 September, 2014</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Bernard Pope</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Lecture</dd>
              
                              <dt class="col-sm-2">Media</dt>
		<dd class="col-sm-10">
                  		    <ul>
		    		      <li><a href="assets/files/games_in_python.pdf">PDF slides</a></li>
                    		      <li><a href="https://github.com/bjpop/asteroids">Code example on GitHub</a></li>
                    		    </ul>
                                  </dd>
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">Foundations of Computing Advanced Lecture, The University of Melbourne</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Melbourne, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">19 September, 2014</dd>
              
                              <dt class="col-sm-2">Abstract </dt>
                <dd class="col-sm-10">These slides show how to implement arcade games in Python using the [PyGame](www.pygame.org/) library.  To demonstrate the main ideas of game programming in Python I wrote a simple asteroids game which is available in the linked code repository.
</dd>
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Ray tracing in Python, Foundations of Computing Advanced Lecture, The University of Melbourne, 5 September, 2014</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Bernard Pope</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Lecture</dd>
              
                              <dt class="col-sm-2">Media</dt>
		<dd class="col-sm-10">
                  		    <ul>
		    		      <li><a href="assets/files/ray_tracing.pdf">PDF slides</a></li>
                    		      <li><a href="https://github.com/bjpop/pyray">Code example on GitHub</a></li>
                    		    </ul>
                                  </dd>
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">Foundations of Computing Advanced Lecture, The University of Melbourne</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Melbourne, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">5 September, 2014</dd>
              
                              <dt class="col-sm-2">Abstract </dt>
                <dd class="col-sm-10">These slides explain the central ideas behind the recursive ray-tracing algorithm and demonstrate how to implement it in Python. A basic Phong Illumination model is used in the slides and program.
</dd>
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Implementing Python in Haskell, twice, The Melbourne Python Users Group, 7 July, 2014</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Bernard Pope</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Oral presentation</dd>
              
                              <dt class="col-sm-2">Media</dt>
		<dd class="col-sm-10">
                  		    <ul>
		    		      <li><a href="assets/files/MPUG_python_haskell.pdf">PDF slides</a></li>
                    		      <li><a href="https://github.com/bjpop/blip">Code repository for Blip</a></li>
                    		      <li><a href="https://github.com/bjpop/berp">Code repository for Berp</a></li>
                    		    </ul>
                                  </dd>
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">The Melbourne Python Users Group</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Melbourne, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">7 July, 2014</dd>
              
                              <dt class="col-sm-2">Abstract </dt>
                <dd class="col-sm-10">Over the past couple of years I've developed an unusual hobby: implementing Python in Haskell.  The first iteration resulted in berp, a Python-to-Haskell translator. The second iteration resulted in blip, which compiles Python to bytecode, and is compatible with the standard Python implementation CPython. Note: I've given a similar talk to the Haskell User's Group (see below). In this version of the talk the focus is oriented to a Python audience.
</dd>
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Functional graphics in Scala, The Melbourne Scala Users Group, 26 May, 2014</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Bernard Pope</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Oral presentation</dd>
              
                              <dt class="col-sm-2">Media</dt>
		<dd class="col-sm-10">
                  		    <ul>
		    		      <li><a href="assets/files/fungraph_scala.pdf">PDF slides</a></li>
                    		      <li><a href="https://github.com/bjpop/scala-fungraph">Example code repository</a></li>
                    		    </ul>
                                  </dd>
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">The Melbourne Scala Users Group</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Melbourne, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">26 May, 2014</dd>
              
                              <dt class="col-sm-2">Abstract </dt>
                <dd class="col-sm-10">Computer graphics are typically represented using two-dimensional arrays of pixels. In this talk I will demonstrate an alternative, and perhaps surprising, representation based on functions. We assume that images are defined (infinitely) over the two-dimensional real coordinate space, which avoids the need to worry about boundary conditions or discretisation. I will also show how this idea can be extended to animations, by making them functions over time. In addition to showing some pretty pictures, I demonstrate that higher-order functions are a powerful tool for program abstraction.
</dd>
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Implementing Python in Haskell, twice, The Melbourne Haskell Users Group, 24 April, 2014</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Bernard Pope</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Oral presentation</dd>
              
                              <dt class="col-sm-2">Media</dt>
		<dd class="col-sm-10">
                  		    <ul>
		    		      <li><a href="assets/files/ImplementPythonInHaskell.pdf">PDF slides</a></li>
                    		      <li><a href="https://github.com/bjpop/blip">Code repository for Blip</a></li>
                    		      <li><a href="https://github.com/bjpop/berp">Code repository for Berp</a></li>
                    		    </ul>
                                  </dd>
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">The Melbourne Haskell Users Group</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Melbourne, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">24 April, 2014</dd>
              
                              <dt class="col-sm-2">Abstract </dt>
                <dd class="col-sm-10">Over the past couple of years I've developed an unusual hobby: implementing Python in Haskell.  The first iteration resulted in berp, a Python-to-Haskell translator. The second iteration resulted in blip, which compiles Python to bytecode, and is compatible with the standard Python implementation CPython.  In this talk I explain how I picked up this hobby, and where it might be going. I also talk about various Haskell features which underpin the two implementations.
</dd>
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Rubra - flexible distributed pipelines, Bioinformatics Open Source Conference (BOSC), 19 July, 2013</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">C Sloggett, M Wakefield, G Philip, B Pope</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Poster</dd>
              
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">Bioinformatics Open Source Conference (BOSC)</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Berlin, Germany</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">19 July, 2013</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Automating Pipelines for Variant Detection, Lorne Genome Conference, 17 - 19 February, 2013</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">C Sloggett, G Philip, B Pope</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Poster</dd>
              
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">Lorne Genome Conference</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Lorne, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">17 - 19 February, 2013</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

  
      </div> <!-- container -->

      <!-- jQuery first, then Popper.js, then Bootstrap JS -->
      <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.3/umd/popper.min.js" integrity="sha384-vFJXuSJphROIrBnz7yo7oB41mKfc8JzQZiCq4NCceLEaO4IHwicKwpJf9c9IpFgh" crossorigin="anonymous"></script>
      <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/js/bootstrap.min.js" integrity="sha384-alpBpkh1PFOepccYVYDB4do5UnbKysX5WZXm3XxPqe5iKTfUKjNkCk9SaVuEZflJ" crossorigin="anonymous"></script>
      <script src="assets/js/site.js"></script>
      <script src="assets/js/details-element-polyfill.js"></script>
  </body>
</html>
