<!doctype html>
<html lang="en">
  <head>
     
    <title>Bernie Pope: Presentations, Posters and Media</title>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css"
          integrity="sha384-PsH8R72JQ3SOdhVi3uxftmaW6Vc51MKb0q5P2rRUpPvrszuE4W1povHYgTpBfshb" crossorigin="anonymous">
    <link rel="stylesheet" href="assets/css/website.css">
      </head>

  <body>
      <div class="container"> 
      <!-- <div class="container-fluid"">  -->
         

      <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
            <a class="navbar-brand" href="index.html">Bernie Pope</a>
            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
              <span class="navbar-toggler-icon"></span>
            </button>
            <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
              <div class="navbar-nav">

                                  <a class="nav-item nav-link"href="index.html">Home</a>
                                  <a class="nav-item nav-link"href="biography.html">Biography</a>
                                  <a class="nav-item nav-link"href="publications.html">Publications</a>
                                  <a class="nav-item nav-link active"href="presentations.html">Presentations</a>
                                  <a class="nav-item nav-link"href="software.html">Software</a>
                                  <a class="nav-item nav-link"href="funding.html">Funding</a>
                                  <a class="nav-item nav-link"href="teaching.html">Teaching</a>
                                  <a class="nav-item nav-link"href="students.html">Students</a>
                                  <a class="nav-item nav-link"href="contact.html">Contact</a>
                
              </div>
            </div>
          </nav>
      

<h1 class="display-4 my-5">Presentations, Posters and Media</h1>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Using genomics for better detection and treatment of Lynch Syndrome, Eastern Gynaecological and Obstetrical Society, Annual Partners Dinner, 28 November, 2017</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Bernard Pope</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Oral presentation</dd>
              
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">Eastern Gynaecological and Obstetrical Society, Annual Partners Dinner</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Melbourne, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">28 November, 2017</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Translating genomics-driven bioinformatics into improved prevention and treatment of colorectal cancer, Colon CFR Steering Committee Meeting, 13 September, 2017</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Bernard Pope</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Oral presentation</dd>
              
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">Colon CFR Steering Committee Meeting</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Kauai, Hawaii, USA</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">13 September, 2017</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Colorectal cancer susceptibility genes: findings from whole genome, exome and targeted sequencing of early onset and multiple-case families, Familial Aspects of Cancer: Research and Practice, 1 September, 2017</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Bernard Pope, Khalid Mahmood, Mark Clendenning, Christophe Rosty, Marie Lorans, Harindra Jayasekara, Neil O’Callaghan, Susan Preston, Daniel J. Park, Fleur Hammet, Tu Nguyen-Dumont, Ashton Connor, Steven Gallinger, David Duggan, Graham Casey, Stephen N. Thibodeau, John L. Hopper, Melissa C. Southey, Aung K. Win, Finlay A. Macrae, Ingrid M. Winship, Mark A. Jenkins, Daniel D. Buchanan for the Australasian Colorectal Cancer Family Registry</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Oral presentation</dd>
              
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">Familial Aspects of Cancer: Research and Practice</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Kingscliff, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">1 September, 2017</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>The Genetics of Colonic Polyposis Study: identifying the genetic basis of unexplained colonic polyposis, Familial Aspects of Cancer: Research and Practice, 29 August - 1 September, 2017</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Sharelle Joseland, Mark Clendenning, Bernard J. Pope, Khalid Mahmood, Julie Arnold, Sonja Woodall, Marie Lorans, Neil O’Callaghan, Susan Preston, Chloe Wheatland, Kevin Sweet, Kara Semotiuk, Melissa C. Southey, Aung Ko Win, Mark A. Jenkins, Christophe Rosty, Finlay A. Macrae, Ingrid M. Winship, Susan Parry, Daniel D. Buchanan and FCCs</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Poster</dd>
              
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">Familial Aspects of Cancer: Research and Practice</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Kingscliff, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">29 August - 1 September, 2017</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Hi-Plex Origin for the high-throughput screening of disease genes, Familial Aspects of Cancer: Research and Practice, 29 August - 1 September, 2017</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Thomas Green, Fleur Hammet, Khalid Mahmood, Tu Nguyen-Dumont, Melissa Southey, Bernard Pope, Daniel Park</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Poster</dd>
              
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">Familial Aspects of Cancer: Research and Practice</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Kingscliff, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">29 August - 1 September, 2017</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>A registry of Australian Genomics bioinformatics pipelines incorporating a structured representation and interactive visualisation, Australian Genomics National Conference, 4 August 2017</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Mailie Gall, Bernard Pope, John Pearson, Natalie Thorne</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Poster</dd>
              
                              <dt class="col-sm-2">Media</dt>
		<dd class="col-sm-10">
                  		      <a href="assets/files/AGHA_2017_poster.pdf">PDF poster</a>
                                  </dd>
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">Australian Genomics National Conference</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Brisbane, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">4 August 2017</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Scientists encode GIF into DNA, ABC Radio National Interview, 13 July 2017</summary>
          <div class="card card-body my-3">
            <dl class="row">

              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Interview</dd>
              
                              <dt class="col-sm-2">Media</dt>
		<dd class="col-sm-10">
                  		      <a href="http://www.abc.net.au/radionational/programs/drive/dna-data-storage/8706456">Link to ABC Radio National interview recording</a>
                                  </dd>
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">ABC Radio National Interview</dd>
              
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">13 July 2017</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Characterisation of Mismatch Repair variants submitted to the International Mismatch Repair Consortium (IMRC), The International Society for Gastrointestinal Hereditary Tumours (InSiGHT), biennial meeting, 6 - 8 July, 2017</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Jeanette Reece, Daniel Buchanan, John-Paul Plazzer, Khalid Mahmood, Bernard Pope, Mark Clendenning, Grant Lee, Aung Win, Allyson Templeton, Robert Haile, Gabriela Moslein, Finlay Macrae, Mark Jenkins</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Poster</dd>
              
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">The International Society for Gastrointestinal Hereditary Tumours (InSiGHT), biennial meeting</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Florence, Italy</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">6 - 8 July, 2017</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Whole genome sequencing as a diagnostic tool for Lynch Syndrome, The International Society for Gastrointestinal Hereditary Tumours (InSiGHT), biennial meeting, 6 July, 2017</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">B Pope, M Clendenning, K Mahmood, C Rosty, R Walters, A Win, J Hopper, M Southey, F Macrae, I Winship, M Jenkins, A Spurdle, D Buchanan</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Oral presentation</dd>
              
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">The International Society for Gastrointestinal Hereditary Tumours (InSiGHT), biennial meeting</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Florence, Italy</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">6 July, 2017</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Fuzz testing with American Fuzzy Lop (AFL), VASET Group, The University of Melbourne, 18 August 2016</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Bernard Pope</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Oral presentation</dd>
              
                              <dt class="col-sm-2">Media</dt>
		<dd class="col-sm-10">
                  		      <a href="assets/files/afl_fuzz.pdf">PDF slides</a>
                                  </dd>
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">VASET Group, The University of Melbourne</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Melbourne, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">18 August 2016</dd>
              
                              <dt class="col-sm-2">Abstract </dt>
                <dd class="col-sm-10"><a href="http://lcamtuf.coredump.cx/afl/">American Fuzzy Lop (AFL)</a> is a fuzz testing tool that has helped detect an impressive set of bugs. It uses compile-time program instrumentation and simple run-time statistics to detect program inputs which trigger novel execution paths. An initial program input is supplied by the user, from which a stream of new cases are generated based on a carefully tuned "genetic" algorithm (author's terminology). In some cases AFL can synthesise complex input file structure from fairly distant starting points. A useful by-product of running AFL is a compact collection of synthetic test inputs which can be used for other testing regimes.  AFL's runtime overheads are very low, and, according to the author, real-word fuzzed programs can run at near-native speed.  In this talk I give an overview of AFL and attempt to explain some of its implementation details.
</dd>
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Clinical Genomics: a Computational Perspective, Linux Conference Australia, 1 February 2016</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Bernard Pope</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Oral presentation</dd>
              
                              <dt class="col-sm-2">Media</dt>
		<dd class="col-sm-10">
                  		    <ul>
		    		      <li><a href="assets/files/clinical_genomics_lca_2016.pdf">PDF slides</a></li>
                    		      <li><a href="https://www.youtube.com/watch?v=tGTVB-dqji4&feature=youtu.be">YouTube video</a></li>
                    		    </ul>
                                  </dd>
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">Linux Conference Australia</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Geelong, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">1 February 2016</dd>
              
                              <dt class="col-sm-2">Abstract </dt>
                <dd class="col-sm-10">Low cost highly accurate DNA sequencing is quickly becoming an option for clinical diagnosis and treatment. In this talk I discuss the computational aspects of Clinical Genomics, based on recent experiences with the Melbourne Genomics Health Alliance. This was an invited talk for the Open Source and Bioinformatics miniconf, as part of the Linux Conference in Geelong 2016.  
</dd>
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>OptimusPrime: Multiplex Primer Design Tool for Hi-Plex Sequencing, Australasian Genomic Technologies Association (AGTA), Annual Conference, 9 - 12 October, 2016</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Edmund Lau, Daniel Park, Bernard Pope</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Poster</dd>
              
                              <dt class="col-sm-2">Media</dt>
		<dd class="col-sm-10">
                  		      <a href="assets/files/hiplex_AGTA_poster_2016.pdf">PDF poster</a>
                                  </dd>
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">Australasian Genomic Technologies Association (AGTA), Annual Conference</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Auckland, New Zealand</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">9 - 12 October, 2016</dd>
              
                              <dt class="col-sm-2">Abstract </dt>
                <dd class="col-sm-10">The advent of Massively Parallel Sequencing (MPS) has dramatically reduced the cost and increased the throughput of DNA sequencing. A number of methods have been  developed that target specified genomic regions for MPS. However, these, variously, are compromised by issues of relative expense, accuracy, requirement for specialist  equipment and the cumbersome nature of protocols. In this work we present Hi-Plex, a novel MPS platform, and multiplex Polymerase Chain Reaction (PCR) primer design software to meet the challenges the platform presents.
</dd>
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Navigating the Research Data Life Cycle, The Australian Bioinformatics and Computational Biology Society (AB3ACBS) Conference, 31 October - 2 November, 2016</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">P Griffin, R Appels, D Bulach, K Dudley, G Keeble-Gagnere, A Pask, B Pope, U Roessner, T Seemann, D Bolser, J Khadake, S Lewis, S Orchard, S Tyagi, A Lonie, MV Schneider</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Poster</dd>
              
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">The Australian Bioinformatics and Computational Biology Society (AB3ACBS) Conference</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Brisbane, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">31 October - 2 November, 2016</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Whole-genome analysis of paediatric cancer from cases with a family cancer history: early insights from Victoria, Australia, American Society of Human Genetics (ASHG) annual meeting, 18 - 22 October, 2016</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">T Nguyen-Dumont, B Feng, H Tsimiklis, D Amor, E Algar, E Milne, L Spector, J Ng, A Lonie, D Park, B Pope, E Smibert, G Dite, M Southey, J Heath, J Hopper</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Poster</dd>
              
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">American Society of Human Genetics (ASHG) annual meeting</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Vancouver, Canada</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">18 - 22 October, 2016</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>RNASEL, a modifier of breast cancer risk in BRCA1 and BRCA2 carriers?, 6th International Symposium on Hereditary Breast and Ovarian Cancer, 10 – 13 May, 2016</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">T Nguyen-Dumont, A Roberge, D Park, M Kapuscinski, Z Teo, F Hammet, M Mahmoodi, H Tsimiklis, B Pope, A Lonie, K Mahmood, kConFab, ABCFR, J Hopper, G Giles, M Southey</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Poster</dd>
              
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">6th International Symposium on Hereditary Breast and Ovarian Cancer</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Montreal, Canada</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">10 – 13 May, 2016</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Another look at declarative debugging for Haskell, VASET Group, The University of Melbourne, 30 October 2015</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Bernard Pope</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Oral presentation</dd>
              
                              <dt class="col-sm-2">Media</dt>
		<dd class="col-sm-10">
                  		      <a href="assets/files/dd_vaset.pdf">PDF slides</a>
                                  </dd>
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">VASET Group, The University of Melbourne</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Melbourne, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">30 October 2015</dd>
              
                              <dt class="col-sm-2">Abstract </dt>
                <dd class="col-sm-10">Despite rising notoriety, Haskell still lacks effective debugging support. A breakpoint debugger exists, but, as expected, it is difficult to use in the presence of lazy evaluation and higher-order functions. We have previously considered declarative debugging as a more suitable approach for Haskell, but a number of technical challenges remain unsolved. In this talk I will revisit the main ideas behind declarative debugging and outline a few interesting areas of potential research in the context of non-strict purely functional languages, namely: generalising the structure of the debugging tree; interleaving debugging and program execution to reduce memory consumption.  
</dd>
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Mutation screening of RNASEL, a candidate breast cancer susceptibility gene identified via whole-exome sequencing, Familial Aspects of Cancer: Research and Practice, 25 - 28 August, 2015</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">A Roberge, T Nguyen-Dumont, D Park, M Kapuscinski, F Hammet, M Mahmoodi, H Tsimiklis, B Pope, A Lonie, K Mahmood, kConFab, ABCFS, J Hopper, M Southey</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Poster</dd>
              
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">Familial Aspects of Cancer: Research and Practice</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Kingscliff, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">25 - 28 August, 2015</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Lindenmayer Systems, Foundations of Computing Advanced Lecture, The University of Melbourne, 22 August, 2014</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Bernard Pope</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Lecture</dd>
              
                              <dt class="col-sm-2">Media</dt>
		<dd class="col-sm-10">
                  		      <a href="assets/files/lsystem.pdf">PDF slides</a>
                                  </dd>
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">Foundations of Computing Advanced Lecture, The University of Melbourne</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Melbourne, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">22 August, 2014</dd>
              
                              <dt class="col-sm-2">Abstract </dt>
                <dd class="col-sm-10">These slides introduce the idea of turtle graphics in Python and show how they can be used to implement Lindenmayer Systems (term rewriting languages with interesting interpretations as graphics).
</dd>
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Visualizing and analyzing heterogeneous DNA methylation patterns by massively parallel sequencing using methpat, Australian Bioinformatics Conference (ABiC), 4 June, 2014</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">N Wong, I Candiloro, B Pope, D Korbie, M Trau, S Wong, T Mikeska, X Zhang, M Pitman, A Dobrovic</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Poster</dd>
              
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">Australian Bioinformatics Conference (ABiC)</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Melbourne, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">4 June, 2014</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Introducing RedDog, a mapping-based genome comparison pipeline for bacterial high-throughput sequence data, Lorne Infection and Immunity Conference, 19 - 21 February, 2014</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">David Edwards, Bernard Pope, Kat Holt</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Poster</dd>
              
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">Lorne Infection and Immunity Conference</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Lorne, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">19 - 21 February, 2014</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Hi-Plex: a flexible, streamlined and cost-effective approach to targeted massively parallel sequencing, American Society of Human Genetics (ASHG) annual meeting, 18 - 22 October, 2014</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">T Nguyen-Dumont, B Pope, F Hammet, M Mahmoodi, H Tsimiklis, M Southey, D Park</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Poster</dd>
              
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">American Society of Human Genetics (ASHG) annual meeting</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">San Diego, USA</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">18 - 22 October, 2014</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Hi-Plex: a flexible, streamlined and cost-effective approach to targeted massively parallel sequencing, 10th Australasian Mutation Detection meeting, 1 - 4 September, 2014</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">T Nguyen-Dumont, B Pope, F Hammet, M Mahmoodi, H Tsimiklis, M Southey</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Poster</dd>
              
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">10th Australasian Mutation Detection meeting</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Daydream Island, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">1 - 4 September, 2014</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>ALLOCATE: sorting ovarian cancer patients into treatment arms based on molecular and genetic make up of their tumours, Lorne Cancer Conference, 13 - 15 February, 2014</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">O Kondrashova, S Lunke, L Melishkin, C Scott, B Tran, D Bowtell, O McNally, M Wakefield, C Sloggett, B Pope, C Anderson, G Taylor, P Waring</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Poster</dd>
              
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">Lorne Cancer Conference</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Lorne, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">13 - 15 February, 2014</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>How Python Works, Foundations of Computing Advanced Lecture, The University of Melbourne, 10 October, 2014</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Bernard Pope</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Lecture</dd>
              
                              <dt class="col-sm-2">Media</dt>
		<dd class="col-sm-10">
                  		      <a href="assets/files/how_python_works.pdf">PDF slides</a>
                                  </dd>
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">Foundations of Computing Advanced Lecture, The University of Melbourne</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Melbourne, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">10 October, 2014</dd>
              
                              <dt class="col-sm-2">Abstract </dt>
                <dd class="col-sm-10">An overview of how Python works (CPython specifically). Covers syntax analysis (lexing and parsing), compilation to bytecode and program exeuction. The slides conclude with an example bytecode execution of a simple Python program.
</dd>
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Computer games in Python, Foundations of Computing Advanced Lecture, The University of Melbourne, 19 September, 2014</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Bernard Pope</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Lecture</dd>
              
                              <dt class="col-sm-2">Media</dt>
		<dd class="col-sm-10">
                  		    <ul>
		    		      <li><a href="assets/files/games_in_python.pdf">PDF slides</a></li>
                    		      <li><a href="https://github.com/bjpop/asteroids">Code example on GitHub</a></li>
                    		    </ul>
                                  </dd>
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">Foundations of Computing Advanced Lecture, The University of Melbourne</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Melbourne, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">19 September, 2014</dd>
              
                              <dt class="col-sm-2">Abstract </dt>
                <dd class="col-sm-10">These slides show how to implement arcade games in Python using the [PyGame](www.pygame.org/) library.  To demonstrate the main ideas of game programming in Python I wrote a simple asteroids game which is available in the linked code repository.
</dd>
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Ray tracing in Python, Foundations of Computing Advanced Lecture, The University of Melbourne, 5 September, 2014</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Bernard Pope</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Lecture</dd>
              
                              <dt class="col-sm-2">Media</dt>
		<dd class="col-sm-10">
                  		    <ul>
		    		      <li><a href="assets/files/ray_tracing.pdf">PDF slides</a></li>
                    		      <li><a href="https://github.com/bjpop/pyray">Code example on GitHub</a></li>
                    		    </ul>
                                  </dd>
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">Foundations of Computing Advanced Lecture, The University of Melbourne</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Melbourne, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">5 September, 2014</dd>
              
                              <dt class="col-sm-2">Abstract </dt>
                <dd class="col-sm-10">These slides explain the central ideas behind the recursive ray-tracing algorithm and demonstrate how to implement it in Python. A basic Phong Illumination model is used in the slides and program.
</dd>
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Implementing Python in Haskell, twice, The Melbourne Python Users Group, 7 July, 2014</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Bernard Pope</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Oral presentation</dd>
              
                              <dt class="col-sm-2">Media</dt>
		<dd class="col-sm-10">
                  		    <ul>
		    		      <li><a href="assets/files/MPUG_python_haskell.pdf">PDF slides</a></li>
                    		      <li><a href="https://github.com/bjpop/blip">Code repository for Blip</a></li>
                    		      <li><a href="https://github.com/bjpop/berp">Code repository for Berp</a></li>
                    		    </ul>
                                  </dd>
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">The Melbourne Python Users Group</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Melbourne, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">7 July, 2014</dd>
              
                              <dt class="col-sm-2">Abstract </dt>
                <dd class="col-sm-10">Over the past couple of years I've developed an unusual hobby: implementing Python in Haskell.  The first iteration resulted in berp, a Python-to-Haskell translator. The second iteration resulted in blip, which compiles Python to bytecode, and is compatible with the standard Python implementation CPython. Note: I've given a similar talk to the Haskell User's Group (see below). In this version of the talk the focus is oriented to a Python audience.
</dd>
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Functional graphics in Scala, The Melbourne Scala Users Group, 26 May, 2014</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Bernard Pope</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Oral presentation</dd>
              
                              <dt class="col-sm-2">Media</dt>
		<dd class="col-sm-10">
                  		    <ul>
		    		      <li><a href="assets/files/fungraph_scala.pdf">PDF slides</a></li>
                    		      <li><a href="https://github.com/bjpop/scala-fungraph">Example code repository</a></li>
                    		    </ul>
                                  </dd>
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">The Melbourne Scala Users Group</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Melbourne, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">26 May, 2014</dd>
              
                              <dt class="col-sm-2">Abstract </dt>
                <dd class="col-sm-10">Computer graphics are typically represented using two-dimensional arrays of pixels. In this talk I will demonstrate an alternative, and perhaps surprising, representation based on functions. We assume that images are defined (infinitely) over the two-dimensional real coordinate space, which avoids the need to worry about boundary conditions or discretisation. I will also show how this idea can be extended to animations, by making them functions over time. In addition to showing some pretty pictures, I demonstrate that higher-order functions are a powerful tool for program abstraction.
</dd>
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Implementing Python in Haskell, twice, The Melbourne Haskell Users Group, 24 April, 2014</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Bernard Pope</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Oral presentation</dd>
              
                              <dt class="col-sm-2">Media</dt>
		<dd class="col-sm-10">
                  		    <ul>
		    		      <li><a href="assets/files/ImplementPythonInHaskell.pdf">PDF slides</a></li>
                    		      <li><a href="https://github.com/bjpop/blip">Code repository for Blip</a></li>
                    		      <li><a href="https://github.com/bjpop/berp">Code repository for Berp</a></li>
                    		    </ul>
                                  </dd>
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">The Melbourne Haskell Users Group</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Melbourne, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">24 April, 2014</dd>
              
                              <dt class="col-sm-2">Abstract </dt>
                <dd class="col-sm-10">Over the past couple of years I've developed an unusual hobby: implementing Python in Haskell.  The first iteration resulted in berp, a Python-to-Haskell translator. The second iteration resulted in blip, which compiles Python to bytecode, and is compatible with the standard Python implementation CPython.  In this talk I explain how I picked up this hobby, and where it might be going. I also talk about various Haskell features which underpin the two implementations.
</dd>
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>A single-tube high-plex PCR approach for targeted massively parallel sequencing applied to FFPE-tumour derived material, American Society of Human Genetics (ASHG) annual meeting, October 22 - 26, 2013</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">T Nguyen-Dumont, B Pope, F Hammet, M Southey, D Park</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Poster</dd>
              
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">American Society of Human Genetics (ASHG) annual meeting</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Boston, USA</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">October 22 - 26, 2013</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Hi-Plex: a PCR-based approach for simple, rapid and cost-effective massively parallel sequencing library building, Familial Aspects of Cancer: Research and Practice, 25 - 28 August, 2013</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">T Nguyen-Dumont, B Pope, F Hammet, M Southey, D Park</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Poster</dd>
              
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">Familial Aspects of Cancer: Research and Practice</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Cairns, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">25 - 28 August, 2013</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Rubra - flexible distributed pipelines, Bioinformatics Open Source Conference (BOSC), 19 July, 2013</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">C Sloggett, M Wakefield, G Philip, B Pope</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Poster</dd>
              
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">Bioinformatics Open Source Conference (BOSC)</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Berlin, Germany</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">19 July, 2013</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>FAVR (Filtering and Annotation of Variants that are Rare): methods to facilitate the analysis of new germline genetics variants from SOLiD and Illumina datasets, European Society of Human Genetics annual meeting, June 8 - 11, 2013</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">T Nguyen-Dumont, B Pope, F Odefrey, R Bell, S Tavtigian, A Lonie, M Southey, D Park</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Poster</dd>
              
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">European Society of Human Genetics annual meeting</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Paris, France</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">June 8 - 11, 2013</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Automating Pipelines for Variant Detection, Lorne Genome Conference, 17 - 19 February, 2013</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">C Sloggett, G Philip, B Pope</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Poster</dd>
              
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">Lorne Genome Conference</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Lorne, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">17 - 19 February, 2013</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Functional Graphics in Python, An advanced lecture in COMP10001 Foundations of Computing at The University of Melbourne, 2013, 2013</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Bernard Pope</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Lecture</dd>
              
                              <dt class="col-sm-2">Media</dt>
		<dd class="col-sm-10">
                  		      <a href="assets/files/functional_graphics_python.pdf">PDF slides</a>
                                  </dd>
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">An advanced lecture in COMP10001 Foundations of Computing at The University of Melbourne, 2013</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Melbourne, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">2013</dd>
              
                              <dt class="col-sm-2">Abstract </dt>
                <dd class="col-sm-10">The conventional way to represent computer graphics is with a two-dimensional array of pixels. In these slides we illustrate an alternative approach using functions from continuous coordinates to pixel values. Apart from being a neat idea, this representation provides great expressiveness for image transformations, and allows us to do impressive things with only a small amount of code. In these slides we use Python, but the same idea can be applied in any language with higher-order functions. 
</dd>
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>FAVR (Filtering and Annotation of Variants that are rare): methods to facilitate the analysis of rare germline genetic variants from SOLiD and Illumina datasets, Familial Aspects of Cancer: Research and Practice, 21 - 24 August, 2012</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">T Nguyen-Dumont, B Pope, F Odefrey, A Lonie, M Southey, D Park</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Poster</dd>
              
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">Familial Aspects of Cancer: Research and Practice</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Kingscliff, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">21 - 24 August, 2012</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Visions: Breast Cancer Gene Discovery, University of Melbourne Publicity, 21 - 24 August, 2012</summary>
          <div class="card card-body my-3">
            <dl class="row">

              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Video</dd>
              
                              <dt class="col-sm-2">Media</dt>
		<dd class="col-sm-10">
                  		      <a href="https://www.youtube.com/watch?v=3kGsTCUOQss">YouTube video</a>
                                  </dd>
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">University of Melbourne Publicity</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Melbourne, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">21 - 24 August, 2012</dd>
              
                              <dt class="col-sm-2">Abstract </dt>
                <dd class="col-sm-10">An international team of researchers, led by the University of Melbourne, has used new technology to fast track the discovery of a new breast cancer risk gene. The finding could assist in the discovery of other cancer risk genes. 
</dd>
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Early insights from whole-exome analysis of early-onset, multiple-case breast cancer pedigrees, Combined American Society of Human Genetics, International Congress of Human Genetics meetings, 11 - 15 October, 2011</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">T Nguyen-Dumont, D Park, F Odefrey, F Hammet, Z Teo, BCFR, BRCAX Consortium, D Goldgar D, S Tavtigian S, B Pope, A Lonie, M Southey</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Poster</dd>
              
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">Combined American Society of Human Genetics, International Congress of Human Genetics meetings</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Montreal, Canada</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">11 - 15 October, 2011</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Identification of new breast cancer predisposition genes via whole-exome sequencing, Familial Aspects of Cancer: Research and Practice, 23 - 26 August, 2011</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">M Southey, D Park D, F Lesueur, F Odefrey, T Nguyen-Dumont, F Hammet, D Neuhausen, L Baglietto, A Lonie, B Pope, O Sinilnikova, H Tsimiklis, MCCS, BCFR, kConFab, G Giles, J Hopper, S Tavtigian, D Goldgar</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Poster</dd>
              
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">Familial Aspects of Cancer: Research and Practice</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Kingscliff, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">23 - 26 August, 2011</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Precision Medicine: Dawn of Supercomputing in omics Research, 5th eResearch Australasia Conference (eResearch), 6 - 11 November, 2011</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">M Reumann, K Holt, M Inouye, T Stinear, B Goudey, G Abraham, Q Wang, F Shi, A Kowalczyk, A Pearce, A Isaac, B Pope, H Butzkueven, J Wagner, S Moore, M Downton, P Church, S Turner, J Field, M Southey, D Bowtell, D Schmidt, E Makalic, J Zobel, J Hopper, S Petrovski, T O’Brien</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Oral presentation</dd>
              
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">5th eResearch Australasia Conference (eResearch)</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Melbourne, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">6 - 11 November, 2011</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Open recursion and fixed points (in Scala), Melbourne Scala Users Group, 25 July 2011</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Bernard Pope</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Oral presentation</dd>
              
                              <dt class="col-sm-2">Media</dt>
		<dd class="col-sm-10">
                  		      <a href="assets/files/open_recursion.pdf">PDF slides</a>
                                  </dd>
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">Melbourne Scala Users Group</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Melbourne, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">25 July 2011</dd>
              
                              <dt class="col-sm-2">Abstract </dt>
                <dd class="col-sm-10">Classes in object oriented languages combine many different programming language features into a single abstraction. One of the key features is open recursion - the recursion on the self parameter which provides a late binding mechanism on method calls. In these slides we show how the same kind of extensibility can be achieved with ordinary higher order functions and fixed points (no classes). We use Scala to demonstrate the key ideas, but they can be easily translated to any language with higher-order functions.
</dd>
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>The road to dependent types, Melbourne Scala Users Group, 28 March, 2011</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Bernard Pope</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Oral presentation</dd>
              
                              <dt class="col-sm-2">Media</dt>
		<dd class="col-sm-10">
                  		      <a href="assets/files/dependent_types.pdf">PDF slides</a>
                                  </dd>
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">Melbourne Scala Users Group</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Melbourne, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">28 March, 2011</dd>
              
                              <dt class="col-sm-2">Abstract </dt>
                <dd class="col-sm-10">Barendregt's Lambda Cube provides a neat characterisation of various kinds of typed Lambda Calculi. In these slides I show how we can explore the axes of the Lambda Cube by studying families of languages indexed by (possibly other) languages. For example the Simply Typed Lambda Calculus supports terms indexed by terms, whereas the Polymorphic Lambda Calculus extends this with terms indexed by types. Taking this idea further we can also consider types indexed by types and finally arrive at types indexed by terms, which is what most people call Dependent Types.
</dd>
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Haskell bindings to MPI, The Melbourne Functional Programming Union, 30 July, 2010</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Bernard Pope</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Oral presentation</dd>
              
                              <dt class="col-sm-2">Media</dt>
		<dd class="col-sm-10">
                  		      <a href="assets/files/mpi_bindings.pdf">PDF slides</a>
                                  </dd>
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">The Melbourne Functional Programming Union</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Melbourne, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">30 July, 2010</dd>
              
                              <dt class="col-sm-2">Abstract </dt>
                <dd class="col-sm-10">MPI (the Message Passing Interface) is a popular communication protocol for distributed parallel programming, providing both point-to- point and collective communication operators. Traditionally MPI is used in high-performance computing applications written in imperative languages, such as C, C++ and Fortran. I've recently been working on a Haskell binding to MPI via C and the Foreign Function Interface (FFI).  One of the main challenges in writing such a binding is deciding how to map the low-level C API into idiomatic Haskell. In this talk I provide a brief overview of MPI, and then discuss the hows and whys of my current implementation effort. 
</dd>
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Berp - an implementation of Python 3 in Haskell, The Melbourne Functional Programming Union, 18 June, 2010</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Bernard Pope</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Oral presentation</dd>
              
                              <dt class="col-sm-2">Media</dt>
		<dd class="col-sm-10">
                  		      <a href="assets/files/Berp.pdf">PDF slides</a>
                                  </dd>
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">The Melbourne Functional Programming Union</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Melbourne, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">18 June, 2010</dd>
              
                              <dt class="col-sm-2">Abstract </dt>
                <dd class="col-sm-10">Berp is an implementation of Python 3. At its heart is a translator, which takes Python code as input and generates Haskell code as output. The Haskell code is fed into a Haskell compiler (GHC) for compilation to machine code or interpretation as byte code. 
</dd>
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Static Pattern Calculus, The Melbourne Functional Programming Union, 22 January, 2010</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Bernard Pope</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Oral presentation</dd>
              
                              <dt class="col-sm-2">Media</dt>
		<dd class="col-sm-10">
                  		      <a href="assets/files/static_pattern_calculus.pdf">PDF slides</a>
                                  </dd>
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">The Melbourne Functional Programming Union</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Melbourne, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">22 January, 2010</dd>
              
                              <dt class="col-sm-2">Abstract </dt>
                <dd class="col-sm-10">The Pattern Calculus extends the Lambda Calculus with first class patterns. Barry Jay recently published a book on the Pattern Calculus (which has been briefly discussed on this list), and I have been reading through it over the summer break. In this talk I give an overview of the term rewriting system which underpins the calculus. The term rewriting system is developed in four stages in the book. The first stage is the Lambda Calculus which is well known. The second stage is the Compound Calculus which is essentially a core version of LISP. The third stage is the Static Pattern Calculus which adds static patterns. The fourth stage is the Dynamic Pattern Calculus which adds dynamic patterns. This talk covers the static variant of the calculus only. 
</dd>
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Monads in Scala, The Melbourne Scala Users Group, 26 October, 2009</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Bernard Pope</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Oral presentation</dd>
              
                              <dt class="col-sm-2">Media</dt>
		<dd class="col-sm-10">
                  		      <a href="assets/files/scala_monads.pdf">PDF slides</a>
                                  </dd>
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">The Melbourne Scala Users Group</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Melbourne, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">26 October, 2009</dd>
              
                              <dt class="col-sm-2">Abstract </dt>
                <dd class="col-sm-10">Monads are a concept from Category Theory which has found many applications in both programming language theory and programming practice (particularly in functional languages such as Haskell). In this talk I discuss the way that monads underpin the for-comprehension notation used in Scala, and provide a simple term language evaluator as an example. I also discuss how the quest for modular denotational semantics in programming language theory led to the use of monads in programming practice. 
</dd>
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Parser combinators in Scala, The Melbourne Scala Users Group, 22 June, 2009</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Bernard Pope</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Oral presentation</dd>
              
                              <dt class="col-sm-2">Media</dt>
		<dd class="col-sm-10">
                  		      <a href="assets/files/scala_parser_combinators.pdf">PDF slides</a>
                                  </dd>
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">The Melbourne Scala Users Group</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Melbourne, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">22 June, 2009</dd>
              
                              <dt class="col-sm-2">Abstract </dt>
                <dd class="col-sm-10">Parser combinators provide an elegant way to specify parsing functions declaratively. Parsers written in this style have a structure which closely follows the EBNF grammar for the accepted language. This results in concise programs which can be readily checked against their formal specifications. Parsers are written entirely in the host language (in this case Scala), which avoids the need for additional languages and tools and their concomitant problems for software development. In this talk I will demonstrate Scala's parser combinator library by constructing a parser for (a non-trivial subset of) XML. If time permits, I will also discuss the key ideas which underpin the implementation of parser combinators, and their historical roots in functional programming. 
</dd>
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Applicative Functors, The Melbourne Functional Programming Union, 3 July, 2009</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Bernard Pope</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Oral presentation</dd>
              
                              <dt class="col-sm-2">Media</dt>
		<dd class="col-sm-10">
                  		      <a href="assets/files/applicative.pdf">PDF slides</a>
                                  </dd>
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">The Melbourne Functional Programming Union</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Melbourne, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">3 July, 2009</dd>
              
                              <dt class="col-sm-2">Abstract </dt>
                <dd class="col-sm-10">Applicative functors provide a programming abstraction which sits between functors and monads. They capture a common pattern of programming with "effects", which do not require the full expressiveness of monads. Applicative functors were introduced in the paper "Applicative Programming with Effects" (2008) by McBride and Paterson. 
</dd>
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Simple graph reduction with visualisation, The Melbourne Functional Programming Union, 12 May, 2009</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Bernard Pope</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Oral presentation</dd>
              
                              <dt class="col-sm-2">Media</dt>
		<dd class="col-sm-10">
                  		      <a href="assets/files/graphreduction.pdf">PDF slides</a>
                                  </dd>
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">The Melbourne Functional Programming Union</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Melbourne, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">12 May, 2009</dd>
              
                              <dt class="col-sm-2">Abstract </dt>
                <dd class="col-sm-10">In 2008, whilst teaching the fourth year functional programming subject, I wrote a little functional language called miniFP for demonstration purposes. Near the end of the course we covered graph reduction. I extended miniFP to compile to a simple graph reduction system, implemented in C. One of the more interesting and useful aspects of the system is that it can produce pictures of the graph at each step of reduction. This allows the user to visualise the graph reduction process. It is especially helpful for understanding how cyclic data structures can be produced by 'tying the knot', which is a difficult topic to explain on paper. In this talk I outline some of the interesting aspects of the implementation, and show how it can be used. 
</dd>
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Stack tracing in Haskell - an exploration of the design space, The Haskell Implementors Workshop, September 5, 2009</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Bernard Pope</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Oral presentation</dd>
              
                              <dt class="col-sm-2">Media</dt>
		<dd class="col-sm-10">
                  		    <ul>
		    		      <li><a href="assets/files/stack_tracing.pdf">PDF slides</a></li>
                    		      <li><a href="http://www.vimeo.com/6575114">Video from the Haskell Implementors Workshop</a></li>
                    		    </ul>
                                  </dd>
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">The Haskell Implementors Workshop</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Edinburgh, Scotland</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">September 5, 2009</dd>
              
                              <dt class="col-sm-2">Abstract </dt>
                <dd class="col-sm-10">The development of debugging tools for Haskell has tended to lag behind its imperative peers. The main technical reason for this is the difficulty of reconciling the operational behaviour of code under lazy evaluation with its static source description. Recently GHC has incorporated a breakpoint debugger into its byte-code interpreter. Experience with the debugger suggests that, while it is a useful addition to the programmer's tool-set, the problems associated with lazy evaluation are (unsurprisingly) still present. Users often find it difficult to understand the chain of causation that leads to a particular expression being evaluated. Normally the chain of causation is represented as a stack of procedure invocations, but this is difficult in Haskell because the context in which a function application is constructed can differ considerably with the context in which it is evaluated. In this talk I will address the question "How can stack tracing be done in Haskell?". The structure of the talk will consist of three main parts: 1) the desirable features of a stack trace; 2) the current state of the art; 3) implementation issues, focusing on GHC. Overall I hope to sketch out the design space and spur interest in possible future implementation efforts. 
</dd>
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Continuations, The Melbourne Functional Programming Union, 3 October, 2008</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Bernard Pope</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Oral presentation</dd>
              
                              <dt class="col-sm-2">Media</dt>
		<dd class="col-sm-10">
                  		      <a href="assets/files/continuations.pdf">PDF slides</a>
                                  </dd>
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">The Melbourne Functional Programming Union</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Melbourne, Australia</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">3 October, 2008</dd>
              
                              <dt class="col-sm-2">Abstract </dt>
                <dd class="col-sm-10">Continuations are a method of reifying the evaluation context of a term within some computation. Intuitively, the continuation describes what may become of the value of the term in the overall computation. In this sense, a continuation provides a concrete representation of program control, and allows it to be manipulated. This "purely functional" account of program control has many useful results, such as: the extension of denotational semantics from the lambda calculus to (constructs from) conventional imperative languages, such as jumps; techniques for compiling high-level languages to machine code; and first class control operators, such as exception handlers. In recent times, more refined versions of continuations have emerged, such as the delimited continuations (or subcontinuations), which reify only a part of the evaluation context of a term. This talk covers: the continuation passing style (CPS); the history of continuations; some interesting applications of continuations; the use of continuations in the "direct style" of programming via primitives such as call/cc of Scheme; and connections to logic. 
</dd>
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Reification in Haskell, Implementation of Functional Languages (IFL), 13th International Workshop, 24-26 September, 2001</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Bernard Pope, Lee Naish</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Oral presentation</dd>
              
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">Implementation of Functional Languages (IFL), 13th International Workshop</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Stockholm, Sweden</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">24-26 September, 2001</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none"> 
      <li>
        <details>
          <summary>Specialisation of Higher-Order Functions for Debugging, Workshop on Functional and Logic Programming, 13 - 15 September, 2001</summary>
          <div class="card card-body my-3">
            <dl class="row">

                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Bernard Pope, Lee Naish</dd>
              
                              <dt class="col-sm-2">Type</dt>
                <dd class="col-sm-10">Oral presentation</dd>
              
              
                              <dt class="col-sm-2">Event </dt>
                <dd class="col-sm-10">Workshop on Functional and Logic Programming</dd>
              
                              <dt class="col-sm-2">Location</dt>
                <dd class="col-sm-10">Kiel, Germany</dd>
              
                              <dt class="col-sm-2">Date</dt>
                <dd class="col-sm-10">13 - 15 September, 2001</dd>
              
              
            </dl>
          </div>
        </details>
      </li>
    </ul>

  
      </div> <!-- container -->

      <!-- jQuery first, then Popper.js, then Bootstrap JS -->
      <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.3/umd/popper.min.js" integrity="sha384-vFJXuSJphROIrBnz7yo7oB41mKfc8JzQZiCq4NCceLEaO4IHwicKwpJf9c9IpFgh" crossorigin="anonymous"></script>
      <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/js/bootstrap.min.js" integrity="sha384-alpBpkh1PFOepccYVYDB4do5UnbKysX5WZXm3XxPqe5iKTfUKjNkCk9SaVuEZflJ" crossorigin="anonymous"></script>
      <script src="assets/js/site.js"></script>
      <script src="assets/js/details-element-polyfill.js"></script>
  </body>
</html>
