<!doctype html>
<html lang="en">
  <head>
     
    <title>Bernie Pope: Publications</title>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css"
          integrity="sha384-PsH8R72JQ3SOdhVi3uxftmaW6Vc51MKb0q5P2rRUpPvrszuE4W1povHYgTpBfshb" crossorigin="anonymous">
    <link rel="stylesheet" href="assets/css/website.css">
      </head>

  <body>
      <div class="container"> 
      <!-- <div class="container-fluid"">  -->
         

      <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
            <a class="navbar-brand" href="index.html">Bernie Pope</a>
            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
              <span class="navbar-toggler-icon"></span>
            </button>
            <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
              <div class="navbar-nav">

                                  <a class="nav-item nav-link"href="index.html">Home</a>
                                  <a class="nav-item nav-link"href="biography.html">Biography</a>
                                  <a class="nav-item nav-link active"href="publications.html">Publications</a>
                                  <a class="nav-item nav-link"href="presentations.html">Presentations</a>
                                  <a class="nav-item nav-link"href="software.html">Software</a>
                                  <a class="nav-item nav-link"href="funding.html">Funding</a>
                                  <a class="nav-item nav-link"href="teaching.html">Teaching</a>
                                  <a class="nav-item nav-link"href="students.html">Students</a>
                                  <a class="nav-item nav-link"href="notes.html">Notes</a>
                                  <a class="nav-item nav-link"href="fun.html">Fun</a>
                                  <a class="nav-item nav-link"href="contact.html">Contact</a>
                
              </div>
            </div>
          </nav>
      

<h1 class="display-4 my-5">Publications</h1>

<div class="my-5"><p>Orcid: <a href="http://orcid.org/0000-0002-4840-1095">0000-0002-4840-1095</a></p></div>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Genomic evolution shapes prostate cancer disease type</a>,  Cell Genomics,  2024.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Dan J Woodcock, Atef Sahli, Ruxandra Teslo, Vinayak Bhandari, Andreas J Gruber, Aleksandra Ziubroniewicz, Gunes Gundem, Yaobo Xu, Adam Butler, Ezequiel Anokian, Bernard J Pope, Chol-Hee Jung, Maxime Tarabichi, Stefan C Dentro, J Henry R Farmery, CRUK ICGC Prostate Group, Peter Van Loo, Anne Y Warren, Vincent Gnanapragasam, Freddie C Hamdy, G Steven Bova, Christopher S Foster, David E Neal, Yong-Jie Lu, Zsofia Kote-Jarai, Michael Fraser, Robert G Bristow, Paul C Boutros, Anthony J Costello, Niall M Corcoran, Christopher M Hovens, Charlie E Massie, Andy G Lynch, Daniel S Brewer, Rosalind A Eeles, Colin S Cooper, David C Wedge</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">Cell Genomics</dd>
            
            
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2024</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1016/j.xgen.2024.100511</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">38428419</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="https://doi.org/10.1016/j.xgen.2024.100511">https://doi.org/10.1016/j.xgen.2024.100511</a></dd>
            
                          <dt class="col-sm-2">Keywords</dt>
              <dd class="col-sm-10">AR binding; cancer evolution; evotype model; evotypes; ordering; prostate cancer</dd>
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">The development of cancer is an evolutionary process involving the sequential acquisition of genetic alterations that disrupt normal biological processes, enabling tumor cells to rapidly proliferate and eventually invade and metastasize to other tissues. We investigated the genomic evolution of prostate cancer through the application of three separate classification methods, each designed to investigate a different aspect of tumor evolution. Integrating the results revealed the existence of two distinct types of prostate cancer that arise from divergent evolutionary trajectories, designated as the Canonical and Alternative evolutionary disease types. We therefore propose the evotype model for prostate cancer evolution wherein Alternative-evotype tumors diverge from those of the Canonical-evotype through the stochastic accumulation of genetic alterations associated with disruptions to androgen receptor DNA binding. Our model unifies many previous molecular observations, providing a powerful new framework to investigate prostate cancer disease progression. 
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>A Family-Based Study of Inherited Genetic Risk in Lipedema</a>,  Lymphatic Research and Biology,  2024.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Steven Morgan, Isabella Reid, Charlotte Bendon, Musarat Ishaq, Ramin Shayan, Bernard Pope, Daniel Park, Tara Karnezis</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">Lymphatic Research and Biology</dd>
            
            
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2024</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1089/lrb.2023.0065</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">38407896</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="https://doi.org/10.1089/lrb.2023.0065">https://doi.org/10.1089/lrb.2023.0065</a></dd>
            
                          <dt class="col-sm-2">Keywords</dt>
              <dd class="col-sm-10">family study; genetic risk; lipedema</dd>
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">Background: Lipedema is a progressive condition involving excessive deposition of subcutaneous adipose tissue, predominantly in the lower limbs, which severely compromises quality of life. Despite the impact of lipedema, its molecular and genetic bases are poorly understood, making diagnosis and treatment difficult. Historical evaluation of individuals with lipedema indicates a positive family history in 60%-80% of cases; however, genetic investigation of larger family cohorts is required. Here, we report the largest family-based sequencing study to date, aimed at identifying genetic changes that contribute to lipedema. Methods and Results: DNA samples from 31 individuals from 9 lipedema families were analyzed to reveal genetic variants predicted to alter protein function, yielding candidate variants in 469 genes. We did not identify any individual genes that contained likely disease-causing variants across all participating families. However, gene ontology analysis highlighted vasopressin receptor activity, microfibril binding, and patched binding as statistically significantly overrepresented categories for the set of candidate variants. Conclusions: Our study suggests that lipedema is not caused by a single exomic genetic factor, providing support for the hypothesis of genetic heterogeneity in the etiology of lipedema. As the largest study of its kind in the lipedema field, the results advance our understanding of the disease and provide a roadmap for future research aimed at improving the lives of those affected by lipedema.
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Ultrasensitive Detection of Circulating Tumour DNA enriches for Patients with a Greater Risk of Recurrence of Clinically Localised Prostate Cancer</a>,  European Urology,  2024.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Bernard Pope, Gahee Park, Edmund Lau, Jelena Belic, Radoslaw Lach, Anne George, Patrick McCoy, Anne Nguyen, Corrina Grima, Bethany Campbell, Chol-hee Jung, Emma-Jane Ditter, Hui Zhao, The Pan Prostate Cancer Group (PPCG), David C. Wedge, Daniel S. Brewer, Andy G. Lynch, Harveer Dev, Vincent J. Gnanpragasam, Nitzan Rosenfeld, Christopher M. Hovens, Niall M. Corcoran, Charles E. Massie</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">European Urology</dd>
            
            
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2024</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1016/j.eururo.2024.01.002</dd>
            
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="https://doi.org/10.1016/j.eururo.2024.01.002">https://doi.org/10.1016/j.eururo.2024.01.002</a></dd>
            
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">Circulating tumour DNA (ctDNA) has clinical applications as a “liquid biopsy” owing to its short half-life, noninvasive collection modalities, and propensity for sampling across populations of tumour cells. While successful ctDNA detection has been demonstrated in metastatic prostate cancer, localised disease yields low levels of ctDNA, making detection difficult using conventional methods.

In this study, we assessed ctDNA detection in localised prostate cancer using the high-sensitivity integration of variant reads (INVAR) method and tested the hypothesis that the presence of ctDNA is associated with high-risk disease. Tumour information was used to create marker panels of patient-specific mutations that were used to identify ctDNA molecules in patient-matched plasma samples. After initial background error calculations and filtering steps, the detected ctDNA fraction was estimated as a global integrated mutant allele fraction (IMAF) using the background-subtracted mean allele fraction across the patient-specific loci in each sample. Patient-specific scores were compared to the threshold value for classification, which was calculated using data from control samples and set to 95% specificity.
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Inherited BRCA1 and RNF43 pathogenic variants in a familial colorectal cancer type X family</a>,  Familial Cancer,  2023.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">James M. Chan, Mark Clendenning, Sharelle Joseland, Peter Georgeson, Khalid Mahmood, Jihoon E. Joo, Romy Walker, Julia Como, Susan Preston, Shuyi Marci Chai, Yen Lin Chu, Aaron L. Meyers, Bernard J. Pope, David Duggan, J. Lynn Fink, Finlay A. Macrae, Christophe Rosty, Ingrid M. Winship, Mark A. Jenkins & Daniel D. Buchanan</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">Familial Cancer</dd>
            
            
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2023</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1007/s10689-023-00351-2</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">38063999</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="https://doi.org/10.1007/s10689-023-00351-2">https://doi.org/10.1007/s10689-023-00351-2</a></dd>
            
                          <dt class="col-sm-2">Keywords</dt>
              <dd class="col-sm-10">BRCA1; Colorectal cancer; Digenic inheritance; FCCTX; Germline pathogenic variant; RNF43; Serrated polyposis syndrome</dd>
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">Genetic susceptibility to familial colorectal cancer (CRC), including for individuals classified as Familial Colorectal Cancer Type X (FCCTX), remains poorly understood. We describe a multi-generation CRC-affected family segregating pathogenic variants in both BRCA1, a gene associated with breast and ovarian cancer and RNF43, a gene associated with Serrated Polyposis Syndrome (SPS). A single family out of 105 families meeting the criteria for FCCTX (Amsterdam I family history criteria with mismatch repair (MMR)-proficient CRCs) recruited to the Australasian Colorectal Cancer Family Registry (ACCFR; 1998–2008) that underwent whole exome sequencing (WES), was selected for further testing. CRC and polyp tissue from four carriers were molecularly characterized including a single CRC that underwent WES to determine tumor mutational signatures and loss of heterozygosity (LOH) events. Ten carriers of a germline pathogenic variant BRCA1:c.2681_2682delAA p.Lys894ThrfsTer8 and eight carriers of a germline pathogenic variant RNF43:c.988 C > T p.Arg330Ter were identified in this family. Seven members carried both variants, four of which developed CRC. A single carrier of the RNF43 variant met the 2019 World Health Organization (WHO2019) criteria for SPS, developing a BRAF p.V600 wildtype CRC. Loss of the wildtype allele for both BRCA1 and RNF43 variants was observed in three CRC tumors while a LOH event across chromosome 17q encompassing both genes was observed in a CRC. Tumor mutational signature analysis identified the homologous recombination deficiency (HRD)-associated COSMIC signatures SBS3 and ID6 in a CRC for a carrier of both variants. Our findings show digenic inheritance of pathogenic variants in BRCA1 and RNF43 segregating with CRC in a FCCTX family. LOH and evidence of BRCA1-associated HRD supports the importance of both these tumor suppressor genes in CRC tumorigenesis.
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>DNA Mismatch Repair Gene Variant Classification: Evaluating the Utility of Somatic Mutations and Mismatch Repair Deficient Colonic Crypts and Endometrial Glands</a>,  Cancers,  2023.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Romy Walker, Khalid Mahmood, Julia Como, Mark Clendenning, Jihoon E. Joo, Peter Georgeson, Sharelle Joseland, Susan G. Preston, Bernard J. Pope, James M. Chan, Rachel Austin, Jasmina Bojadzieva, Ainsley Campbell, Emma Edwards, Margaret Gleeson, Annabel Goodwin, Marion T. Harris, Emilia Ip, Judy Kirk, Julia Mansour, Helen Mar Fan, Cassandra Nichols, Nicholas Pachter, Abiramy Ragunathan, Allan Spigelman, Rachel Susman, Michael Christie, Mark A. Jenkins, Rish K. Pai, Christophe Rosty, Finlay A. Macrae, Ingrid M. Winship, Daniel D. Buchanan</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">Cancers</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">15</dd>
            
                          <dt class="col-sm-2">Issue</dt>
              <dd class="col-sm-10">20</dd>
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2023</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.3390/cancers15204925</dd>
            
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="https://doi.org/10.3390/cancers15204925">https://doi.org/10.3390/cancers15204925</a></dd>
            
                          <dt class="col-sm-2">Keywords</dt>
              <dd class="col-sm-10">Lynch syndrome; DNA mismatch repair gene variant classification; DNA mismatch repair deficient crypts/glands; colorectal cancer; endometrial cancer; variant of uncertain significance; DNA mismatch repair gene somatic mutations</dd>
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">Germline pathogenic variants in the DNA mismatch repair (MMR) genes (Lynch syndrome) predispose to colorectal (CRC) and endometrial (EC) cancer. Lynch syndrome specific tumor features were evaluated for their ability to support the ACMG/InSiGHT framework in classifying variants of uncertain clinical significance (VUS) in the MMR genes. Twenty-eight CRC or EC tumors from 25 VUS carriers (6xMLH1, 9xMSH2, 6xMSH6, 4xPMS2), underwent targeted tumor sequencing for the presence of microsatellite instability/MMR-deficiency (MSI-H/dMMR) status and identification of a somatic MMR mutation (second hit). Immunohistochemical testing for the presence of dMMR crypts/glands in normal tissue was also performed. The ACMG/InSiGHT framework reclassified 7/25 (28%) VUS to likely pathogenic (LP), three (12%) to benign/likely benign, and 15 (60%) VUS remained unchanged. For the seven re-classified LP variants comprising nine tumors, tumor sequencing confirmed MSI-H/dMMR (8/9, 88.9%) and a second hit (7/9, 77.8%). Of these LP reclassified variants where normal tissue was available, the presence of a dMMR crypt/gland was found in 2/4 (50%). Furthermore, a dMMR endometrial gland in a carrier of an MSH2 exon 1-6 duplication provides further support for an upgrade of this VUS to LP. Our study confirmed that identifying these Lynch syndrome features can improve MMR variant classification, enabling optimal clinical care.
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Abstract PR012: Ultra-sensitive detection of circulating tumour DNA enriches for patients with higher risk disease in clinically localised prostate cancer. Proceedings of the AACR Special Conference: Advances in Prostate Cancer Research; 2023 Mar 15-18; Denver, Colorado</a>,  Cancer Research,  2023.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Bernard J. Pope, Gahee Park, Edmund Lau, Jelena Belic, Radoslaw Lach, Anne George, Patrick McCoy, Anne Nguyen, Corrina Grima, Chol-hee Jung, Emma-Jane Ditter, Hui Zhao, David Wedge, Rosalind A Eeles, Daniel Brewer, Andy G. Lynch, Harveer Dev, Christopher M Hovens, Vincent J. Gnanpragasam, Nitzan Rosenfeld, Niall M. Corcoran, Charles E. Massie</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">Cancer Research</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">83</dd>
            
                          <dt class="col-sm-2">Issue</dt>
              <dd class="col-sm-10">11_Supplement</dd>
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2023</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1158/1538-7445.PRCA2023-PR012</dd>
            
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="https://doi.org/10.1158/1538-7445.PRCA2023-PR012">https://doi.org/10.1158/1538-7445.PRCA2023-PR012</a></dd>
            
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">Purpose Circulating tumour DNA (ctDNA) analysis has demonstrated utility for diagnostic and prognostic applications in many cancer types, including metastatic prostate cancer. However, localised prostate cancer yields relatively low levels of ctDNA and therefore it has been difficult to detect in this context using conventional methods. We aimed to assess the limits of detection of ctDNA in localised prostate cancer by leveraging thousands of patient-specific mutations per case with the high-sensitivity INVAR method, and to test the hypothesis that ctDNA detection is associated with high risk disease. Experimental Procedures A total of 128 individuals with clinically localised prostate cancer at the time of sample collection were selected from cohorts in Australia (n=48) and the UK (n=80). Additionally, 27 healthy individuals without prostate cancer were included as negative controls. Primary tumour tissue and matched bloods from all cases were whole genome sequenced (WGS) and somatic variants were called using pipelines from the Pan Prostate Cancer Group. Plasma cell-free DNA (cfDNA) samples from cases and controls were profiled using custom targeted sequencing panels, with saturating coverage of patient-specific mutations identified by WGS. We assessed ctDNA detection in cases using the highly sensitive INVAR pipeline, that leverages consensus read sequencing alignments, background error modelling and integration of signals across thousands of patient-specific variants. Biochemical recurrence and metastasis-free survival curves were used to assess the relationship between ctDNA detection and disease progression. Results We analysed pre-treatment blood plasma ctDNA in a large cohort of localised prostate cancer patients, using error-suppressed targeted sequencing of over 280k patient-specific mutations. To comprehensively assess ctDNA mutation analysis in this context, we combined signals across the maximum number of genome-wide patient specific mutations and leveraged an established analysis pipeline (INVAR) that corrects for background error rates and calculates a global integrated mutant allele fraction. ctDNA was detected in 9.3% of localised prostate cancer patients. In cases where ctDNA was detected we found significant associations with biochemical recurrence (p=0.01) and shorter metastasis-free survival (p < 0.0001). Conclusions Our study provides clear insights into the required analytical sensitivity and potential utility of ctDNA mutation analysis in localised prostate cancer. We found that mutation-based ctDNA detection rates were low in localised prostate cancer (<10% cases), but in ctDNA positive cases there was a significant association with relapse after surgical intervention alone. This raises the potential for including ctDNA detection as an additional tool for patient stratification in future neo/adjuvant treatment trials aiming to assess the impact of treatment escalation in men at high risk of relapse with current standard of care treatment alone.
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Workflow for SNP genotyping using the Hi-Plex method</a>,  protocols.io,  2023.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Anne-Laure Besnard, Daniel J. Park, Bernard J. Pope, Fleur Hammet, Sophie Michon-Coudouel, Marine Biget, Stacy A. Krueger-Hadfield, Stéphane Mauger, Eric J. Petit</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">protocols.io</dd>
            
            
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2023</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.17504/protocols.io.8epv5jnnnl1b/v1</dd>
            
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="https://doi.org/10.17504/protocols.io.8epv5jnnnl1b/v1">https://doi.org/10.17504/protocols.io.8epv5jnnnl1b/v1</a></dd>
            
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">Many research questions in ecology and evolution require balancing sampling strategies between their spatial (how many populations? on which geographical, environmental gradients?), temporal (diachronic approaches), and genomic (how many and which loci?) dimensions. High-throughput molecular biology protocols often offer very good genomic coverage, but this is often only achievable at the expense of other sampling dimensions. This has led to the development of targeted genotyping strategies for SNP locus sets, in addition to whole or reduced genome sequencing strategies. We here present an adaptation of a protocol developed by the University of Melbourne for genotyping rare variants in human oncology to non-model species for use in ecology and evolution. Hi-Plex is an amplicon sequencing technique in which all loci are co-amplified in a multiplex reaction before Illumina or Ion-Torrent sequencing (we used Illumina). Intermediate steps include dual indexing of individual samples used for demultiplexing.
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>A tumor focused approach to resolving the etiology of DNA mismatch repair deficient tumors classified as suspected Lynch syndrome</a>,  Journal of Translational Medicine,  2023.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Romy Walker, Khalid Mahmood, Jihoon E. Joo, Mark Clendenning, Peter Georgeson, Julia Como, Sharelle Joseland, Susan G. Preston, Yoland Antill, Rachel Austin, Alex Boussioutas, Michelle Bowman, Jo Burke, Ainsley Campbell, Simin Daneshvar, Emma Edwards, Margaret Gleeson, Annabel Goodwin, Marion T. Harris, Alex Henderson, Megan Higgins, John L. Hopper, Ryan A. Hutchinson, Emilia Ip, Joanne Isbister, Kais Kasem, Helen Marfan, Di Milnes, Annabelle Ng, Cassandra Nichols, Shona O’Connell, Nicholas Pachter, Bernard J. Pope, Nicola Poplawski, Abiramy Ragunathan, Courtney Smyth, Allan Spigelman, Kirsty Storey, Rachel Susman, Jessica A. Taylor, Linda Warwick, Mathilda Wilding, Rachel Williams, Aung K. Win, Michael D. Walsh, Finlay A. Macrae, Mark A. Jenkins, Christophe Rosty, Ingrid M. Winship, Daniel D. Buchanan, Family Cancer Clinics of Australia</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">Journal of Translational Medicine</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">21</dd>
            
                          <dt class="col-sm-2">Issue</dt>
              <dd class="col-sm-10">282</dd>
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2023</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1186/s12967-023-04143-1</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">37101184</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="https://doi.org/10.1186/s12967-023-04143-1">https://doi.org/10.1186/s12967-023-04143-1</a></dd>
            
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">Routine screening of tumors for DNA mismatch repair (MMR) deficiency (dMMR) in colorectal (CRC), endometrial (EC) and sebaceous skin (SST) tumors leads to a significant proportion of unresolved cases classified as suspected Lynch syndrome (SLS). SLS cases (n = 135) were recruited from Family Cancer Clinics across Australia and New Zealand. Targeted panel sequencing was performed on tumor (n = 137; 80×CRCs, 33×ECs and 24xSSTs) and matched blood-derived DNA to assess for microsatellite instability status, tumor mutation burden, COSMIC tumor mutational signatures and to identify germline and somatic MMR gene variants. MMR immunohistochemistry (IHC) and MLH1 promoter methylation were repeated. In total, 86.9% of the 137 SLS tumors could be resolved into established subtypes. For 22.6% of these resolved SLS cases, primary MLH1 epimutations (2.2%) as well as previously undetected germline MMR pathogenic variants (1.5%), tumor MLH1 methylation (13.1%) or false positive dMMR IHC (5.8%) results were identified. Double somatic MMR gene mutations were the major cause of dMMR identified across each tumor type (73.9% of resolved cases, 64.2% overall, 70% of CRC, 45.5% of ECs and 70.8% of SSTs). The unresolved SLS tumors (13.1%) comprised tumors with only a single somatic (7.3%) or no somatic (5.8%) MMR gene mutations. A tumor-focused testing approach reclassified 86.9% of SLS into Lynch syndrome, sporadic dMMR or MMR-proficient cases. These findings support the incorporation of tumor sequencing and alternate MLH1 methylation assays into clinical diagnostics to reduce the number of SLS patients and provide more appropriate surveillance and screening recommendations.
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Somatic mutation landscape in a cohort of meningiomas that have undergone grade progression</a>,  BMC Cancer,  2023.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Sarah A Cain, Bernard Pope, Stefano Mangiola, Theo Mantamadiotis, Katharine J Drummond</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">BMC Cancer</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">23</dd>
            
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2023</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1186/s12885-023-10624-9</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">36882706</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="https://doi.org/10.1186/s12885-023-10624-9">https://doi.org/10.1186/s12885-023-10624-9</a></dd>
            
                          <dt class="col-sm-2">Keywords</dt>
              <dd class="col-sm-10">Meningioma; Next generation sequencing; Anaplastic; Atypicall; Malignant</dd>
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10"><p>Background<p></p><p>A subset of meningiomas progress in histopathological grade but drivers of progression are poorly understood. We aimed to identify somatic mutations and copy number alterations (CNAs) associated with grade progression in a unique matched tumour dataset.</p><p>Methods</p><p> Utilising a prospective database, we identified 10 patients with meningiomas that had undergone grade progression and for whom matched pre- and post-progression tissue (n = 50 samples) was available for targeted next-generation sequencing.</p> <p>Results</p><p>Mutations in NF2 were identified in 4/10 patients, of these 94% were non-skull base tumours. In one patient, three different NF2 mutations were identified in four tumours. NF2 mutated tumours showed large-scale CNAs, with highly recurrent losses in 1p, 10, 22q, and frequent CNAs on chromosomes 2, 3 and 4. There was a correlation between grade and CNAs in two patients. Two patients with tumours without detected NF2 mutations showed a combination of loss and high gain on chromosome 17q. Mutations in SETD2, TP53, TERT promoter and NF2 were not uniform across recurrent tumours, however did not correspond with the onset of grade progression.</p><p>Conclusion</p><p>Meningiomas that progress in grade generally have a mutational profile already detectable in the pre-progressed tumour, suggesting an aggressive phenotype. CNA profiling shows frequent alterations in NF2 mutated tumours compared to non NF2 mutated tumours. The pattern of CNAs may be associated with grade progression in a subset of cases.</p>
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>A polygenic two-hit hypothesis for prostate cancer</a>,  Journal of the National Cancer Institute,  2023.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Kathleen E Houlahan, Julie Livingstone, Natalie S Fox, Natalie Kurganovs, Helen Zhu, Jocelyn Sietsma Penington, Chol-Hee Jung, Takafumi N Yamaguchi, Lawrence E Heisler, Richard Jovelin, Anthony J Costello, Bernard J Pope, Amar U Kishan, Niall M Corcoran, Robert G Bristow, Sebastian M Waszak, Joachim Weischenfeldt, Housheng H He, Rayjean J Hung, Christopher M Hovens, Paul C Boutros</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">Journal of the National Cancer Institute</dd>
            
            
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2023</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1093/jnci/djad001</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">36610996</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="https://doi.org/10.1093/jnci/djad001">https://doi.org/10.1093/jnci/djad001</a></dd>
            
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">Prostate cancer is one of the most heritable cancers. Hundreds of germline polymorphisms have been linked to prostate cancer diagnosis and prognosis. Polygenic risk scores can predict genetic risk of a prostate cancer diagnosis. While these scores inform on the probability of developing a tumor, it remains unknown how germline risk influences the tumor molecular evolution. We cultivated a cohort of 1,250 localized European-descent patients with germline and somatic DNA profiling. Men of European descent with higher genetic risk were diagnosed earlier, had less genomic instability, and fewer driver genes mutated. Higher genetic risk was associated with better outcome. These data imply a polygenic “two-hit” model where germline risk reduces the number of somatic alterations required for tumorigenesis. These findings support further clinical studies of PRS as inexpensive and minimally invasive adjuncts to standard risk stratification. Further studies are required to interrogate generalizability to more ancestrally and clinically diverse populations.
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Evaluating multiple next-generation sequencing derived tumor features to accurately predict DNA mismatch repair status</a>,  The Journal of Molecular Diagnostics,  2022.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Romy Walker, Peter Georgeson, Khalid Mahmood, Jihoon Joo, Enes Makalic, Mark Clendenning, Julia Como, Susan Preston, Sharelle Joseland, Bernard Pope, Ryan Hutchinson, Kais Kasem, Michael Walsh, Finlay Macrae, Aung Win, John Hopper, Dmitri Mouradov, Peter Gibbs, Oliver Sieber, Dylan  O'Sullivan, Darren Brenner, Steven Gallinger, Mark Jenkins, Christophe Rosty, Ingrid Winship, Daniel Buchanan</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">The Journal of Molecular Diagnostics</dd>
            
            
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2022</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1016/j.jmoldx.2022.10.003</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">36396080</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="https://doi.org/10.1016/j.jmoldx.2022.10.003">https://doi.org/10.1016/j.jmoldx.2022.10.003</a></dd>
            
                          <dt class="col-sm-2">Keywords</dt>
              <dd class="col-sm-10">Colorectal cancer; DNA mismatch repair deficiency; endometrial cancer; Lynch syndrome; microsatellite instability; MLH1 promoter methylation; sebaceous skin tumor; tumor mutation burden; tumor mutational signatures</dd>
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">Identifying tumor DNA mismatch repair deficiency (dMMR) is important for precision medicine. Tumor features, individually and in combination, derived from whole-exome sequenced (WES) colorectal cancers (CRCs) and panel sequenced CRCs, endometrial cancers (ECs) and sebaceous skin tumors (SSTs) were assessed for their accuracy in detecting dMMR. CRCs (n=300) with WES, where MMR status was determined by immunohistochemistry, were assessed for microsatellite instability (MSMuTect, MANTIS, MSIseq, MSISensor), COSMIC tumor mutational signatures (TMS) and somatic mutation counts. A 10-fold cross-validation approach (100 repeats) evaluated the dMMR prediction accuracy for 1) individual features, 2) Lasso statistical model and 3) an additive feature combination approach. Panel sequenced tumors (29 CRCs, 22 ECs, 20 SSTs) were assessed for the top performing dMMR predicting features/models using these three approaches. For WES CRCs, 10 features provided >80% dMMR prediction accuracy, with MSMuTect, MSIseq, and MANTIS achieving ≥99% accuracy. The Lasso model achieved 98.3%. The additive feature approach with ≥3/6 of MSMuTect, MANTIS, MSIseq, MSISensor, INDEL count or TMS ID2+ID7 achieved 99.7% accuracy. For the panel sequenced tumors, the additive feature combination approach of ≥3/6 achieved accuracies of 100%, 95.5% and 100%, for CRCs, ECs, and SSTs, respectively. The microsatellite instability calling tools performed well in WES CRCs, however, an approach combining tumor features may improve dMMR prediction in both WES and panel sequenced data across tissue types.
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>VIVID: A Web Application for Variant Interpretation and Visualization in Multi-dimensional Analyses</a>,  Molecular Biology and Evolution,  2022.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Swapnil Tichkule, Yoochan Myung, Myo T Naung, Brendan R E Ansell, Andrew J Guy, Namrata Srivastava, Somya Mehra, Simone M Cacciò, Ivo Mueller, Alyssa E Barry, Cock van Oosterhout, Bernard Pope, David B Ascher, Aaron R Jex</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">Molecular Biology and Evolution</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">39</dd>
            
                          <dt class="col-sm-2">Issue</dt>
              <dd class="col-sm-10">9</dd>
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2022</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1093/molbev/msac196</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">36103257</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="https://doi.org/10.1093/molbev/msac196">https://doi.org/10.1093/molbev/msac196</a></dd>
            
                          <dt class="col-sm-2">Keywords</dt>
              <dd class="col-sm-10">data visualization; evolution; multi-dimensional analysis; population genetics; protein structure; variant interpretation</dd>
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10"><p>Large-scale comparative genomics- and population genetic studies generate enormous amounts of polymorphism data in the form of DNA variants. Ultimately, the goal of many of these studies is to associate genetic variants to phenotypes or fitness. We introduce VIVID, an interactive, user-friendly web application that integrates a wide range of approaches for encoding genotypic to phenotypic information in any organism or disease, from an individual or population, in three-dimensional (3D) space. It allows mutation mapping and annotation, calculation of interactions and conservation scores, prediction of harmful effects, analysis of diversity and selection, and 3D visualization of genotypic information encoded in Variant Call Format on AlphaFold2 protein models. VIVID enables the rapid assessment of genes of interest in the study of adaptive evolution and the genetic load, and it helps prioritizing targets for experimental validation. We demonstrate the utility of VIVID by exploring the evolutionary genetics of the parasitic protist Plasmodium falciparum, revealing geographic variation in the signature of balancing selection in potential targets of functional antibodies.</p>
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Perish and publish: Dynamics of biomedical publications by deceased authors</a>,  PLOS ONE,  2022.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Chol-Hee Jung, Paul C. Boutros, Daniel J. Park, Niall M. Corcoran, Bernard J. Pope, Christopher M. Hovens</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">PLOS ONE</dd>
            
            
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2022</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1371/journal.pone.0273783</dd>
            
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="https://doi.org/10.1371/journal.pone.0273783">https://doi.org/10.1371/journal.pone.0273783</a></dd>
            
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10"><p>The question of whether it is appropriate to attribute authorship to deceased individuals of original studies in the biomedical literature is contentious. Authorship guidelines utilized by journals do not provide a clear consensus framework that is binding on those in the field. To guide and inform the implementation of authorship frameworks it would be useful to understand the extent of the practice in the scientific literature, but studies that have systematically quantified the prevalence of this phenomenon in the biomedical literature have not been performed to date. To address this issue, we quantified the prevalence of publications by deceased authors in the biomedical literature from the period 1990–2020. We screened 2,601,457 peer-reviewed papers from the full text Europe PubMed Central database. We applied natural language processing, stringent filtering and manual curation to identify a final set of 1,439 deceased authors. We then determined these authors published a total of 38,907 papers over their careers with 5,477 published after death. The number of deceased publications has been growing rapidly, a 146-fold increase since the year 2000. This rate of increase was still significant when accounting for the growing total number of publications and pool of authors. We found that more than 50% of deceased author papers were first submitted after the death of the author and that over 60% of these papers failed to acknowledge the deceased authors status. Most deceased authors published less than 10 papers after death but a small pool of 30 authors published significantly more. A pool of 266 authors published more than 90% of their total publications after death. Our analysis indicates that the attribution of deceased authorship in the literature is not an occasional occurrence but a burgeoning trend. A consensus framework to address authorship by deceased scientists is warranted.</p>
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Identifying colorectal cancer caused by biallelic MUTYH pathogenic variants using tumor mutational signatures</a>,  Nature Communications,  2022.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Peter Georgeson, Tabitha A. Harrison, Bernard J. Pope, Syed H. Zaidi, Conghui Qu, Robert S. Steinfelder, Yi Lin, Jihoon E. Joo, Khalid Mahmood, Mark Clendenning, Romy Walker, Efrat L. Amitay, Sonja I. Berndt, Hermann Brenner, Peter T. Campbell, Yin Cao, Andrew T. Chan, Jenny Chang-Claude, Kimberly F. Doheny, David A. Drew, Jane C. Figueiredo, Amy J. French, Steven Gallinger, Marios Giannakis, Graham G. Giles, Andrea Gsur, Marc J. Gunter, Michael Hoffmeister, Li Hsu, Wen-Yi Huang, Paul Limburg, JoAnn E. Manson, Victor Moreno, Rami Nassir, Jonathan A. Nowak, Mireia Obón-Santacana, Shuji Ogino, Amanda I. Phipps, John D. Potter, Robert E. Schoen, Wei Sun, Amanda E. Toland, Quang M. Trinh, Tomotaka Ugai, Finlay A. Macrae, Christophe Rosty, Thomas J. Hudson, Mark A. Jenkins, Stephen N. Thibodeau, Ingrid M. Winship, Ulrike Peters, Daniel D. Buchanan</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">Nature Communications</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">13</dd>
            
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2022</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1038/s41467-022-30916-1</dd>
            
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="https://doi.org/10.1038/s41467-022-30916-1">https://doi.org/10.1038/s41467-022-30916-1</a></dd>
            
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10"><p>Carriers of germline biallelic pathogenic variants in the MUTYH gene have a high risk of colorectal cancer. We test 5649 colorectal cancers to evaluate the discriminatory potential of a tumor mutational signature specific to MUTYH for identifying biallelic carriers and classifying variants of uncertain clinical significance (VUS). Using a tumor and matched germline targeted multi-gene panel approach, our classifier identifies all biallelic MUTYH carriers and all known non-carriers in an independent test set of 3019 colorectal cancers (accuracy = 100% (95% confidence interval 99.87–100%)). All monoallelic MUTYH carriers are classified with the non-MUTYH carriers. The classifier provides evidence for a pathogenic classification for two VUS and a benign classification for five VUS. Somatic hotspot mutations KRAS p.G12C and PIK3CA p.Q546K are associated with colorectal cancers from biallelic MUTYH carriers compared with non-carriers (p = 2 × 10−23 and p = 6 × 10−11, respectively). Here, we demonstrate the potential application of mutational signatures to tumor sequencing workflows to improve the identification of biallelic MUTYH carriers.</p>
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Rare Germline Variants Are Associated with Rapid Biochemical Recurrence After Radical Prostate Cancer Treatment: A Pan Prostate Cancer Group Study</a>,  European Urology,  2022.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Daniel Burns, Ezequiel Anokian, Edward J. Saunders, Robert G. Bristow, Michael Fraser, Juri Reimand, Thorsten Schlomm, Guido Sauter, Benedikt Brors, Jan Korbel, Joachim Weischenfeldt, Sebastian M. Waszak, Niall M. Corcoran, Chol-Hee Jung, Bernard J. Pope, Chris M. Hovens, Geraldine Cancel-Tassin, Olivier Cussenot, Massimo Loda, Chris Sander, Vanessa M. Hayes, Karina Dalsgaard Sorensen, Yong-Jie Lu, Freddie C. Hamdy, Christopher S. Foster, Vincent Gnanapragasam, Adam Butler, Andy G. Lynch, Charlie E. Massie, Dan J. Woodcock, Colin S. Cooper, David C. Wedge, Daniel S. Brewer, Zsofia Kote-Jarai, Rosalind A. Eeles</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">European Urology</dd>
            
            
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2022</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1016/j.eururo.2022.05.007</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">35659150</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="https://doi.org/10.1016/j.eururo.2022.05.007">https://doi.org/10.1016/j.eururo.2022.05.007</a></dd>
            
                          <dt class="col-sm-2">Keywords</dt>
              <dd class="col-sm-10">Germline variants; Prostate cancer; Biochemical recurrence; Pan Prostate Cancer Group</dd>
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10"><p>Background</p><p>Germline variants explain more than a third of prostate cancer (PrCa) risk, but very few associations have been identified between heritable factors and clinical progression.</p><p>Objective</p><p>To find rare germline variants that predict time to biochemical recurrence (BCR) after radical treatment in men with PrCa and understand the genetic factors associated with such progression.</p><p>esign, setting, and participant</p><p>Whole-genome sequencing data from blood DNA were analysed for 850 PrCa patients with radical treatment from the Pan Prostate Cancer Group (PPCG) consortium from the UK, Canada, Germany, Australia, and France. Findings were validated using 383 patients from The Cancer Genome Atlas (TCGA) dataset.</p><p>Outcome measurements and statistical analysis</p><p>A total of 15, 822 rare (MAF <1%) predicted-deleterious coding germline mutations were identified. Optimal multifactor and univariate Cox regression models were built to predict time to BCR after radical treatment, using germline variants grouped by functionally annotated gene sets. Models were tested for robustness using bootstrap resampling.</p><p>Results and limitations</p><p>Optimal Cox regression multifactor models showed that rare predicted-deleterious germline variants in “Hallmark” gene sets were consistently associated with altered time to BCR. Three gene sets had a statistically significant association with risk-elevated outcome when modelling all samples: PI3K/AKT/mTOR, Inflammatory response, and KRAS signalling (up). PI3K/AKT/mTOR and KRAS signalling (up) were also associated among patients with higher-grade cancer, as were Pancreas-beta cells, TNFA signalling via NKFB, and Hypoxia, the latter of which was validated in the independent TCGA dataset.</p><p>Conclusions</p><p>We demonstrate for the first time that rare deleterious coding germline variants robustly associate with time to BCR after radical treatment, including cohort-independent validation. Our findings suggest that germline testing at diagnosis could aid clinical decisions by stratifying patients for differential clinical management.</p> 
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Phase 2 Study of Neoadjuvant FGFR Inhibition and Androgen Deprivation Therapy Prior to Prostatectomy</a>,  Clinical Genitourinary Cancer,  2022.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Elizabeth Liow, Nicholas Howard, Chol-Hee Jung, Bernard Pope, Bethany K. Campbell, Anne Nguyen, Michael Kerger, Jonathan B. Ruddle, Angelyn Anton, Benjamin Thomas, Kevin Chu, Philip Dundee, Justin S. Peters, Anthony J. Costello, Andrew S. Ryan, Christopher M. Hovens, Ben Tran, Niall M. Corcoran</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">Clinical Genitourinary Cancer</dd>
            
            
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2022</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1016/j.clgc.2022.05.007</dd>
            
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="https://doi.org/10.1016/j.clgc.2022.05.007">https://doi.org/10.1016/j.clgc.2022.05.007</a></dd>
            
                          <dt class="col-sm-2">Keywords</dt>
              <dd class="col-sm-10">High-risk Prostate Cancer; Neoadjuvant therapy; FGFR Inhibition; Androgen Deprivation Therapy; Phase 2 Study</dd>
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10"><p>Background</p><p>Disease recurrence is common following prostatectomy in patients with localised prostate cancer with high-risk features. Although androgen deprivation therapy increases the rates of organ-confined disease and negative surgical margins, there is no significant benefit on disease recurrence. Multiple lines of evidence suggest that FGF/FGFR signalling is important in supporting prostate epithelial cell survival in hostile conditions, including acute androgen deprivation. Given the recent availability of oral FGFR inhibitors, we investigated whether combination therapy could improve tumour response in the neo-adjuvant setting.</p><p>Methods</p><p>We conducted an open label phase II study of the combination of erdafitinib (3 months) and androgen deprivation therapy (4 months) in men with localised prostate cancer with high-risk features prior to prostatectomy using a Simon's two stage design. The co-primary endpoints were safety and tolerability and pathological response in the prostatectomy specimen. The effect of treatment on residual tumours was explored by global transcriptional profiling with RNA-sequencing.</p><p>Results</p><p>Nine patients were enrolled in the first stage of the trial. The treatment combination was poorly tolerated. Erdafitinib treatment was discontinued early in 6 patients, three of whom also required dose interruptions/reductions. Androgen deprivation therapy for 4 months was completed in all patients. The most common adverse events were hyperphosphataemia, taste disturbance, dry mouth and nail changes. No patients achieved a complete pathological response, although patients who tolerated erdafitinib for longer had smaller residual tumours, associated with reduced transcriptional signatures of epithelial cell proliferation.</p><p>Conclusions</p><p>Although there was a possible enhanced anti-tumour effect of androgen deprivation therapy in combination with erdafitnib in treatment naïve prostate cancer, the poor tolerability in this patient population prohibits the use of this combination in this setting.</p><p>Clinical Practice Points</p><p>Disease recurrence is common following prostatectomy in patients with localised prostate cancer with high-risk features. Although androgen deprivation therapy increases the rates of organ-confined disease and negative surgical margins, there is no significant benefit on disease recurrence. Multiple lines of evidence suggest that FGF/FGFR signalling is important in supporting prostate epithelial cell survival in hostile conditions, including acute androgen deprivation. We conducted an open label phase II study of the combination of erdafitinib (3 months) and androgen deprivation therapy (4 months) in men with localised prostate cancer with high-risk features prior to prostatectomy using a Simon's two stage design. The co-primary endpoints were safety and tolerability and pathological response in the prostatectomy specimen. The treatment combination was poorly tolerated. The most common adverse events were hyperphosphataemia, taste disturbance, dry mouth and nail changes. No patients achieved a complete pathological response, although patients who tolerated erdafitinib for longer had smaller residual tumours, associated with reduced transcriptional signatures of epithelial cell proliferation. Although there was a possible enhanced anti-tumour effect of androgen deprivation therapy in combination with erdafitnib in treatment naïve prostate cancer, the poor tolerability in this patient population prohibits the use of this combination in this setting.</p>
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Population-based estimates of breast cancer risk for carriers of pathogenic variants identified by gene-panel testing</a>,  npj Breast Cancer,  2021.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Melissa C. Southey, James G. Dowty, Moeen Riaz, Jason A. Steen, Anne-Laure Renault, Katherine Tucker, Judy Kirk, Paul James, Ingrid Winship, Nicholas Pachter, Nicola Poplawski, Scott Grist, Daniel J. Park, Bernard J. Pope, Khalid Mahmood, Fleur Hammet, Maryam Mahmoodi, Helen Tsimiklis, Derrick Theys, Amanda Rewse, Amanda Willis, April Morrow, Catherine Speechly, Rebecca Harris, Robert Sebra, Eric Schadt, Paul Lacaze, John J. McNeil, Graham G. Giles, Roger L. Milne, John L. Hopper, Tú Nguyen-Dumont</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">npj Breast Cancer</dd>
            
            
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2021</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1038/s41523-021-00360-3</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">34887416</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="https://doi.org/10.1038/s41523-021-00360-3">https://doi.org/10.1038/s41523-021-00360-3</a></dd>
            
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">Population-based estimates of breast cancer risk for carriers of pathogenic variants identified by gene-panel testing are urgently required. Most prior research has been based on women selected for high-risk features and more data is needed to make inference about breast cancer risk for women unselected for family history, an important consideration of population screening. We tested 1464 women diagnosed with breast cancer and 862 age-matched controls participating in the Australian Breast Cancer Family Study (ABCFS), and 6549 healthy, older Australian women enroled in the ASPirin in Reducing Events in the Elderly (ASPREE) study for rare germline variants using a 24-gene-panel. Odds ratios (ORs) were estimated using unconditional logistic regression adjusted for age and other potential confounders. We identified pathogenic variants in 11.1% of the ABCFS cases, 3.7% of the ABCFS controls and 2.2% of the ASPREE (control) participants. The estimated breast cancer OR [95% confidence interval] was 5.3 [2.1–16.2] for BRCA1, 4.0 [1.9–9.1] for BRCA2, 3.4 [1.4–8.4] for ATM and 4.3 [1.0–17.0] for PALB2. Our findings provide a population-based perspective to gene-panel testing for breast cancer predisposition and opportunities to improve predictors for identifying women who carry pathogenic variants in breast cancer predisposition genes.
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>SNPPar: identifying convergent evolution and other homoplasies from microbial whole-genome alignments</a>,  Microbial Genomics,  2021.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">David J. Edwards, Sebastián Duchene, Bernard Pope, Kathryn E. Holt</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">Microbial Genomics</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">7</dd>
            
                          <dt class="col-sm-2">Issue</dt>
              <dd class="col-sm-10">12</dd>
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2021</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1099/mgen.0.000694</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">34874243</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="https://doi.org/10.1099/mgen.0.000694">https://doi.org/10.1099/mgen.0.000694</a></dd>
            
                          <dt class="col-sm-2">Keywords</dt>
              <dd class="col-sm-10">bacteria; evolution; homoplasy; phylogeny</dd>
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">Homoplasic SNPs are considered important signatures of strong (positive) selective pressure, and hence of adaptive evolution for clinically relevant traits such as antibiotic resistance and virulence. Here we present a new tool, SNPPar, for efficient detection and analysis of homoplasic SNPs from large whole genome sequencing datasets (>1000 isolates and/or >100 000 SNPs). SNPPar takes as input an SNP alignment, tree and annotated reference genome, and uses a combination of simple monophyly tests and ancestral state reconstruction (ASR, via TreeTime) to assign mutation events to branches and identify homoplasies. Mutations are annotated at the level of codon and gene, to facilitate analysis of convergent evolution. Testing on simulated data (120 Mycobacterium tuberculosis alignments representing local and global samples) showed SNPPar can detect homoplasic SNPs with very high specificity (zero false-positives in all tests) and high sensitivity (zero false-negatives in 89 % of tests). SNPPar analysis of three empirically sampled datasets (Elizabethkingia anophelis, Burkholderia dolosa and M. tuberculosis) produced results that were in concordance with previous studies, in terms of both individual homoplasies and evidence of convergence at the codon and gene levels. SNPPar analysis of a simulated alignment of ~64 000 genome-wide SNPs from 2000 M. tuberculosis genomes took ~23 min and ~2.6 GB of RAM to generate complete annotated results on a laptop. This analysis required ASR be conducted for only 1.25 % of SNPs, and the ASR step took ~23 s and 0.4 GB of RAM. SNPPar automates the detection and annotation of homoplasic SNPs efficiently and accurately from large SNP alignments. As demonstrated by the examples included here, this information can be readily used to explore the role of homoplasy in parallel and/or convergent evolution at the level of nucleotide, codon and/or gene. 
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Rare germline variants in the AXIN2 gene in families with colonic polyposis and colorectal cancer</a>,  Familial Cancer,  2021.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">James M Chan, Mark Clendenning, Sharelle Joseland, Peter Georgeson2, Khalid Mahmood, Romy Walker, Julia Como, Jihoon E Joo, Susan Preston, Ryan A Hutchinson, Bernard J Pope, Andrew Metz, Catherine Beard, Rebecca Purvis, Julie Arnold, Varnika Vijay, Galina Konycheva, Nathan Atkinson, Susan Parry, Mark A Jenkins, Finlay A Macrae, Christophe Rosty, Ingrid M Winship, Daniel D Buchanan</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">Familial Cancer</dd>
            
            
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2021</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1007/s10689-021-00283-9</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">34817745</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="https://doi.org/10.1007/s10689-021-00283-9">https://doi.org/10.1007/s10689-021-00283-9</a></dd>
            
                          <dt class="col-sm-2">Keywords</dt>
              <dd class="col-sm-10">AXIN2; Colonic polyposis; Colorectal cancer; Ectodermal dysplasia; Oligodontia</dd>
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">Germline loss-of-function variants in AXIN2 are associated with oligodontia and ectodermal dysplasia. The association between colorectal cancer (CRC) and colonic polyposis is less clear despite this gene now being included in multi-gene panels for CRC. Study participants were people with genetically unexplained colonic polyposis recruited to the Genetics of Colonic Polyposis Study who had a rare germline AXIN2 gene variant identified from either clinical multi-gene panel testing (n=2) or from whole genome/exome sequencing (n=2). Variant segregation in relatives and characterisation of tumour tissue were performed where possible. Four different germline pathogenic variants in AXIN2 were identified in four families. Five of the seven carriers of the c.1049delC, p.Pro350Leufs*13 variant, two of the six carriers of the c.1994dupG, p.Asn666Glnfs*41 variant, all three carriers of c.1972delA, p.Ser658Alafs*31 variant and the single proband carrier of the c.2405G>C, p.Arg802Thr variant, which creates an alternate splice form resulting in a frameshift mutation (p.Glu763Ilefs*42), were affected by CRC and/or polyposis. Carriers had a mean age at diagnosis of CRC/polyposis of 52.5 ± 9.2 years. Colonic polyps were typically pan colonic with counts ranging from 5 to >100 (median 12.5) comprising predominantly adenomatous polyps but also serrated polyps. Two CRCs from carriers displayed evidence of a second hit via loss of heterozygosity. Oligodontia was observed in carriers from two families. Germline AXIN2 pathogenic variants from four families were associated with CRC and/or polyposis in multiple family members. These findings support the inclusion of AXIN2 in CRC and polyposis multigene panels for clinical testing.
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Long-read assembly and comparative evidence-based reanalysis of Cryptosporidium genome sequences reveals expanded transporter repertoire and duplication of entire chromosome ends including subtelomeric regions</a>,  Genome Research,  2021.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Rodrigo P Baptista, Yiran Li, Adam Sateriale, Karen L Brooks, Alan Tracey, Mandy J Sanders, Brendan R E Ansell, Aaron R Jex, Garrett W Cooper, Ethan D Smith, Rui Xiao, Jennifer E Dumaine, Peter Georgeson, Bernard Pope, Matthew Berriman, Boris Striepen, James A Cotton, Jessica C Kissinger</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">Genome Research</dd>
            
            
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2021</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1101/gr.275325.121</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">34764149</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="https://doi.org/10.1101/gr.275325.121">https://doi.org/10.1101/gr.275325.121</a></dd>
            
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">Cryptosporidiosis is a leading cause of waterborne diarrheal disease globally and an important contributor to mortality in infants and the immunosuppressed. Despite its importance, the Cryptosporidium community has only had access to a good, but incomplete, Cryptosporidium parvum IOWA reference genome sequence. Incomplete reference sequences hamper annotation, experimental design and interpretation. We have generated a new C. parvum IOWA genome assembly supported by PacBio and Oxford Nanopore long-read technologies and a new comparative and consistent genome annotation for three closely related species C. parvum, Cryptosporidium hominis and Cryptosporidium tyzzeri We made 1,926 C. parvum annotation updates based on experimental evidence. They include new transporters, ncRNAs, introns and altered gene structures. The new assembly and annotation revealed a complete Dnmt2 methylase ortholog. Comparative annotation between C. parvum, C. hominis and C. tyzzeri revealed that most "missing" orthologs are found suggesting that the biological differences between the species must result from gene copy number variation, differences in gene regulation and single nucleotide variants (SNVs). Using the new assembly and annotation as reference, 190 genes are identified as evolving under positive selection, including many not detected previously. The new C. parvum IOWA reference genome assembly is larger, gap free and lacks ambiguous bases. This chromosomal assembly recovers all 16 chromosome ends, 13 of which are contiguously assembled. The three remaining chromosome ends are provisionally placed. These ends represent duplication of entire chromosome ends including subtelomeric regions revealing a new level of genome plasticity that will both inform and impact future research.
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>MSH2-deficient prostate tumours have a distinct immune response and clinical outcome compared to MSH2-deficient colorectal or endometrial cancer</a>,  Prostate Cancer and Prostatic Diseases,  2021.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Patrick McCoy, Stefano Mangiola, Geoff Macintyre, Ryan Hutchinson, Ben Tran, Bernard Pope, Peter Georgeson, Matthew K. H. Hong, Natalie Kurganovs, Sebastian Lunke, Michael J. Clarkson, Marek Cmero, Michael Kerger, Ryan Stuchbery, Ken Chow, Izhak Haviv, Andrew Ryan, Anthony J. Costello, Niall M. Corcoran, Christopher M. Hovens</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">Prostate Cancer and Prostatic Diseases</dd>
            
            
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2021</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1038/s41391-021-00379-4</dd>
            
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="https://doi.org/10.1038/s41391-021-00379-4">https://doi.org/10.1038/s41391-021-00379-4</a></dd>
            
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10"><p>Background</p><p>Recent publications have shown patients with defects in the DNA mismatch repair (MMR) pathway driven by either MSH2 or MSH6 loss experience a significant increase in the incidence of prostate cancer. Moreover, this increased incidence of prostate cancer is accompanied by rapid disease progression and poor clinical outcomes.</p><p>Methods and results</p><p>We show that androgen-receptor activation, a key driver of prostate carcinogenesis, can disrupt the MSH2 gene in prostate cancer. We screened tumours from two cohorts (recurrent/non-recurrent) of prostate cancer patients to confirm the loss of MSH2 protein expression and identified decreased MSH2 expression in recurrent cases. Stratifying the independent TCGA prostate cancer cohort for MSH2/6 expression revealed that patients with lower levels of MSH2/6 had significant worse outcomes, in contrast, endometrial and colorectal cancer patients with lower MSH2/6 levels. MMRd endometrial and colorectal tumours showed the expected increase in mutational burden, microsatellite instability and enhanced immune cell mobilisation but this was not evident in prostate tumours.</p><p>Conclusions</p><p>We have shown that loss or reduced levels of MSH2/MSH6 protein in prostate cancer is associated with poor outcome. However, our data indicate that this is not associated with a statistically significant increase in mutational burden, microsatellite instability or immune cell mobilisation in a cohort of primary prostate cancers.</p>
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Evaluating the utility of tumour mutational signatures for identifying hereditary colorectal cancer and polyposis syndrome carriers</a>,  Gut,  2021.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Peter Georgeson, Bernard J Pope, Christophe Rosty, Mark Clendenning, Khalid Mahmood, Jihoon E Joo, Romy Walker, Ryan A Hutchinson, Susan Preston, Julia Como, Sharelle Joseland, Aung Ko Win, Finlay A Macrae, John L Hopper, Dmitri Mouradov, Peter Gibbs, Oliver M Sieber, Dylan E O'Sullivan, Darren R Brenner, Steve Gallinger, Mark A Jenkins, Ingrid M Winship, Daniel D Buchanan</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">Gut</dd>
            
            
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2021</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1136/gutjnl-2019-320462</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">33414168</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="https://doi.org/10.1136/gutjnl-2019-320462">https://doi.org/10.1136/gutjnl-2019-320462</a></dd>
            
                          <dt class="col-sm-2">Keywords</dt>
              <dd class="col-sm-10">colorectal cancer; colorectal cancer screening; molecular pathology; mutations; tumour markers</dd>
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10"><p>Objective</p><p>Germline pathogenic variants (PVs) in the DNA mismatch repair (MMR) genes and in the base excision repair gene MUTYH underlie hereditary colorectal cancer (CRC) and polyposis syndromes. We evaluated the robustness and discriminatory potential of tumour mutational signatures in CRCs for identifying germline PV carriers.</p><p>Design</p><p>Whole-exome sequencing of formalin-fixed paraffin-embedded (FFPE) CRC tissue was performed on 33 MMR germline PV carriers, 12 biallelic MUTYH germline PV carriers, 25 sporadic MLH1 methylated MMR-deficient CRCs (MMRd controls) and 160 sporadic MMR-proficient CRCs (MMRp controls) and included 498 TCGA CRC tumours. COSMIC V3 single base substitution (SBS) and indel (ID) mutational signatures were assessed for their ability to differentiate CRCs that developed in carriers from non-carriers.</p><p>Results</p><p>The combination of mutational signatures SBS18 and SBS36 contributing >30% of a CRC’s signature profile was able to discriminate biallelic MUTYH carriers from all other non-carrier control CRCs with 100% accuracy (area under the curve (AUC) 1.0). SBS18 and SBS36 were associated with specific MUTYH variants p.Gly396Asp (p=0.025) and p.Tyr179Cys (p=5×10-5), respectively. The combination of ID2 and ID7 could discriminate the 33 MMR PV carrier CRCs from the MMRp control CRCs (AUC 0.99); however, SBS and ID signatures, alone or in combination, could not provide complete discrimination (AUC 0.79) between CRCs from MMR PV carriers and sporadic MMRd controls.</p><p>Conclusion</p><p>Assessment of SBS and ID signatures can discriminate CRCs from biallelic MUTYH carriers and MMR PV carriers from non-carriers with high accuracy, demonstrating utility as a potential diagnostic and variant classification tool.</p>
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Germline and Tumor Sequencing as a Diagnostic Tool to Resolve Suspected Lynch Syndrome</a>,  The Journal of Molecular Diagnostics,  2020.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Bernard J. Pope, Mark Clendenning, Christophe Rosty, Khalid Mahmood, Peter Georgeson, Jihoon E. Joo, Romy Walker, Ryan A. Hutchinson, Harindra Jayasekara, Sharelle Joseland, Julia Como, Susan Preston, Amanda B. Spurdle, Finlay A. Macrae, Aung K. Win, John L. Hopper, Mark A. Jenkins, Ingrid M. Winship, Daniel D. Buchanan</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">The Journal of Molecular Diagnostics</dd>
            
            
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2020</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1016/j.jmoldx.2020.12.003</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">33383211</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="https://doi.org/10.1016/j.jmoldx.2020.12.003">https://doi.org/10.1016/j.jmoldx.2020.12.003</a></dd>
            
                          <dt class="col-sm-2">Keywords</dt>
              <dd class="col-sm-10">Lynch syndrome; Suspected Lynch syndrome; Whole Genome Sequencing; Mismatch Repair Deficiency; double somatic mutations; colorectal cancer</dd>
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10"><p>People who develop mismatch repair (MMR) deficient cancer in the absence of a germline MMR gene pathogenic variant or somatic hypermethylation of the MLH1 gene promoter are classified as having suspected Lynch syndrome (SLS). Germline whole genome sequencing (WGS) and targeted and genome-wide tumor sequencing was applied to identify the underlying cause of tumor MMR-deficiency in SLS. Germline WGS was performed on 14 cancer-affected people with SLS, including two sets of first-degree relatives. Germline pathogenic variants, including complex structural rearrangements and non-coding variants, were assessed for the MMR genes. Tumor tissue was sequenced for somatic MMR gene mutations by targeted, whole exome sequencing (WES) or WGS. Germline WGS identified pathogenic MMR variants in 3 of the 14 (21.4%) SLS cases including a 9.5Mb inversion disrupting MSH2 in a mother and daughter. Excluding these 3 MMR carriers, tumor sequencing identified at least two somatic MMR gene mutations in 8/11 (72.7%) tumors tested. In a second mother-daughter pair, a somatic cause of their tumor MMR-deficiency was supported by the presence of double somatic MSH2 mutations in their respective tumors. More than 70% of SLS were resolved as having double somatic MMR mutations in the absence of germline pathogenic variants in the MMR or other DNA repair-related genes as determine by WGS and, therefore, confidently assigned a non-inherited cause for their tumor MMR-deficiency.</p>
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Monoallelic NTHL1 Loss of Function Variants and Risk of Polyposis and Colorectal Cancer</a>,  Gastroenterology,  2020.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Fadwa A Elsayed, Judith E Grolleman, Abiramy Ragunathan, Arnoud Boot, Marija Staninova Stojovska, Khalid Mahmood, Mark Clendenning, Noel de Miranda, Dagmara Dymerska, Demi van Egmond, Steven Gallinger, Peter Georgeson, Nicoline Hoogerbrugge, John L. Hopper, Erik A.M. Jansen, Mark A. Jenkins, Jihoon E. Joo, Roland P. Kuiper, Marjolijn J.L. Ligtenberg, Jan Lubinski, Finlay A. Macrae, Hans Morreau, Polly Newcomb, Maartje Nielsen, Claire Palles, Daniel J. Park, Bernard J. Pope, Christophe Rosty, Clara Ruiz Ponte, Hans K. Schackert, Rolf H. Sijmons, Ian P. Tomlinson, Carli M. J. Tops, Lilian Vreede, Romy Walker, Aung K. Win, Colon Cancer Family Registry Cohort Investigators, Aleksandar J. Dimovski, Ingrid M. Winship, Daniel D Buchanan, Tom van Wezel, Richarda M de Voer</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">Gastroenterology</dd>
            
            
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2020</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1053/j.gastro.2020.08.042</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">32860789</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="https://doi.org/10.1053/j.gastro.2020.08.042">https://doi.org/10.1053/j.gastro.2020.08.042</a></dd>
            
            
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Detection of ctDNA in plasma of patients with clinically localised prostate cancer is associated with rapid disease progression</a>,  Genome Medicine,  2020.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Edmund Lau, Patrick McCoy, Fairleigh Reeves, Ken Chow, Michael Clarkson, Edmond M. Kwan, Kate Packwood, Helen Northen, Miao He, Zoya Kingsbury, Stefano Mangiola, Michael Kerger, Marc A. Furrer, Helen Crowe, Anthony J. Costello, David J. McBride, Mark T. Ross, Bernard Pope, Christopher M. Hovens, Niall M. Corcoran</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">Genome Medicine</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">12</dd>
            
                          <dt class="col-sm-2">Issue</dt>
              <dd class="col-sm-10">1</dd>
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2020</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1186/s13073-020-00770-1</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">32807235</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="https://doi.org/10.1186/s13073-020-00770-1">https://doi.org/10.1186/s13073-020-00770-1</a></dd>
            
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10"><p>Background</p><p>DNA originating from degenerate tumour cells can be detected in the circulation in many tumour types, where it can be used as a marker of disease burden as well as to monitor treatment response. Although circulating tumour DNA (ctDNA) measurement has prognostic/predictive value in metastatic prostate cancer, its utility in localised disease is unknown.</p><p>Methods</p><p>We performed whole-genome sequencing of tumour-normal pairs in eight patients with clinically localised disease undergoing prostatectomy, identifying high confidence genomic aberrations. A bespoke DNA capture and amplification panel against the highest prevalence, highest confidence aberrations for each individual was designed and used to interrogate ctDNA isolated from plasma prospectively obtained pre- and post- (24 h and 6 weeks) surgery. In a separate cohort (n = 189), we identified the presence of ctDNA TP53 mutations in preoperative plasma in a retrospective cohort and determined its association with biochemical- and metastasis-free survival.</p><p>Results</p>Tumour variants in ctDNA were positively identified pre-treatment in two of eight patients, which in both cases remained detectable postoperatively. Patients with tumour variants in ctDNA had extremely rapid disease recurrence and progression compared to those where variants could not be detected. In terms of aberrations targeted, single nucleotide and structural variants outperformed indels and copy number aberrations. Detection of ctDNA TP53 mutations was associated with a significantly shorter metastasis-free survival (6.2 vs. 9.5 years (HR 2.4; 95% CIs 1.2–4.8, p = 0.014).</p><p>Conclusions</p><p>CtDNA is uncommonly detected in localised prostate cancer, but its presence portends more rapidly progressive disease.</p>
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Genetic testing in Poland and Ukraine: should comprehensive germline testing of BRCA1 and BRCA2 be recommended for women with breast and ovarian cancer?</a>,  Genetics Research,  2020.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Tu Nguyen-Dumont, Pawel Karpinski Maria M. Sasiadek, Hayane Akopyan, Jason A. Steen, Derrick Theys, Fleur Hammet, Helen Tsimiklis, Daniel J. Park, Bernard J. Pope, Ryszard Slezak, Agnieszka Stembalska, Karolina Pesz, Nataliya Kitsera, Aleksandra Siekierzynska, Melissa C. Southey, Aleksander Myszka</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">Genetics Research</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">102</dd>
            
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2020</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1017/S0016672320000075</dd>
            
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="https://doi.org/10.1017/S0016672320000075">https://doi.org/10.1017/S0016672320000075</a></dd>
            
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10"><p>Purpose</p><p>To characterize the spectrum of BRCA1 and BRCA2 pathogenic germline variants in women from south-west Poland and west Ukraine affected with breast or ovarian cancer. Testing in women at high risk of breast and ovarian cancer in these regions is currently mainly limited to founder mutations.</p><p>Methods</p><p>Unrelated women affected with breast and/or ovarian cancer from Poland (n = 337) and Ukraine (n = 123) were screened by targeted sequencing. Excluded from targeted sequencing were 34 Polish women who had previously been identified as carrying a founder mutation in BRCA1. No prior testing had been conducted among the Ukrainian women. Thus, this study screened BRCA1 and BRCA2 in the germline DNA of 426 women in total.</p><p>Results</p><p>We identified 31 and 18 women as carriers of pathogenic/likely pathogenic (P/LP) genetic variants in BRCA1 and BRCA2, respectively. We observed five BRCA1 and eight BRCA2 P/LP variants (13/337, 3.9%) in the Polish women. Combined with the 34/337 (10.1%) founder variants identified prior to this study, the overall P/LP variant frequency in the Polish women was thus 14% (47/337). Among the Ukrainian women, 16/123 (13%) women were identified as carrying a founder mutation and 20/123 (16.3%) were found to carry non-founder P/LP variants (10 in BRCA1 and 10 in BRCA2).</p><p>Conclusions</p><p>These results indicate that genetic testing in women at high risk of breast and ovarian cancer in Poland and Ukraine should not be limited to founder mutations. Extended testing will enhance risk stratification and management for these women and their families.</p>
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>HiTIME: An efficient model-selection approach for the detection of unknown drug metabolites in LC-MS data</a>,  SoftwareX,  2020.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Michael G. Leeming, Andrew P. Isaac, Luke Zappia, Richard A. J. O'Hair, William A. Donald, Bernard J. Pope</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">SoftwareX</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">12</dd>
            
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2020</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1016/j.softx.2020.100559</dd>
            
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="https://doi.org/10.1016/j.softx.2020.100559">https://doi.org/10.1016/j.softx.2020.100559</a></dd>
            
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10"><p>The identification of metabolites plays an important role in understanding drug efficacy and safety however these compounds are often difficult to identify in complex mixtures. One approach to identify drug metabolites involves utilising differentially isotopically labelled drug compounds to create unique isotopic signals that can be detected by liquid chromatography-mass spectrometry (LC-MS). User-friendly, efficient, computational tools that allow selective detection of these signals are lacking. We have developed an efficient open-source software tool called HiTIME (High-Resolution Twin-Ion Metabolite Extraction) which filters twin-ion signals in LC-MS data. The intensity of each data point in the input is replaced by a Z-score describing how well the point matches an idealised twin-ion signal versus alternative ion signatures. Here we provide a detailed description of the algorithm and demonstrate its performance on simulated and experimental data.</p>
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Rare germline genetic variants and risk of aggressive prostate cancer</a>,  International Journal of Cancer,  2020.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Nguyen-Dumont T, MacInnis R J, Steen J A, Theys D, Tsimiklis H, Hammet F, Mahmoodi M, Pope B J, Park D J, Mahmood K, Severi G, Bolton D, Milne R L, Giles G G, Southey M C</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">International Journal of Cancer</dd>
            
            
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2020</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1002/ijc.33024</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">32338768</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="https://doi.org/10.1002/ijc.33024">https://doi.org/10.1002/ijc.33024</a></dd>
            
                          <dt class="col-sm-2">Keywords</dt>
              <dd class="col-sm-10">aggressive prostate cancer; gene panel testing; germline genetic variants</dd>
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10"><p>Few genetic risk factors have been demonstrated to be specifically associated with aggressive prostate cancer (PrCa). Here, we report a case-case study of PrCa comparing the prevalence of germline pathogenic/likely pathogenic (P/LP) genetic variants in 787 men with aggressive disease and 769 with non-aggressive disease. Overall, we observed P/LP variants in 11.4% of men with aggressive PrCa and 9.8% of men with non-aggressive PrCa (two-tailed Fisher's exact tests, P = 0.28). The proportion of BRCA2 and ATM P/LP variant carriers in men with aggressive PrCa exceeded that observed in men with non-aggressive PrCa; 18/787 carriers (2.3%) and 4/769 carriers (0.5%), P = 0.004, and 14/787 carriers (0.02%) and 5/769 carriers (0.01%), P = 0.06, respectively. Our findings contribute to the extensive international effort to interpret the genetic variation identified in genes included on gene-panel tests, for which there is currently an insufficient evidence-base for clinical translation in the context of PrCa risk.</p>
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Mismatch repair gene pathogenic germline variants in a population-based cohort of breast cancer</a>,  Familial Cancer,  2020.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Tu Nguyen‑Dumont, Jason A. Steen, Ingrid Winship, Daniel J. Park, Bernard J. Pope, Fleur Hammet, Maryam Mahmoodi, Helen Tsimiklis, Derrick Theys, Mark Clendenning, Graham G. Giles, John L. Hopper, Melissa C. Southey</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">Familial Cancer</dd>
            
            
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2020</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1007/s10689-020-00164-7</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">32060697</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="https://doi.org/10.1007/s10689-020-00164-7">https://doi.org/10.1007/s10689-020-00164-7</a></dd>
            
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10"><p>The advent of gene panel testing is challenging the previous practice of using clinically defined cancer family syndromes to inform single-gene genetic screening. Individual and family cancer histories that would have previously indicated testing of a single gene or a small number of related genes are now, increasingly, leading to screening across gene panels that contain larger numbers of genes. We have applied a gene panel test that included four DNA mismatch repair (MMR) genes (MLH1, MSH2, MSH6 and PMS2) to an Australian population-based case–control-family study of breast cancer. Altogether, eight pathogenic variants in MMR genes were identified: six in 1421 case-families (0.4%, 4 MSH6 and 2 PMS2) and two in 833 control-families (0.2%, one each of MLH1 and MSH2). This testing highlights the current and future challenges for clinical genetics in the context of anticipated gene panel-based population-based screening that includes the MMR genes. This test-ing is likely to provide additional opportunities for cancer prevention via cascade testing for Lynch syndrome and precision medicine for breast cancer treatment.</p>
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Bionitio: demonstrating and facilitating best practices for bioinformatics command-line software</a>,  GigaScience,  2019.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Peter Georgeson, Anna Syme, Clare Sloggett, Jessica Chung, Harriet Dashnow, Michael Milton, Andrew Lonsdale, David Powell, Torsten Seemann, Bernard Pope</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">GigaScience</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">8</dd>
            
                          <dt class="col-sm-2">Issue</dt>
              <dd class="col-sm-10">9</dd>
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2019</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1093/gigascience/giz109</dd>
            
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="https://doi.org/10.1093/gigascience/giz109">https://doi.org/10.1093/gigascience/giz109</a></dd>
            
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10"><p>Background. Bioinformatics software tools are often created ad hoc, frequently by people without extensive training in software development. In particular, for beginners, the barrier to entry in bioinformatics software development is high, especially if they want to adopt good programming practices. Even experienced developers do not always follow best practices. This results in the proliferation of poorer-quality bioinformatics software, leading to limited scalability and inefficient use of resources; lack of reproducibility, usability, adaptability, and interoperability; and erroneous or inaccurate results.</p><p>Findings. We have developed Bionitio, a tool that automates the process of starting new bioinformatics software projects following recommended best practices. With a single command, the user can create a new well-structured project in 1 of 12 programming languages. The resulting software is functional, carrying out a prototypical bioinformatics task, and thus serves as both a working example and a template for building new tools. Key features include command-line argument parsing, error handling, progress logging, defined exit status values, a test suite, a version number, standardized building and packaging, user documentation, code documentation, a standard open source software license, software revision control, and containerization.</p><p>Conclusions. Bionitio serves as a learning aid for beginner-to-intermediate bioinformatics programmers and provides an excellent starting point for new projects. This helps developers adopt good programming practices from the beginning of a project and encourages high-quality tools to be developed more rapidly. This also benefits users because tools are more easily installed and consistent in their usage. Bionitio is released as open source software under the MIT License and is available at <a href="https://github.com/bionitio-team/bionitio">https://github.com/bionitio-team/bionitio</a></p>
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Tumor mutational signatures in sebaceous skin lesions from individuals with Lynch syndrome</a>,  Molecular Genetics & Genomic Medicine,  2019.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Georgeson P, Walsh MD, Clendenning M, Daneshvar S, Pope BJ, Mahmood K, Joo JE, Jayasekara H, Jenkins M, Winship IM, Buchanan DD</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">Molecular Genetics & Genomic Medicine</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">7</dd>
            
                          <dt class="col-sm-2">Issue</dt>
              <dd class="col-sm-10">7</dd>
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2019</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1002/mgg3.781</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">31162827</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="https://doi.org/10.1002/mgg3.781">https://doi.org/10.1002/mgg3.781</a></dd>
            
                          <dt class="col-sm-2">Keywords</dt>
              <dd class="col-sm-10">Lynch syndrome; Muir-Torre syndrome; mismatch repair deficiency; mismatch repair immunohistochemistry; sebaceoma; sebaceous adenoma; sebaceous carcinoma</dd>
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10"><p>BACKGROUND: Muir-Torre syndrome is defined by the development of sebaceous skin lesions in individuals who carry a germline mismatch repair (MMR) gene mutation. Loss of expression of MMR proteins is frequently observed in sebaceous skin lesions, but MMR-deficiency alone is not diagnostic for carrying a germline MMR gene mutation.</p><p> METHODS: Whole exome sequencing was performed on three MMR-deficient sebaceous lesions from individuals with MSH2 gene mutations (Lynch syndrome) and three MMR-proficient sebaceous lesions from individuals without Lynch syndrome with the aim of characterizing the tumor mutational signatures, somatic mutation burden, and microsatellite instability status. Thirty predefined somatic mutational signatures were calculated for each lesion.</p><p> RESULTS: Signature 1 was ubiquitous across the six lesions tested. Signatures 6 and 15, associated with defective DNA MMR, were significantly more prevalent in the MMR-deficient lesions from the MSH2 carriers compared with the MMR-proficient non-Lynch sebaceous lesions (mean ± SD=41.0 ± 8.2% vs. 2.3 ± 4.0%, p = 0.0018). Tumor mutation burden was, on average, significantly higher in the MMR-deficient lesions compared with the MMR-proficient lesions (23.3 ± 11.4 vs. 1.8 ± 0.8 mutations/Mb, p = 0.03). All four sebaceous lesions observed in sun exposed areas of the body demonstrated signature 7 related to ultraviolet light exposure.</p><p> CONCLUSION: Tumor mutational signatures 6 and 15 and somatic mutation burden were effective in differentiating Lynch-related from non-Lynch sebaceous lesions.</p>
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Hi-Plex2: a simple and robust approach to targeted sequencing-based genetic screening</a>,  BioTechniques,  2019.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Fleur Hammet, Khalid Mahmood, Thomas R Green, Tu Nguyen-Dumont, Melissa C Southey, Daniel D Buchanan, Andrew Lonie, Katherine L Nathanson, Fergus J Couch, Bernard J Pope, Daniel J Park</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">BioTechniques</dd>
            
            
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2019</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.2144/btn-2019-0026</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">31267764</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="http://dx.doi.org/10.2144/btn-2019-0026">http://dx.doi.org/10.2144/btn-2019-0026</a></dd>
            
                          <dt class="col-sm-2">Keywords</dt>
              <dd class="col-sm-10">genetic screening, Hi-Plex, multiplex PCR, targeted DNA sequencing, variant detection</dd>
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">We have previously reported Hi-Plex, a multiplex PCR methodology for building targeted DNA sequencing libraries that offers a low-cost protocol compatible with high-throughput processing. Here, we detail an improved protocol, Hi-Plex2, that more effectively enables the robust construction of small-to-medium panel-size libraries while maintaining low cost, simplicity and accuracy benefits of the Hi-Plex platform. Hi-Plex2 was applied to three panels, comprising 291, 740 and 1193 amplicons, targeting genes associated with risk for breast and/or colon cancer. We show substantial reduction of off-target amplification to enable library construction for small-to-medium-sized design panels not possible using the previous Hi-Plex chemistry.</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Annotation of the Giardia proteome through structure-based homology and machine learning</a>,  Gigascience,  2019.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Brendan R E Ansell, Bernard J Pope, Peter Georgeson, Samantha J Emery-Corbin, Aaron R Jex</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">Gigascience</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">8</dd>
            
                          <dt class="col-sm-2">Issue</dt>
              <dd class="col-sm-10">1</dd>
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2019</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1093/gigascience/giy150</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">30520990</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="http://dx.doi.org/10.1093/gigascience/giy150">http://dx.doi.org/10.1093/gigascience/giy150</a></dd>
            
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10"><p>Background</p><p>Large-scale computational prediction of protein structures represents a cost-effective alternative to empirical structure determination with particular promise for non-model organisms and neglected pathogens. Conventional sequence-based tools are insufficient to annotate the genomes of such divergent biological systems. Conversely, protein structure tolerates substantial variation in primary amino acid sequence, and is thus a robust indicator of biochemical function. Structural proteomics is poised to become a standard part of pathogen genomics research, however informatic methods are now required to assign confidence in large volumes of predicted structures.</p><p>Aims</p><p>To predict the proteome of a neglected human pathogen, Giardia duodenalis, and stratify predicted structures into high- and lower-confidence categories using a variety of metrics in isolation and combination.</p><p>Methods</p><p>We used the I-TASSER suite to predict structural models for ∼5000 proteins encoded in Giardia duodenalis and identify their closest empirically determined structural homologues in the Protein Data Bank. Models were assigned to high or lower-confidence categories depending on the presence of matching PFAM domains in query and reference peptides. Metrics output from the suite and derived metrics were assessed for their ability to predict the high confidence category individually, and in combination through development of a random forest classifier.</p><p>Results</p><p>We identified 1095 high confidence models including 212 hypothetical proteins. Amino acid identity between query and reference peptides was the greatest individual predictor of high confidence status, however the random forest classifier out-performed any metric in isolation (AUC = 0.977), and identified a subset of 305 high confidence-like models, corresponding to false positive predictions. High confidence models exhibited higher transcriptional abundance, and the classifier generalized across species, indicating the broad utility of this approach for automatically stratifying predicted structures. Additional structure-based clustering was used to cross-check confidence predictions in an expanded family of Nek kinases. Several high confidence-like proteins yielded substantial new insight into mechanisms of redox balance in Giardia duodenalis—a system central to the efficacy of limited anti-giardial drugs.</p><p>Conclusion</p><p>Structural proteomics combined with machine learning can aid genome annotation for genetically divergent organisms including human pathogens, and stratify predicted structures to promote efficient allocation of limited resources for experimental investigation.</p>
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Reduced familial fertility in carriers of mutations in the BRCA1 and BRCA2 genes</a>,  European Journal of Human Genetics,  2019.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Akopyan H, Kitsera N, Siekierzynska A, Nguyen-Dumont T, Hammet F, Tsimiklis H, Park DJ, Pope BJ, Southey MC, Blonioarz D, Myszka A</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">European Journal of Human Genetics</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">27</dd>
            
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2019</dd>
            
            
            
            
                          <dt class="col-sm-2">Keywords</dt>
              <dd class="col-sm-10">Science & Technology, Life Sciences & Biomedicine, Biochemistry & Molecular Biology, Genetics & Heredity</dd>
            
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Frequency of BRCA1 and BRCA2 Germline Variants in Women With Ovarian Cancer in Malaysia</a>,  Journal of Global Oncology,  2018.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Lim J, Lau SY, Bashah NS Ahmad, Lai KN, Wen WX; Hasan SN, Park DJ, Pope BJ, Nguyen-Dumont T, Southey MC, Rahman N, Woo YL, Thong MK, Ch'ng GS, Teo SH, Yoon SY</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">Journal of Global Oncology</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">4</dd>
            
                          <dt class="col-sm-2">Issue</dt>
              <dd class="col-sm-10">Supplement 2</dd>
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2018</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1200/jgo.18.50600</dd>
            
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="http://dx.doi.org/10.1200/jgo.18.50600">http://dx.doi.org/10.1200/jgo.18.50600</a></dd>
            
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">Background: Germline BRCA1 or BRCA2 pathogenic variants in ovarian cancer patients may be informative in risk management and treatment, with the advent of poly(ADP-ribose) polymerase inhibitors. In the era of precision medicine, companion diagnostics for BRCA1 and BRCA2 genes have been featured as a strategy in the Malaysia National Strategic Plan for Cancer Control Program (2016-2020). To facilitate this strategy, frequency data from Malaysia's understudied multiethnic population will be required. Aim: To determine the prevalence of BRCA1 and BRCA2 germline variants in a population-based cohort of ovarian cancer patients in Malaysia. Methods: From August 2016, women with nonmucinous epithelial ovarian, peritoneal or fallopian tube carcinoma are prospectively recruited to the Malaysia-wide population-based MaGiC Observational Study. DNA were tested using a Hi-Plex next generation sequencing method and multiplex ligation-dependent probe amplification to detect &lt; 10 bp alterations and exon deletions or duplications in the BRCA1 and BRCA2 genes. Results: Interim results from 325 patients tested until March 2018 have identified BRCA1 and BRCA2 pathogenic variants in 9.8% (32/325) and 3.1% (10/325) patients, respectively. Variants of uncertain significance were detected in 13.2% (43/325) patients and no pathogenic variants were detected in 73.8% (240/325) patients. Taken together, the frequency of BRCA1/2 pathogenic variants in ovarian cancer patients is approximately 12.9% (42/325). Conclusion: The identification of BRCA1 or BRCA2 carriers across the country have enabled the concentration of efforts from limited genetic counseling resources to high risk families. Results arising from the completion of this study will supplement cancer control programs and genetic services in Malaysia. 
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>sEst: Accurate Sex-Estimation and Abnormality Detection in Methylation Microarray Data</a>,  International Journal of Molecular Sciences,  2018.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Chol-Hee Jung, Daniel J. Park, Peter Georgeson, Khalid Mahmood, Roger L. Milne, Melissa C. Southey, Bernard J. Pope</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">International Journal of Molecular Sciences</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">19</dd>
            
                          <dt class="col-sm-2">Issue</dt>
              <dd class="col-sm-10">10</dd>
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2018</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.3390/ijms19103172</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">30326623</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="http://dx.doi.org/10.3390/ijms19103172">http://dx.doi.org/10.3390/ijms19103172</a></dd>
            
                          <dt class="col-sm-2">Keywords</dt>
              <dd class="col-sm-10">DNA methylation, sex information, sex-chromosome abnormalities, epigenetics</dd>
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">DNA methylation influences predisposition, development and prognosis for many diseases, including cancer. However, it is not uncommon to encounter samples with incorrect sex labelling or atypical sex chromosome arrangement. Sex is one of the strongest influencers of the genomic distribution of DNA methylation and, therefore, correct assignment of sex and filtering of abnormal samples are essential for the quality control of study data. Differences in sex chromosome copy numbers between sexes and X-chromosome inactivation in females result in distinctive sex-specific patterns in the distribution of DNA methylation levels. In this study, we present a software tool, sEst, which incorporates clustering analysis to infer sex and to detect sex-chromosome abnormalities from DNA methylation microarray data. Testing with two publicly available datasets demonstrated that sEst not only correctly inferred the sex of the test samples, but also identified mislabelled samples and samples with potential sex-chromosome abnormalities, such as Klinefelter syndrome and Turner syndrome, the latter being a feature not offered by existing methods. Considering that sex and the sex-chromosome abnormalities can have large effects on many phenotypes, including diseases, our method can make a significant contribution to DNA methylation studies that are based on microarray platforms.
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Is RNASEL:p.Glu265* a modifier of early-onset breast cancer risk for carriers of high-risk mutations?</a>,  BMC Cancer,  2018.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Tú Nguyen-Dumont, Zhi L. Teo, Fleur Hammet, Alexis Roberge, Maryam Mahmoodi, Helen Tsimiklis, Daniel J. Park, Bernard J. Pope, Andrew Lonie, Miroslav K. Kapuscinski, Khalid Mahmood, ABCFR, David E. Goldgar, Graham G. Giles, Ingrid Winship, John L. Hopper, Melissa C. Southey</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">BMC Cancer</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">18</dd>
            
                          <dt class="col-sm-2">Issue</dt>
              <dd class="col-sm-10">165</dd>
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2018</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1186/s12885-018-4028-z</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">29422015</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="http://dx.doi.org/10.1186/s12885-018-4028-z">http://dx.doi.org/10.1186/s12885-018-4028-z</a></dd>
            
                          <dt class="col-sm-2">Keywords</dt>
              <dd class="col-sm-10">RNASEL:P.Glu265*, Breast cancer, Modifier risk gene, Early-onset cancer</dd>
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10"><p>Background</p><p>Breast cancer risk for BRCA1 and BRCA2 pathogenic mutation carriers is modified by risk factors that cluster in families, including genetic modifiers of risk. We considered genetic modifiers of risk for carriers of high-risk mutations in other breast cancer susceptibility genes.</p><p>Methods</p><p>In a family known to carry the high-risk mutation PALB2:c.3113G>A (p.Trp1038*), whole-exome sequencing was performed on germline DNA from four affected women, three of whom were mutation carriers.</p><p>Results</p><p>RNASEL:p.Glu265* was identified in one of the PALB2 carriers who had two primary invasive breast cancer diagnoses before 50 years. Gene-panel testing of BRCA1, BRCA2, PALB2 and RNASEL in the Australian Breast Cancer Family Registry identified five carriers of RNASEL:p.Glu265* in 591 early onset breast cancer cases. Three of the five women (60%) carrying RNASEL:p.Glu265* also carried a pathogenic mutation in a breast cancer susceptibility gene compared with 30 carriers of pathogenic mutations in the 586 non-carriers of RNASEL:p.Glu265* (5%) (p < 0.002). Taqman genotyping demonstrated that the allele frequency of RNASEL:p.Glu265* was similar in affected and unaffected Australian women, consistent with other populations.</p><p>Conclusion</p><p>Our study suggests that RNASEL:p.Glu265* may be a genetic modifier of risk for early-onset breast cancer predisposition in carriers of high-risk mutations. Much larger case-case and case-control studies are warranted to test the association observed in this report.</p>
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>FANCM and RECQL genetic variants and breast cancer susceptibility: relevance to South Poland and West Ukraine</a>,  BMC Medical Genetics,  2018.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">T Nguyen-Dumont, A Myszka, P Karpinski, M Sasiadek, H Akopyan, F Hammet, H Tsimiklis, D Park, B Pope, R Slezak, N Kitsera, A Siekierzynska, M Southey</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">BMC Medical Genetics</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">19</dd>
            
                          <dt class="col-sm-2">Issue</dt>
              <dd class="col-sm-10">12</dd>
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2018</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1186/s12881-018-0524-x</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">29351780</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="http://dx.doi.org/10.1186/s12881-018-0524-x">http://dx.doi.org/10.1186/s12881-018-0524-x</a></dd>
            
                          <dt class="col-sm-2">Keywords</dt>
              <dd class="col-sm-10">FANCM, RECQL, Breast cancer predisposition, Familial breast cancer, Gene panel testing</dd>
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">Background. FANCM and RECQL have recently been reported as breast cancer susceptibility genes and it has been suggested that they should be included on gene panel tests for breast cancer predisposition. However, the clinical value of testing for mutations in RECQL and FANCM remains to be determined. In this study, we have characterised the spectrum of FANCM and RECQL mutations in women affected with breast or ovarian cancer from South-West Poland and West Ukraine. Methods. We applied Hi-Plex, an amplicon-based enrichment method for targeted massively parallel sequencing, to screen the coding exons and proximal intron-exon junctions of FANCM and RECQL in germline DNA from unrelated women affected with breast cancer (n = 338) and ovarian cancer (n = 89) from Poland (n = 304) and Ukraine (n = 123). These women were at high-risk of carrying a genetic predisposition to breast and/or ovarian cancer due to a family history and/or early-onset disease. Results. Among 427 women screened, we identified one carrier of the FANCM:c.1972C > T nonsense mutation (0.23%), and two carriers of the frameshift insertion FANCM:c.1491dup (0.47%). None of the variants we observed in RECQL were predicted to be loss-of-function mutations by standard variant effect prediction tools. Conclusions. Our study of the Polish and Ukrainian populations has identified a carrier frequency of truncating mutations in FANCM consistent with previous reports. Although initial reports suggesting that mutations in RECQL could be associated with increased breast cancer risk included women from Poland and identified the RECQL:c.1667_1667 + 3delAGTA mutation in 0.23–0.35% of breast cancer cases, we did not observe any carriers in our study cohort. Continued screening, both in research and diagnostic settings, will enable the accumulation of data that is needed to establish the clinical utility of including RECQL and FANCM on gene panel tests.
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Hi-Plex for Simple, Accurate, and Cost-Effective Amplicon-based Targeted DNA Sequencing</a>,  Methods in Molecular Biology,  2018.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Pope B, Hammet, Nguyen-Dumont, Park D</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">Methods in Molecular Biology</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">1712</dd>
            
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2018</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1007/978-1-4939-7514-3_5</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">29224068</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="http://dx.doi.org/10.1007/978-1-4939-7514-3_5">http://dx.doi.org/10.1007/978-1-4939-7514-3_5</a></dd>
            
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">Hi-Plex is a suite of methods to enable simple, accurate, and cost-effective highly multiplex PCR-based targeted sequencing (Nguyen-Dumont et al., Biotechniques 58:33-36, 2015). At its core is the principle of using gene-specific primers (GSPs) to "seed" (or target) the reaction and universal primers to "drive" the majority of the reaction. In this manner, effects on amplification efficiencies across the target amplicons can, to a large extent, be restricted to early seeding cycles. Product sizes are defined within a relatively narrow range to enable high-specificity size selection, replication uniformity across target sites (including in the context of fragmented input DNA such as that derived from fixed tumor specimens (Nguyen-Dumont et al., Biotechniques 55:69-74, 2013; Nguyen-Dumont et al., Anal Biochem 470:48-51, 2015), and application of high-specificity genetic variant calling algorithms (Pope et al., Source Code Biol Med 9:3, 2014; Park et al., BMC Bioinformatics 17:165, 2016). Hi-Plex offers a streamlined workflow that is suitable for testing large numbers of specimens without the need for automation.
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Double somatic mutations as a cause of tumor mismatch repair-deficiency in population-based colorectal and endometrial cancer with Lynch-like syndrome</a>,  Cancer Research,  2017.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Buchanan DD, Clendenning M, Jayasekara H, Joo JE, Wong EM, Southey MC, Walters RJ, Pope BJ, Win AK, Hopper JL, Jenkins MA, Milne RL, Giles GG, English DR, Macrae FA, Spurdle AB, Winship IM, Rosty C</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">Cancer Research</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">77</dd>
            
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2017</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1158/1538-7445.AM2017-4266</dd>
            
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="http://dx.doi.org/10.1158/1538-7445.AM2017-4266">http://dx.doi.org/10.1158/1538-7445.AM2017-4266</a></dd>
            
                          <dt class="col-sm-2">Keywords</dt>
              <dd class="col-sm-10">Science & Technology, Life Sciences & Biomedicine, Oncology</dd>
            
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Risk of colorectal cancer for carriers of a germline mutation in POLE or POLD1</a>,  Genetics in Medicine,  2017.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Buchanan D, Stewart J, Clendenning M, Rosty C, Mahmood K, Pope B, Jenkins M, Hopper J, Southey M, Macrae F, Winship M, Win A</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">Genetics in Medicine</dd>
            
            
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2017</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1038/gim.2017.185</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">29120461</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="http://dx.doi.org/10.1038/gim.2017.185">http://dx.doi.org/10.1038/gim.2017.185</a></dd>
            
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">Background: Germline mutations in the exonuclease domains of the POLE and POLD1 genes are associated with an as yet unquantified increased risk of colorectal cancer (CRC). Methods: We identified families with POLE or POLD1 variants by searching PubMed for relevant studies prior to October 2016 and by genotyping 669 population-based CRC cases diagnosed <60 years of age from the Australasian Colorectal Cancer Family Registry. We estimated the age-specific cumulative risks (penetrance) using a modified segregation analysis. Results: We observed 67 CRCs (mean age at diagnosis=50.2 (standard deviation [SD]=13.8) years) among 364 first- and second- degree relatives from 41 POLE families and 6 CRCs (mean age at diagnosis=39.7 (SD=6.83) years) among 69 relatives from 9 POLD1 families. We estimated risks of CRC to age 70 years (95% confidence interval [CI]) for males and females, respectively, to be: 40%(26%–57%) and 32%(20%–47%) for POLE mutation carriers; and 63%(15%–99%) and 52%(11%–99%) for POLD1 mutation carriers. Conclusion: CRC risks for POLE mutation carriers are sufficiently high warranting consideration of annual colonoscopy screening and management guidelines comparable to Lynch syndrome. Refinement of estimates of CRC risk for POLD1 carriers is needed, however, clinical management recommendations could follow those suggested for POLE carriers.
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Targeted massively parallel sequencing characterises the mutation spectrum of PALB2 in breast and ovarian cancer cases from Poland and Ukraine</a>,  Familial Cancer,  2017.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Myszka A, Nguyen-Dumont T, Karpinski P, Sasiadek MM, Akopyan H, Hammet F, Tsimiklis H, Park DJ, Pope BJ, Slezak R, Kitsera N, Siekierzynska A, Southey MC</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">Familial Cancer</dd>
            
            
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2017</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1007/s10689-017-0050-6</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">29052111</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="http://dx.doi.org/10.1007/s10689-017-0050-6">http://dx.doi.org/10.1007/s10689-017-0050-6</a></dd>
            
                          <dt class="col-sm-2">Keywords</dt>
              <dd class="col-sm-10">PALB2; Breast cancer; Ovarian cancer; Genetic susceptibility; Massively parallel sequencing</dd>
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">Loss-of-function germline mutations in the PALB2 gene are associated with an increase of breast cancer risk. The purpose of this study was to characterise the spectrum of PALB2 mutations in women affected with breast or ovarian cancer from South-West Poland and West Ukraine. We applied Hi-Plex, an amplicon-based enrichment method for targeted massively parallel sequencing, to screen the coding exons and proximal intron–exon junctions of PALB2 in germline DNA from unrelated women affected with breast cancer (n = 338) and ovarian cancer (n = 89) from Poland (n = 304) and Ukraine (n = 123). These women were at high-risk of carrying a genetic predisposition to breast and/or ovarian cancer due to a family history and/or early-onset disease. Targeted-sequencing identified two frameshift deletions: PALB2:c.509_510del; p.R170Ifs in three women affected with breast cancer and PALB2:c.172_175del;p.Q60Rfs in one woman affected with ovarian cancer. A number of other previously described missense (some predicted to be damaging by PolyPhen-2 and CADD) and synonymous mutations were also identified in this population. This study is consistent with previous reports that PALB2:c.509_510del and PALB2:c.172_175del are recurrent mutations associated with breast cancer predisposition in Polish women with a family history of the disease. Our study contributes to the accumulating evidence indicating that PALB2 should be included in genetic testing for breast cancer susceptibility in these populations to enhance risk assessment and management of women at high-risk of developing breast cancer. This data could also contribute to ongoing work that is assessing the possible association between ovarian cancer risk and PALB2 mutations for which there is currently no evidence.
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>A novel Drosophila injury model reveals severed axons are cleared through a Draper/MMP-1 signaling cascade</a>,  eLife,  2017.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Purice MD, Ray A, Münzel EJ, Pope BJ, Park DJ, Speese SD, Logan MA</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">eLife</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">6</dd>
            
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2017</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.7554/eLife.23611</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">28825401</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="http://dx.doi.org/10.7554/eLife.23611">http://dx.doi.org/10.7554/eLife.23611</a></dd>
            
                          <dt class="col-sm-2">Keywords</dt>
              <dd class="col-sm-10">glial immunity; Wallerian degeneration; extracellular matrix; Draper; matrix metalloproteinase</dd>
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10"><p>Neural injury triggers swift responses from glia, including glial migration and phagocytic clearance of damaged neurons. The transcriptional programs governing these complex innate glial immune responses are still unclear. Here, we describe a novel injury assay in adult Drosophila that elicits widespread glial responses in the ventral nerve cord (VNC). We profiled injury-induced changes in VNC gene expression by RNA sequencing (RNA-seq) and found that responsive genes fall into diverse signaling classes. One factor, matrix metalloproteinase-1 (MMP-1), is induced in Drosophila ensheathing glia responding to severed axons. Interestingly, glial induction of MMP-1 requires the highly conserved engulfment receptor Draper, as well as AP-1 and STAT92E. In MMP-1 depleted flies, glia do not properly infiltrate neuropil regions after axotomy and, as a consequence, fail to clear degenerating axonal debris. This work identifies Draper-dependent activation of MMP-1 as a novel cascade required for proper glial clearance of severed axons.</p>
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Somatic mutations of the coding microsatellites within the beta-2-microglobulin gene in mismatch repair-deficient colorectal cancers and adenomas</a>,  Familial Cancer,  2017.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Clendenning M, Huang A, Jayasekara H, Lorans M, Preston S, O'Callaghan N, Pope BJ, Macrae FA, Winship IM, Milne RL, Giles GG, English DR, Hopper JL, Win AK, Jenkins MA, Southey MC, Rosty C, Buchanan DD, investigators from the Melbourne Collaborative Cohort Study and the Australasian Colorectal Cancer Family Registry Cohort</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">Familial Cancer</dd>
            
            
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2017</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1007/s10689-017-0013-y</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">28616688</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="http://dx.doi.org/10.1007/s10689-017-0013-y">http://dx.doi.org/10.1007/s10689-017-0013-y</a></dd>
            
                          <dt class="col-sm-2">Keywords</dt>
              <dd class="col-sm-10">B2M; Colorectal cancer; Lynch syndrome; MLH1 methylation; Microsatellite instability; Mismatch repair deficiency</dd>
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">In colorectal cancers (CRCs) with tumour mismatch repair (MMR) deficiency, genes involved in the host immune response that contain microsatellites in their coding regions, including beta-2-microglobulin (B2M), can acquire mutations that may alter the immune response, tumour progression and prognosis. We screened the coding microsatellites within B2M for somatic mutations in MMR-deficient CRCs and adenomas to determine associations with tumour subtypes, clinicopathological features and survival. Incident MMR-deficient CRCs from Australasian Colorectal Cancer Family Registry (ACCFR) and the Melbourne Collaborative Cohort Study participants (n = 144) and 63 adenomas from 41 MMR gene mutation carriers from the ACCFR were screened for somatic mutations within five coding microsatellites of B2M. Hazard ratios (HR) and 95% confidence intervals (CI) for overall survival by B2M mutation status were estimated using Cox regression, adjusting for age at CRC diagnosis, sex, AJCC stage and grade. B2M mutations occurred in 30 (20.8%) of the 144 MMR-deficient CRCs (29% of the MLH1-methylated, 17% of the Lynch syndrome and 9% of the suspected Lynch CRCs). No B2M mutations were identified in the 63 adenomas tested. B2M mutations differed by site, stage, grade and lymphocytic infiltration although none reached statistical significance (p > 0.05). The HR for overall survival for B2M mutated CRC was 0.65 (95% CI 0.29-1.48) compared with B2M wild-type. We observed differences in B2M mutation status in MMR-deficient CRC by tumour subtypes, site, stage, grade, immune infiltrate and for overall survival that warrant further investigation in larger studies before B2M mutation status can be considered to have clinical utility.</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Four simple recommendations to encourage best practices in research software</a>,  F1000 Research,  2017.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Rafael C. Jiménez, Mateusz Kuzak, Monther Alhamdoosh, Michelle Barker, Bérénice Batut, Mikael Borg, Salvador Capella-Gutierrez, Neil Chue Hong, Martin Cook, Manuel Corpas, Madison Flannery, Leyla Garcia, Josep Ll. Gelpí, Simon Gladman, Carole Goble, Montserrat González Ferreiro, Alejandra Gonzalez-Beltran, Philippa C. Griffin, Björn Grüning, Jonas Hagberg, Petr Holub, Rob Hooft, Jon Ison, Daniel S. Katz, Brane Leskošek, Federico López Gómez, Luis J. Oliveira, David Mellor, Rowland Mosbergen, Nicola Mulder, Yasset Perez-Riverol, Robert Pergl, Horst Pichler, Bernard Pope, Ferran Sanz, Maria V. Schneider, Victoria Stodden, Radosław Suchecki, Radka Svobodová Vařeková, Harry-Anton Talvik, Ilian Todorov, Andrew Treloar, Sonika Tyagi, Maarten van Gompel, Daniel Vaughan, Allegra Via, Xiaochuan Wang, Nathan S. Watson-Haigh, Steve Crouch</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">F1000 Research</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">13</dd>
            
                          <dt class="col-sm-2">Issue</dt>
              <dd class="col-sm-10">6</dd>
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2017</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.12688/f1000research.11407.1</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">28751965</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="http://dx.doi.org/10.12688/f1000research.11407.1">http://dx.doi.org/10.12688/f1000research.11407.1</a></dd>
            
                          <dt class="col-sm-2">Keywords</dt>
              <dd class="col-sm-10">FAIR; Open Science; Open Source; best practices; code; guidelines; quality; recommendations; software; sustainability</dd>
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">Scientific research relies on computer software, yet software is not always developed following practices that ensure its quality and sustainability. This manuscript does not aim to propose new software development best practices, but rather to provide simple recommendations that encourage the adoption of existing best practices. Software development best practices promote better quality software, and better quality software improves the reproducibility and reusability of research. These recommendations are designed around Open Source values, and provide practical suggestions that contribute to making research software and its source code more discoverable, reusable and transparent. This manuscript is aimed at developers, but also at organisations, projects, journals and funders that can increase the quality and sustainability of research software by encouraging the adoption of these recommendations.
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Best practice data life cycle approaches for the life sciences</a>,  F1000 Research,  2017.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Griffin PC, Khadake J, LeMay KS, Lewis SE, Orchard S, Pask A, Pope B, Roessner U, Russell K, Seemann T, Treloar A, Tyagi S, Christiansen JH, Dayalan S, Gladman S, Hangartner SB, Hayden HL, Ho WWH, Keeble-Gagnère G, Korhonen PK, Neish P, Prestes PR, Richardson MF, Watson-Haigh NS, Wyres KL, Young ND, Schneider MV</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">F1000 Research</dd>
            
            
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2017</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.12688/f1000research.12344.1</dd>
            
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="http://dx.doi.org/10.12688/f1000research.12344.1">http://dx.doi.org/10.12688/f1000research.12344.1</a></dd>
            
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10"><p>Throughout history, the life sciences have been revolutionised by technological advances; in our era this is manifested by advances in instrumentation for data generation, and consequently researchers now routinely handle large amounts of heterogeneous data in digital formats. The simultaneous transitions towards biology as a data science and towards a ‘life cycle’ view of research data pose new challenges. Researchers face a bewildering landscape of data management requirements, recommendations and regulations, without necessarily being able to access data management training or possessing a clear understanding of practical approaches that can assist in data management in their particular research domain.</p><p>Here we provide an overview of best practice data life cycle approaches for researchers in the life sciences/bioinformatics space with a particular focus on ‘omics’ datasets and computer-based data processing and analysis. We discuss the different stages of the data life cycle and provide practical suggestions for useful tools and resources to improve data management practices.</p>
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Variant effect prediction tools assessed using independent, functional assay-based datasets: implications for discovery and diagnostics</a>,  Human Genomics,  2017.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Mahmood K, Jung CH, Philip G, Georgeson P, Chung J, Pope BJ, Park DJ</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">Human Genomics</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">11</dd>
            
                          <dt class="col-sm-2">Issue</dt>
              <dd class="col-sm-10">1</dd>
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2017</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1186/s40246-017-0104-8</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">28511696</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="http://dx.doi.org/10.1186/s40246-017-0104-8">http://dx.doi.org/10.1186/s40246-017-0104-8</a></dd>
            
                          <dt class="col-sm-2">Keywords</dt>
              <dd class="col-sm-10">Benchmarking; Functional assays; Functional datasets; Genomic screening; Mutation assessment; Pathogenicity prediction; Protein function; Variant effect prediction</dd>
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">BACKGROUND: Genetic variant effect prediction algorithms are used extensively in clinical genomics and research to determine the likely consequences of amino acid substitutions on protein function. It is vital that we better understand their accuracies and limitations because published performance metrics are confounded by serious problems of circularity and error propagation. Here, we derive three independent, functionally determined human mutation datasets, UniFun, BRCA1-DMS and TP53-TA, and employ them, alongside previously described datasets, to assess the pre-eminent variant effect prediction tools. RESULTS: Apparent accuracies of variant effect prediction tools were influenced significantly by the benchmarking dataset. Benchmarking with the assay-determined datasets UniFun and BRCA1-DMS yielded areas under the receiver operating characteristic curves in the modest ranges of 0.52 to 0.63 and 0.54 to 0.75, respectively, considerably lower than observed for other, potentially more conflicted datasets. CONCLUSIONS: These results raise concerns about how such algorithms should be employed, particularly in a clinical setting. Contemporary variant effect prediction tools are unlikely to be as accurate at the general prediction of functional impacts on proteins as reported prior. Use of functional assay-based datasets that avoid prior dependencies promises to be valuable for the ongoing development and accurate benchmarking of such tools.</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Mutation screening of ACKR3 and COPS8 in kidney cancer cases from the CONFIRM study</a>,  Familial Cancer,  2017.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Mahmoodi M, Nguyen-Dumont T, Hammet F, Pope BJ, Park DJ, Southey MC, Darlow JM, Bruinsma F, Winship I</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">Familial Cancer</dd>
            
            
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2017</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1007/s10689-016-9961-x</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">28063109</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="http://dx.doi.org/10.1007/s10689-016-9961-x">http://dx.doi.org/10.1007/s10689-016-9961-x</a></dd>
            
                          <dt class="col-sm-2">Keywords</dt>
              <dd class="col-sm-10">ACKR3; COPS8; Hi-Plex; Kidney cancer; Massively parallel sequencing; Mutation screening</dd>
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">An apparently balanced t(2;3)(q37.3;q13.2) translocation that appears to segregate with renal cell carcinoma (RCC) has indicated potential areas to search for the elusive genetic basis of clear cell RCC. We applied Hi-Plex targeted sequencing to analyse germline DNA from 479 individuals affected with clear cell RCC for this breakpoint translocation and genetic variants in neighbouring genes on chromosome 2, ACKR3 and COPS8. While only synonymous variants were found in COPS8, one of the missense variants in ACKR3:c.892C>T, observed in 4/479 individuals screened (0.8%), was predicted likely to damage ACKR3 function. Identification of causal genes for RCC has potential clinical utility, where risk assessment and risk management can offer better outcomes, with surveillance for at-risk relatives and nephron sparing surgery through earlier intervention.</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Single nucleotide-level mapping of DNA double-strand breaks in human HEK293T cells</a>,  Genomics Data,  2017.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Pope BJ, Mahmood K, Jung CH, Georgeson P, Park DJ</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">Genomics Data</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">11</dd>
            
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2017</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1016/j.gdata.2016.11.007</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">27942458</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="http://dx.doi.org/10.1016/j.gdata.2016.11.007">http://dx.doi.org/10.1016/j.gdata.2016.11.007</a></dd>
            
                          <dt class="col-sm-2">Keywords</dt>
              <dd class="col-sm-10">Double-strand breaks; Fragile sites; Human genome; Forum domains; HEK293T</dd>
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">Constitutional biological processes involve the generation of DNA double-strand breaks (DSBs). The production of such breaks and their subsequent resolution are also highly relevant to neurodegenerative diseases and cancer, in which extensive DNA fragmentation has been described Stephens et al. (2011), Blondet et al. (2001). Tchurikov et al. Tchurikov et al. (2011, 2013) have reported previously that frequent sites of DSBs occur in chromosomal domains involved in the co-ordinated expression of genes. This group report that hot spots of DSBs in human HEK293T cells often coincide with H3K4me3 marks, associated with active transcription Kravatsky et al. (2015) and that frequent sites of DNA double-strand breakage are likely to be relevant to cancer genomics Tchurikov et al. (2013, 2016) . Recently, they applied a RAFT (rapid amplification of forum termini) protocol that selects for blunt-ended DSB sites and mapped these to the human genome within defined co-ordinate 'windows'. In this paper, we re-analyse public RAFT data to derive sites of DSBs at the single-nucleotide level across the built genome for human HEK293T cells (https://figshare.com/s/35220b2b79eaaaf64ed8). This refined mapping, combined with accessory ENCODE data tracks and ribosomal DNA-related sequence annotations, will likely be of value for the design of clinically relevant targeted assays such as those for cancer susceptibility, diagnosis, treatment-matching and prognostication.</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>MethPat: a tool for the analysis and visualisation of complex methylation patterns obtained by massively parallel sequencing</a>,  BMC Bioinformatics,  2016.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Wong NC, Pope BJ, Candiloro IL, Korbie D, Trau M, Wong SQ, Mikeska T, Zhang X, Pitman M, Eggers S, Doyle SR, Dobrovic A</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">BMC Bioinformatics</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">17</dd>
            
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2016</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1186/s12859-016-0950-8</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">26911705</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="http://dx.doi.org/10.1186/s12859-016-0950-8">http://dx.doi.org/10.1186/s12859-016-0950-8</a></dd>
            
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">BACKGROUND: DNA methylation at a gene promoter region has the potential to regulate gene transcription. Patterns of methylation over multiple CpG sites in a region are often complex and cell type specific, with the region showing multiple allelic patterns in a sample. This complexity is commonly obscured when DNA methylation data is summarised as an average percentage value for each CpG site (or aggregated across CpG sites). True representation of methylation patterns can only be fully characterised by clonal analysis. Deep sequencing provides the ability to investigate clonal DNA methylation patterns in unprecedented detail and scale, enabling the proper characterisation of the heterogeneity of methylation patterns. However, the sheer amount and complexity of sequencing data requires new synoptic approaches to visualise the distribution of allelic patterns. RESULTS: We have developed a new analysis and visualisation software tool Methpat.</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Fine resolution mapping of double-strand break sites for human ribosomal DNA units</a>,  Genomics Data,  2016.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Pope BJ, Mahmood K, Jung CH, Park DJ</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">Genomics Data</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">10</dd>
            
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2016</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1016/j.gdata.2016.08.012</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">27656414</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="http://dx.doi.org/10.1016/j.gdata.2016.08.012">http://dx.doi.org/10.1016/j.gdata.2016.08.012</a></dd>
            
                          <dt class="col-sm-2">Keywords</dt>
              <dd class="col-sm-10">Double-strand breaks; Forum domains; Fragile sites; HEK293T; rDNA</dd>
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">DNA breakage arises during a variety of biological processes, including transcription, replication and genome rearrangements. In the context of disease, extensive fragmentation of DNA has been described in cancer cells and during early stages of neurodegeneration (Stephens et al., 2011 Stephens et al. (2011) [5]; Blondet et al., 2001 Blondet et al. (2001) [1]). Stults et al. (2009) Stults et al. (2009) [6] reported that human rDNA gene clusters are hotspots for recombination and that rDNA restructuring is among the most common chromosomal alterations in adult solid tumours. As such, analysis of rDNA regions is likely to have significant prognostic and predictive value, clinically. Tchurikov et al. (2015a, 2016) Tchurikov et al. (2015a, 2016) [7], [9] have made major advances in this direction, reporting that sites of human genome double-strand breaks (DSBs) occur frequently at sites in rDNA that are tightly linked with active transcription - the authors used a RAFT (rapid amplification of forum termini) protocol that selects for blunt-ended sites. They reported the relative frequency of these rDNA DSBs within defined co-ordinate 'windows' of varying size and made these data (as well as the relevant 'raw' sequencing information) available to the public (Tchurikov et al., 2015b). Assay designs targeting rDNA DSB hotspots will benefit greatly from the publication of break sites at greater resolution. Here, we re-analyse public RAFT data and make available rDNA DSB co-ordinates to the single-nucleotide level.</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>UNDR ROVER - a fast and accurate variant caller for targeted DNA sequencing</a>,  BMC Bioinformatics,  2016.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Park DJ, Li R, Lau E, Georgeson P, Nguyen-Dumont T, Pope BJ</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">BMC Bioinformatics</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">17</dd>
            
                          <dt class="col-sm-2">Issue</dt>
              <dd class="col-sm-10">1</dd>
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2016</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1186/s12859-016-1014-9</dd>
            
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="http://dx.doi.org/10.1186/s12859-016-1014-9">http://dx.doi.org/10.1186/s12859-016-1014-9</a></dd>
            
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10"><p>Background:Previously, we described ROVER, a DNA variant caller which identifies genetic variants from PCR-targeted massively parallel sequencing (MPS) datasets generated by the Hi-Plex protocol. ROVER permits stringent filtering of sequencing chemistry-induced errors by requiring reported variants to appear in both reads of overlapping pairs above certain thresholds of occurrence. ROVER was developed in tandem with Hi-Plex and has been used successfully to screen for genetic mutations in the breast cancer predisposition gene PALB2.</p><p>ROVER is applied to MPS data in BAM format and, therefore, relies on sequence reads being mapped to a reference genome. In this paper, we describe an improvement to ROVER, called UNDR ROVER (Unmapped primer-Directed ROVER), which accepts MPS data in FASTQ format, avoiding the need for a computationally expensive mapping stage. It does so by taking advantage of the location-specific nature of PCR-targeted MPS data.</p><p>Results: The UNDR ROVER algorithm achieves the same stringent variant calling as its predecessor with a significant runtime performance improvement. In one indicative sequencing experiment, UNDR ROVER (in its fastest mode) required 8-fold less sequential computation time than the ROVER pipeline and 13-fold less sequential computation time than a variant calling pipeline based on the popular GATK tool.</p><p>UNDR ROVER is implemented in Python and runs on all popular POSIX-like operating systems (Linux, OS X). It requires as input a tab-delimited format file containing primer sequence information, a FASTA format file containing the reference genome sequence, and paired FASTQ files containing sequence reads. Primer sequences at the 5′ end of reads associate read-pairs with their targeted amplicon and, thus, their expected corresponding coordinates in the reference genome. The primer-intervening sequence of each read is compared against the reference sequence from the same location and variants are identified using the same algorithm as ROVER. Specifically, for a variant to be ‘called’ it must appear at the same location in both of the overlapping reads above user-defined thresholds of minimum number of reads and proportion of reads.</p><p>Conclusions: UNDR ROVER provides the same rapid and accurate genetic variant calling as its predecessor with greatly reduced computational costs.</p>
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>High-resolution twin-ion metabolite extraction (HiTIME) mass spectrometry: nontargeted detection of unknown drug metabolites by isotope labeling, liquid chromatography mass spectrometry, and automated high-performance computing</a>,  Analytical Chemistry,  2015.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Leeming MG, Isaac AP, Pope BJ, Cranswick N, Wright CE, Ziogas J, O'Hair RA, Donald WA</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">Analytical Chemistry</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">87</dd>
            
                          <dt class="col-sm-2">Issue</dt>
              <dd class="col-sm-10">8</dd>
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2015</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1021/ac504767d</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">25818563</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="http://dx.doi.org/10.1021/ac504767d">http://dx.doi.org/10.1021/ac504767d</a></dd>
            
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">'The metabolic fate of a compound can often determine the success of a new drug lead. Thus, significant effort is directed toward identifying the metabolites formed from a given molecule. Here, an automated and nontargeted procedure is introduced for detecting drug metabolites without authentic metabolite standards via the use of stable isotope labeling, liquid chromatography mass spectrometry (LC/MS), and high-performance computing. LC/MS of blood plasma extracts from rats that were administered a 1:1 mixture of acetaminophen (APAP) and 13C6-APAP resulted in mass spectra that contained “twin” ions for drug metabolites that were not detected in control spectra (i.e., no APAP administered). Because of the development of a program (high-resolution twin-ion metabolite extraction; HiTIME) that can identify twin-ions in high-resolution mass spectra without centroiding (i.e., reduction of mass spectral peaks to single data points), 9 doublets corresponding to APAP metabolites were identified. This is nearly twice that obtained by use of existing programs that make use of centroiding to reduce computational cost under these conditions with a quadrupole time-of-flight mass spectrometer. By a manual search for all reported APAP metabolite ions, no additional twin-ion signals were assigned. These data indicate that all the major metabolites of APAP and multiple low-abundance metabolites (e.g., acetaminophen hydroxy- and methoxysulfate) that are rarely reported were detected. This methodology can be used to detect drug metabolites without prior knowledge of their identity. HiTIME is freely available from <a href="https://github.com/bjpop/HiTIME">https://github.com/bjpop/HiTIME</a>'
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Exemplary multiplex bisulfite amplicon data used to demonstrate the utility of Methpat</a>,  GigaScience,  2015.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Wong NC, Pope BJ, Candiloro I, Korbie D, Trau M, Wong SQ, Mikeska T, van Denderen BJ, Thompson EW, Eggers S, Doyle SR, Dobrovic A</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">GigaScience</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">4</dd>
            
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2015</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1186/s13742-015-0098-x</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">26613017</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="http://dx.doi.org/10.1186/s13742-015-0098-x">http://dx.doi.org/10.1186/s13742-015-0098-x</a></dd>
            
                          <dt class="col-sm-2">Keywords</dt>
              <dd class="col-sm-10">Bisulfite sequencing; Cancer; DNA methylation; Epialleles; Epigenetics; PCR; Visualization</dd>
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">BACKGROUND: DNA methylation is a complex epigenetic marker that can be analyzed using a wide variety of methods. Interpretation and visualization of DNA methylation data can mask complexity in terms of methylation status at each CpG site, cellular heterogeneity of samples and allelic DNA methylation patterns within a given DNA strand. Bisulfite sequencing is considered the gold standard, but visualization of massively parallel sequencing results remains a significant challenge. FINDINGS: We created a program called Methpat that facilitates visualization and interpretation of bisulfite sequencing data generated by massively parallel sequencing. To demonstrate this, we performed multiplex PCR that targeted 48 regions of interest across 86 human samples. The regions selected included known gene promoters associated with cancer, repetitive elements, known imprinted regions and mitochondrial genomic sequences. We interrogated a range of samples including human cell lines, primary tumours and primary tissue samples. Methpat generates two forms of output: a tab-delimited text file for each sample that summarizes DNA methylation patterns and their read counts for each amplicon, and a HTML file that summarizes this data visually. Methpat can be used with publicly available whole genome bisulfite sequencing and reduced representation bisulfite sequencing datasets with sufficient read depths. CONCLUSIONS: Using Methpat, complex DNA methylation data derived from massively parallel sequencing can be summarized and visualized for biological interpretation. By accounting for allelic DNA methylation states and their abundance in a sample, Methpat can unmask the complexity of DNA methylation and yield further biological insight in existing datasets.</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Abridged adapter primers increase the target scope of Hi-Plex</a>,  BioTechniques,  2015.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Nguyen-Dumont T, Hammet F, Mahmoodi M, Pope BJ, Giles GG, Hopper JL, Southey MC, Park DJ</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">BioTechniques</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">58</dd>
            
                          <dt class="col-sm-2">Issue</dt>
              <dd class="col-sm-10">1</dd>
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2015</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.2144/000114247</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">25605578</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="http://dx.doi.org/10.2144/000114247">http://dx.doi.org/10.2144/000114247</a></dd>
            
                          <dt class="col-sm-2">Keywords</dt>
              <dd class="col-sm-10">Hi-Plex; genetic screening; massively parallel sequencing; mutation screening; next-generation sequencing; targeted sequencing</dd>
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">Previously, we reported Hi-Plex, an amplicon-based method for targeted massively parallel sequencing capable of generating 60 amplicons simultaneously. In further experiments, however, we found our approach did not scale to higher amplicon numbers. Here, we report a modification to the original Hi-Plex protocol that includes the use of abridged adapter oligonucleotides as universal primers (bridge primers) in the initial PCR mixture. Full-length adapter primers (indexing primers) are included only during latter stages of thermal cycling with concomitant application of elevated annealing temperatures. Using this approach, we demonstrate the application of Hi-Plex across a broad range of amplicon numbers (16-plex, 62-plex, 250-plex, and 1003-plex) while preserving the low amount (25 ng) of input DNA required.</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Mutation screening of PALB2 in clinically ascertained families from the Breast Cancer Family Registry</a>,  Breast Cancer Research and Treatment,  2015.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Nguyen-Dumont T, Hammet F, Mahmoodi M, Tsimiklis H, Teo ZL, Li R, Pope BJ, Terry MB, Buys SS, Daly M, Hopper JL, Winship I, Goldgar DE, Park DJ, Southey MC</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">Breast Cancer Research and Treatment</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">149</dd>
            
                          <dt class="col-sm-2">Issue</dt>
              <dd class="col-sm-10">2</dd>
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2015</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1007/s10549-014-3260-8</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">25575445</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="http://dx.doi.org/10.1007/s10549-014-3260-8">http://dx.doi.org/10.1007/s10549-014-3260-8</a></dd>
            
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">Loss-of-function mutations in PALB2 are associated with an increased risk of breast cancer, with recent data showing that female breast cancer risks for PALB2 mutation carriers are comparable in magnitude to those for BRCA2 mutation carriers. This study applied targeted massively parallel sequencing to characterize the mutation spectrum of PALB2 in probands attending breast cancer genetics clinics in the USA. The coding regions and proximal intron-exon junctions of PALB2 were screened in probands not known to carry a mutation in BRCA1 or BCRA2 from 1,250 families enrolled through familial cancer clinics by the Breast Cancer Family Registry. Mutation screening was performed using Hi-Plex, an amplicon-based targeted massively parallel sequencing platform. Screening of PALB2 was successful in 1,240/1,250 probands and identified nine women with protein-truncating mutations (three nonsense mutations and five frameshift mutations). Four of the 33 missense variants were predicted to be deleterious to protein function by in silico analysis using two different programs. Analysis of tumors from carriers of truncating mutations revealed that the majority were high histological grade, invasive ductal carcinomas. Young onset was apparent in most families, with 19 breast cancers under 50 years of age, including eight under the age of 40 years. Our data demonstrate the utility of Hi-Plex in the context of high-throughput testing for rare genetic mutations and provide additional timely information about the nature and prevalence of PALB2 mutations, to enhance risk assessment and risk management of women at high risk of cancer attending clinical genetic services.</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>SRST2: Rapid genomic surveillance for public health and hospital microbiology labs</a>,  Genome Medicine,  2014.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Inouye M, Dashnow H, Raven LA, Schultz MB, Pope BJ, Tomita T, Zobel J, Holt KE</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">Genome Medicine</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">6</dd>
            
                          <dt class="col-sm-2">Issue</dt>
              <dd class="col-sm-10">11</dd>
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2014</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1186/s13073-014-0090-6</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">25422674</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="http://dx.doi.org/10.1186/s13073-014-0090-6">http://dx.doi.org/10.1186/s13073-014-0090-6</a></dd>
            
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">Rapid molecular typing of bacterial pathogens is critical for public health epidemiology, surveillance and infection control, yet routine use of whole genome sequencing (WGS) for these purposes poses significant challenges. Here we present SRST2, a read mapping-based tool for fast and accurate detection of genes, alleles and multi-locus sequence types (MLST) from WGS data. Using >900 genomes from common pathogens, we show SRST2 is highly accurate and outperforms assembly-based methods in terms of both gene detection and allele assignment. We include validation of SRST2 within a public health laboratory, and demonstrate its use for microbial genome surveillance in the hospital setting. In the face of rising threats of antimicrobial resistance and emerging virulence among bacterial pathogens, SRST2 represents a powerful tool for rapidly extracting clinically useful information from raw WGS data. Source code is available from http://katholt.github.io/srst2/.</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>ROVER variant caller: read-pair overlap considerate variant-calling software applied to PCR-based massively parallel sequencing datasets</a>,  Source Code for Biology and Medicine,  2014.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Pope BJ, Nguyen-Dumont T, Hammet F, Park DJ</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">Source Code for Biology and Medicine</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">9</dd>
            
                          <dt class="col-sm-2">Issue</dt>
              <dd class="col-sm-10">1</dd>
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2014</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1186/1751-0473-9-3</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">24461215</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="http://dx.doi.org/10.1186/1751-0473-9-3">http://dx.doi.org/10.1186/1751-0473-9-3</a></dd>
            
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">BACKGROUND: We recently described Hi-Plex, a highly multiplexed PCR-based target-enrichment system for massively parallel sequencing (MPS), which allows the uniform definition of library size so that subsequent paired-end sequencing can achieve complete overlap of read pairs. Variant calling from Hi-Plex-derived datasets can thus rely on the identification of variants appearing in both reads of read-pairs, permitting stringent filtering of sequencing chemistry-induced errors. These principles underly ROVER software (derived from Read Overlap PCR-MPS variant caller), which we have recently used to report the screening for genetic mutations in the breast cancer predisposition gene PALB2. Here, we describe the algorithms underlying ROVER and its usage. RESULTS: ROVER enables users to quickly and accurately identify genetic variants from PCR-targeted, overlapping paired-end MPS datasets. The open-source availability of the software and threshold tailorability enables broad access for a range of PCR-MPS users. METHODS: ROVER is implemented in Python and runs on all popular POSIX-like operating systems (Linux, OS X). The software accepts a tab-delimited text file listing the coordinates of the target-specific primers used for targeted enrichment based on a specified genome-build. It also accepts aligned sequence files resulting from mapping to the same genome-build. ROVER identifies the amplicon a given read-pair represents and removes the primer sequences by using the mapping co-ordinates and primer co-ordinates. It considers overlapping read-pairs with respect to primer-intervening sequence. Only when a variant is observed in both reads of a read-pair does the signal contribute to a tally of read-pairs containing or not containing the variant. A user-defined threshold informs the minimum number of, and proportion of, read-pairs a variant must be observed in for a 'call' to be made. ROVER also reports the depth of coverage across amplicons to facilitate the identification of any regions that may require further screening. CONCLUSIONS: ROVER can facilitate rapid and accurate genetic variant calling for a broad range of PCR-MPS users.</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Rare mutations in RINT1 predispose carriers to breast and Lynch syndrome-spectrum cancers</a>,  Cancer Discovery,  2014.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Park DJ, Tao K, Le Calvez-Kelm F, Nguyen-Dumont T, Robinot N, Hammet F, Odefrey F, Tsimiklis H, Teo ZL, Thingholm LB, Young EL, Voegele C, Lonie A, Pope BJ, Roane TC, Bell R, Hu H, Shankaracharya, Huff CD, Ellis J, Li J, Makunin IV, John EM, Andrulis IL, Terry MB, Daly M, Buys SS, Snyder C, Lynch HT, Devilee P, Giles GG, Hopper JL, Feng BJ, Lesueur F, Tavtigian SV, Southey MC, Goldgar DE</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">Cancer Discovery</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">4</dd>
            
                          <dt class="col-sm-2">Issue</dt>
              <dd class="col-sm-10">7</dd>
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2014</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1158/2159-8290.CD-14-0212</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">25050558</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="http://dx.doi.org/10.1158/2159-8290.CD-14-0212">http://dx.doi.org/10.1158/2159-8290.CD-14-0212</a></dd>
            
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">Approximately half of the familial aggregation of breast cancer remains unexplained. A multiple-case breast cancer family exome-sequencing study identified three likely pathogenic mutations in RINT1 (NM_021930.4) not present in public sequencing databases: RINT1 c.343C>T (p.Q115X), c.1132_1134del (p.M378del), and c.1207G>T (p.D403Y). On the basis of this finding, a population-based case-control mutation-screening study was conducted that identified 29 carriers of rare (minor allele frequency < 0.5%), likely pathogenic variants: 23 in 1,313 early-onset breast cancer cases and six in 1,123 frequency-matched controls [OR, 3.24; 95% confidence interval (CI), 1.29-8.17; P = 0.013]. RINT1 mutation screening of probands from 798 multiple-case breast cancer families identified four additional carriers of rare genetic variants. Analysis of the incidence of first primary cancers in families of women carrying RINT1 mutations estimated that carriers were at increased risk of Lynch syndrome-spectrum cancers [standardized incidence ratio (SIR), 3.35; 95% CI, 1.7-6.0; P = 0.005], particularly for relatives diagnosed with cancer under the age of 60 years (SIR, 10.9; 95% CI, 4.7-21; P = 0.0003). SIGNIFICANCE: The work described in this study adds RINT1 to the growing list of genes in which rare sequence variants are associated with intermediate levels of breast cancer risk. Given that RINT1 is also associated with a spectrum of cancers with mismatch repair defects, these findings have clinical applications and raise interesting biological questions.</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Annokey: an annotation tool based on key term search of the NCBI Entrez Gene database</a>,  Source Code for Biology and Medicine,  2014.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Park DJ, Nguyen-Dumont T, Kang S, Verspoor K, Pope BJ</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">Source Code for Biology and Medicine</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">9</dd>
            
                          <dt class="col-sm-2">Issue</dt>
              <dd class="col-sm-10">1</dd>
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2014</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1186/1751-0473-9-15</dd>
            
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="http://dx.doi.org/10.1186/1751-0473-9-15">http://dx.doi.org/10.1186/1751-0473-9-15</a></dd>
            
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10"><p>Background: The NCBI Entrez Gene and PubMed databases contain a wealth of high-quality information about genes for many different organisms. The NCBI Entrez online web-search interface is convenient for simple manual search for a small number of genes but impractical for the kinds of outputs seen in typical genomics projects.</p><p>Results: We have developed an efficient open source tool implemented in Python called Annokey, which annotates gene lists with the results of a keyword search of the NCBI Entrez Gene database and linked Pubmed article information. The user steers the search by specifying a ranked list of keywords (including multi-word phrases and regular expressions) that are correlated with their topic of interest. Rank information of matched terms allows the user to guide further investigation.</p><p>We applied Annokey to the entire human Entrez Gene database using the key-term “DNA repair” and assessed its performance in identifying the 176 members of a published “gold standard” list of genes established to be involved in this pathway. For this test case we observed a sensitivity and specificity of 97% and 96%, respectively.</p><p>Conclusions: Annokey facilitates the identification of genes related to an area of interest, a task which can be onerous if performed manually on a large number of genes. Annokey provides a way to capitalize on the high quality information provided by the Entrez Gene database allowing both scalability and compatibility with automated analysis pipelines, thus offering the potential to significantly enhance research productivity.</p>
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>MYRF is a membrane-associated transcription factor that autoproteolytically cleaves to directly activate myelin genes</a>,  PLoS Biology,  2013.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Bujalka H, Koenning M, Jackson S, Perreau VM, Pope B, Hay CM, Mitew S, Hill AF, Lu QR, Wegner M, Srinivasan R, Svaren J, Willingham M, Barres BA, Emery B</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">PLoS Biology</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">11</dd>
            
                          <dt class="col-sm-2">Issue</dt>
              <dd class="col-sm-10">8</dd>
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2013</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1371/journal.pbio.1001625</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">23966833</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="http://dx.doi.org/10.1371/journal.pbio.1001625">http://dx.doi.org/10.1371/journal.pbio.1001625</a></dd>
            
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">The myelination of axons is a crucial step during vertebrate central nervous system (CNS) development, allowing for rapid and energy efficient saltatory conduction of nerve impulses. Accordingly, the differentiation of oligodendrocytes, the myelinating cells of the CNS, and their expression of myelin genes are under tight transcriptional control. We previously identified a putative transcription factor, Myelin Regulatory Factor (Myrf), as being vital for CNS myelination. Myrf is required for the generation of CNS myelination during development and also for its maintenance in the adult. It has been controversial, however, whether Myrf directly regulates transcription, with reports of a transmembrane domain and lack of nuclear localization. Here we show that Myrf is a membrane-associated transcription factor that undergoes an activating proteolytic cleavage to separate its transmembrane domain-containing C-terminal region from a nuclear-targeted N-terminal region. Unexpectedly, this cleavage event occurs via a protein domain related to the autoproteolytic intramolecular chaperone domain of the bacteriophage tail spike proteins, the first time this domain has been found to play a role in eukaryotic proteins. Using ChIP-Seq we show that the N-terminal cleavage product directly binds the enhancer regions of oligodendrocyte-specific and myelin genes. This binding occurs via a defined DNA-binding consensus sequence and strongly promotes the expression of target genes. These findings identify Myrf as a novel example of a membrane-associated transcription factor and provide a direct molecular mechanism for its regulation of oligodendrocyte differentiation and CNS myelination.</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>FAVR (Filtering and Annotation of Variants that are Rare): methods to facilitate the analysis of rare germline genetic variants from massively parallel sequencing datasets</a>,  BMC Bioinformatics,  2013.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Pope BJ, Nguyen-Dumont T, Odefrey F, Hammet F, Bell R, Tao K, Tavtigian SV, Goldgar DE, Lonie A, Southey MC, Park DJ</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">BMC Bioinformatics</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">14</dd>
            
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2013</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1186/1471-2105-14-65</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">23441864</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="http://dx.doi.org/10.1186/1471-2105-14-65">http://dx.doi.org/10.1186/1471-2105-14-65</a></dd>
            
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">BACKGROUND: Characterising genetic diversity through the analysis of massively parallel sequencing (MPS) data offers enormous potential to significantly improve our understanding of the genetic basis for observed phenotypes, including predisposition to and progression of complex human disease. Great challenges remain in resolving genetic variants that are genuine from the millions of artefactual signals. RESULTS: FAVR is a suite of new methods designed to work with commonly used MPS analysis pipelines to assist in the resolution of some of the issues related to the analysis of the vast amount of resulting data, with a focus on relatively rare genetic variants. To the best of our knowledge, no equivalent method has previously been described. The most important and novel aspect of FAVR is the use of signatures in comparator sequence alignment files during variant filtering, and annotation of variants potentially shared between individuals. The FAVR methods use these signatures to facilitate filtering of (i) platform and/or mapping-specific artefacts, (ii) common genetic variants, and, where relevant, (iii) artefacts derived from imbalanced paired-end sequencing, as well as annotation of genetic variants based on evidence of co-occurrence in individuals. We applied conventional variant calling applied to whole-exome sequencing datasets, produced using both SOLiD and TruSeq chemistries, with or without downstream processing by FAVR methods. We demonstrate a 3-fold smaller rare single nucleotide variant shortlist with no detected reduction in sensitivity. This analysis included Sanger sequencing of rare variant signals not evident in dbSNP131, assessment of known variant signal preservation, and comparison of observed and expected rare variant numbers across a range of first cousin pairs. The principles described herein were applied in our recent publication identifying XRCC2 as a new breast cancer risk gene and have been made publically available as a suite of software tools. CONCLUSIONS: FAVR is a platform-agnostic suite of methods that significantly enhances the analysis of large volumes of sequencing data for the study of rare genetic variants and their influence on phenotypes.</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>A high-plex PCR approach for massively parallel sequencing</a>,  BioTechniques,  2013.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Nguyen-Dumont T, Pope BJ, Hammet F, Southey MC, Park DJ</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">BioTechniques</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">55</dd>
            
                          <dt class="col-sm-2">Issue</dt>
              <dd class="col-sm-10">2</dd>
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2013</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.2144/000114052</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">23931594</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="http://dx.doi.org/10.2144/000114052">http://dx.doi.org/10.2144/000114052</a></dd>
            
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">Current methods for targeted massively parallel sequencing (MPS) have several drawbacks, including limited design flexibility, expense, and protocol complexity, which restrict their application to settings involving modest target size and requiring low cost and high throughput. To address this, we have developed Hi-Plex, a PCR-MPS strategy intended for high-throughput screening of multiple genomic target regions that integrates simple, automated primer design software to control product size. Featuring permissive thermocycling conditions and clamp bias reduction, our protocol is simple, cost- and time-effective, uses readily available reagents, does not require expensive instrumentation, and requires minimal optimization. In a 60-plex assay targeting the breast cancer predisposition genes PALB2 and XRCC2, we applied Hi-Plex to 100 ng LCL-derived DNA, and 100 ng and 25 ng FFPE tumor-derived DNA. Altogether, at least 86.94% of the human genome-mapped reads were on target, and 100% of targeted amplicons were represented within 25-fold of the mean. Using 25 ng FFPE-derived DNA, 95.14% of mapped reads were on-target and relative representation ranged from 10.1-fold lower to 5.8-fold higher than the mean. These results were obtained using only the initial automatically-designed primers present in equal concentration. Hi-Plex represents a powerful new approach for screening panels of genomic target regions.</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Cross-platform compatibility of Hi-Plex, a streamlined approach for targeted massively parallel sequencing</a>,  Analytical biochemistry,  2013.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Nguyen-Dumont T, Pope BJ, Hammet F, Mahmoodi M, Tsimiklis H, Southey MC, Park DJ</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">Analytical biochemistry</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">442</dd>
            
                          <dt class="col-sm-2">Issue</dt>
              <dd class="col-sm-10">2</dd>
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2013</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1016/j.ab.2013.07.046</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">23933242</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="http://dx.doi.org/10.1016/j.ab.2013.07.046">http://dx.doi.org/10.1016/j.ab.2013.07.046</a></dd>
            
                          <dt class="col-sm-2">Keywords</dt>
              <dd class="col-sm-10">Disease gene screening; High-Plex PCR; Massively parallel sequencing; Molecular diagnostics; Targeted sequencing</dd>
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">Although per-base sequencing costs have decreased during recent years, library preparation for targeted massively parallel sequencing remains constrained by high reagent cost, limited design flexibility, and protocol complexity. To address these limitations, we previously developed Hi-Plex, a polymerase chain reaction (PCR) massively parallel sequencing strategy for screening panels of genomic target regions. Here, we demonstrate that Hi-Plex applied with hybrid adapters can generate a library suitable for sequencing with both the Ion Torrent and the TruSeq chemistries and that adjusting primer concentrations improves coverage uniformity. These results expand Hi-Plex capabilities as an accurate, affordable, flexible, and rapid approach for various genetic screening applications.</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Hi-Plex for high-throughput mutation screening: application to the breast cancer susceptibility gene PALB2</a>,  BMC medical genomics,  2013.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Nguyen-Dumont T, Teo ZL, Pope BJ, Hammet F, Mahmoodi M, Tsimiklis H, Sabbaghian N, Tischkowitz M, Foulkes WD, Kathleen Cuningham Foundation Consortium for research into Familial Breast cancer (kConFab), Giles GG, Hopper JL, Australian Breast Cancer Family Registry, Southey MC, Park DJ</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">BMC medical genomics</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">6</dd>
            
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2013</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1186/1755-8794-6-48</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">24206657</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="http://dx.doi.org/10.1186/1755-8794-6-48">http://dx.doi.org/10.1186/1755-8794-6-48</a></dd>
            
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">BACKGROUND: Massively parallel sequencing (MPS) has revolutionised biomedical research and offers enormous capacity for clinical application. We previously reported Hi-Plex, a streamlined highly-multiplexed PCR-MPS approach, allowing a given library to be sequenced with both the Ion Torrent and TruSeq chemistries. Comparable sequencing efficiency was achieved using material derived from lymphoblastoid cell lines and formalin-fixed paraffin-embedded tumour. METHODS: Here, we report high-throughput application of Hi-Plex by performing blinded mutation screening of the coding regions of the breast cancer susceptibility gene PALB2 on a set of 95 blood-derived DNA samples that had previously been screened using Sanger sequencing and high-resolution melting curve analysis (n = 90), or genotyped by Taqman probe-based assays (n = 5). Hi-Plex libraries were prepared simultaneously using relatively inexpensive, readily available reagents in a simple half-day protocol followed by MPS on a single MiSeq run. RESULTS: We observed that 99.93% of amplicons were represented at ≥10X coverage. All 56 previously identified variant calls were detected and no false positive calls were assigned. Four additional variant calls were made and confirmed upon re-analysis of previous data or subsequent Sanger sequencing. CONCLUSIONS: These results support Hi-Plex as a powerful approach for rapid, cost-effective and accurate high-throughput mutation screening. They further demonstrate that Hi-Plex methods are suitable for and can meet the demands of high-throughput genetic testing in research and clinical settings.</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Identification of new breast cancer predisposition genes via whole exome sequencing</a>,  Hereditary cancer in clinical practice,  2012.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Southey MC, Park DJ, Lesueur F, Odefrey F, Nguyen-Dumont T, Hammet F, Neuhausen SL, John EM, Andrulis IL, Chenevix-Trench G, Baglietto L, Le Calvez-Kelm F, Pertesi M, Lonie A, Pope B, Sinilnikova O, Tsimiklis H, Giles GG, Hopper JL, Tavtigian SV, Goldgar DE</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">Hereditary cancer in clinical practice</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">10</dd>
            
                          <dt class="col-sm-2">Issue</dt>
              <dd class="col-sm-10">2</dd>
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2012</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1186/1897-4287-10-S2-A40</dd>
            
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="http://dx.doi.org/10.1186/1897-4287-10-S2-A40">http://dx.doi.org/10.1186/1897-4287-10-S2-A40</a></dd>
            
            
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Expanded genetic analysis of a PALB2 c. 3113G> A mutation carrying multiple-case breast cancer family via exome sequencing</a>,  Hereditary cancer in clinical practice,  2012.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Teo ZL, Park DJ, Odefrey F, Hammet F, Nguyen-Dumont T, Tsimiklis H, Pope BJ, Lonie A, Winship I, Giles GG, Others</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">Hereditary cancer in clinical practice</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">10</dd>
            
                          <dt class="col-sm-2">Issue</dt>
              <dd class="col-sm-10">2</dd>
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2012</dd>
            
            
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="https://hccpjournal.biomedcentral.com/articles/10.1186/1897-4287-10-S2-A92">https://hccpjournal.biomedcentral.com/articles/10.1186/1897-4287-10-S2-A92</a></dd>
            
            
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Bpipe: a tool for running and managing bioinformatics pipelines</a>,  Bioinformatics ,  2012.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Sadedin SP, Pope B, Oshlack A</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">Bioinformatics </dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">28</dd>
            
                          <dt class="col-sm-2">Issue</dt>
              <dd class="col-sm-10">11</dd>
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2012</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1093/bioinformatics/bts167</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">22500002</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="http://dx.doi.org/10.1093/bioinformatics/bts167">http://dx.doi.org/10.1093/bioinformatics/bts167</a></dd>
            
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">SUMMARY: Bpipe is a simple, dedicated programming language for defining and executing bioinformatics pipelines. It specializes in enabling users to turn existing pipelines based on shell scripts or command line tools into highly flexible, adaptable and maintainable workflows with a minimum of effort. Bpipe ensures that pipelines execute in a controlled and repeatable fashion and keeps audit trails and logs to ensure that experimental results are reproducible. Requiring only Java as a dependency, Bpipe is fully self-contained and cross-platform, making it very easy to adopt and deploy into existing environments. AVAILABILITY AND IMPLEMENTATION: Bpipe is freely available from http://bpipe.org under a BSD License.</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Rare mutations in XRCC2 increase the risk of breast cancer</a>,  American journal of human genetics,  2012.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Park DJ, Lesueur F, Nguyen-Dumont T, Pertesi M, Odefrey F, Hammet F, Neuhausen SL, John EM, Andrulis IL, Terry MB, Daly M, Buys S, Le Calvez-Kelm F, Lonie A, Pope BJ, Tsimiklis H, Voegele C, Hilbers FM, Hoogerbrugge N, Barroso A, Osorio A, Breast Cancer Family Registry, Kathleen Cuningham Foundation Consortium for Research into Familial Breast Cancer, Giles GG, Devilee P, Benitez J, Hopper JL, Tavtigian SV, Goldgar DE, Southey MC</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">American journal of human genetics</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">90</dd>
            
                          <dt class="col-sm-2">Issue</dt>
              <dd class="col-sm-10">4</dd>
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2012</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1016/j.ajhg.2012.02.027</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">22464251</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="http://dx.doi.org/10.1016/j.ajhg.2012.02.027">http://dx.doi.org/10.1016/j.ajhg.2012.02.027</a></dd>
            
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">An exome-sequencing study of families with multiple breast-cancer-affected individuals identified two families with XRCC2 mutations, one with a protein-truncating mutation and one with a probably deleterious missense mutation. We performed a population-based case-control mutation-screening study that identified six probably pathogenic coding variants in 1,308 cases with early-onset breast cancer and no variants in 1,120 controls (the severity grading was p < 0.02). We also performed additional mutation screening in 689 multiple-case families. We identified ten breast-cancer-affected families with protein-truncating or probably deleterious rare missense variants in XRCC2. Our identification of XRCC2 as a breast cancer susceptibility gene thus increases the proportion of breast cancers that are associated with homologous recombination-DNA-repair dysfunction and Fanconi anemia and could therefore benefit from specific targeted treatments such as PARP (poly ADP ribose polymerase) inhibitors. This study demonstrates the power of massively parallel sequencing for discovering susceptibility genes for common, complex diseases.</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>A Computational Model for Retinal Ganglion Cell Axon Pathfinding</a>,  Investigative Ophthalmology & Visual Science,  2012.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Andrew Turpin, Bernard Pope, Jonathan Denniss</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">Investigative Ophthalmology & Visual Science</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">53</dd>
            
                          <dt class="col-sm-2">Issue</dt>
              <dd class="col-sm-10">14</dd>
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2012</dd>
            
            
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="https://iovs.arvojournals.org/article.aspx?articleid=2358170">https://iovs.arvojournals.org/article.aspx?articleid=2358170</a></dd>
            
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">Purpose: To investigate whether an algorithm for simulating human retinal ganglion cell (RGC) axon pathfinding during development that does not rely on a chemical gradient produces plausible axon paths.

Methods: A computational model of human RGC axon pathfinding was developed as follows. The algorithm assumes a lattice covering the retina that is populated with RGCs according to known average cell density for a healthy human retina (Curcio CA, Allen KA, J Comp Neurol, 1990: 300; 5-25). Each cell sends an axon growth cone out in "search mode" that locates a nearby axon. When an axon is located, the growth cone enters "bundle mode", and follows on top of the axon until a critical retinal thickness constraint is met. If there is no room to stack on top of the existing axon, the growth cone reverts to search mode, and finds either an unfilled blank location, or another axon to follow.Two restrictions on this basic algorithm prevent random axon pathways forming. Firstly, cells nearest the optic nerve head (ONH) develop first, so following an existing axon will take a new axon towards the ONH. Secondly, when in search mode, searching only happens within ±90 degrees of the current trajectory.The thickness constraint is mediated as a linear function beginning at zero at the fovea, and reaching a maximum of 60 axons at a distance of 3mm from the fovea. There is no thickness constraint outside of a radius of 3mm from the fovea.Output from the model was compared with a previous mathematical model of retinal nerve fibre bundles based on hand-drawn traces of human RGC axon pathways (Jansonius NM et al, Vis Res, 2009: 49; 2157-63).

Results: Allowing for a plus or minus 5 degree error in the entry point of the ONH, 60% of our generated paths in the superior retina fell within the 95% confidence limits given by Jansonius, and 17% inferiorly. Allowing for plus or minus 10 degrees: 83% superior; and 33% inferior. The presence of a chemical gradient was not necessary for such patterns to be produced. Generally our inferior pathways were straighter than those reported by Jansonius. A non-symmetrical thickness constraint may be required to increase the curvature of inferior axon pathways.

Conclusions: Local cues are sufficient to guide RGC axons across the retina to the ONH in the absence of a chemical gradient. This model does not exclude the existence of a gradient, or partial local gradients, but simply demonstrates that a gradient is not essential to support current empirical data.
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Precision Medicine: Dawn of Supercomputing in ‘omics Research</a>,  2011.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Reumann M, Holt KE, Inouye M, Stinear T, Goudey B, Abraham G, WANG Q, Shi F, Kowalczyk A, Pearce A, Isaac A, Pope BJ, Butzkueven H, Wagner J, Moore S, Downton M, Church PC, Turner SJ, Field J, Southey M, Bowtell D, Schmidt D, Makalic E, Zobel J, Hopper J, Petrovski S, O'Brien T</dd>
            
            
            
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2011</dd>
            
            
            
            
            
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Performance of hybrid programming models for multiscale cardiac simulations: preparing for petascale computation</a>,  IEEE transactions on bio-medical engineering,  2011.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Pope BJ, Fitch BG, Pitman MC, Rice JJ, Reumann M</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">IEEE transactions on bio-medical engineering</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">58</dd>
            
                          <dt class="col-sm-2">Issue</dt>
              <dd class="col-sm-10">10</dd>
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2011</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1109/TBME.2011.2161580</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">21768044</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="http://dx.doi.org/10.1109/TBME.2011.2161580">http://dx.doi.org/10.1109/TBME.2011.2161580</a></dd>
            
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">Future multiscale and multiphysics models that support research into human disease, translational medical science, and treatment can utilize the power of high-performance computing (HPC) systems. We anticipate that computationally efficient multiscale models will require the use of sophisticated hybrid programming models, mixing distributed message-passing processes [e.g., the message-passing interface (MPI)] with multithreading (e.g., OpenMP, Pthreads). The objective of this study is to compare the performance of such hybrid programming models when applied to the simulation of a realistic physiological multiscale model of the heart. Our results show that the hybrid models perform favorably when compared to an implementation using only the MPI and, furthermore, that OpenMP in combination with the MPI provides a satisfactory compromise between performance and code complexity. Having the ability to use threads within MPI processes enables the sophisticated use of all processor cores for both computation and communication phases. Considering that HPC systems in 2012 will have two orders of magnitude more cores than what was used in this study, we believe that faster than real-time multiscale cardiac simulations can be achieved on these systems.</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Petascale computation performance of lightweight multiscale cardiac models using hybrid programming models</a>,  Annual International Conference of the IEEE Engineering in Medicine and Biology Society. IEEE Engineering in Medicine and Biology Society. Conference,  2011.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Pope BJ, Fitch BG, Pitman MC, Rice JJ, Reumann M</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">Annual International Conference of the IEEE Engineering in Medicine and Biology Society. IEEE Engineering in Medicine and Biology Society. Conference</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">2011</dd>
            
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2011</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1109/IEMBS.2011.6090058</dd>
            
                          <dt class="col-sm-2">Pubmed ID</dt>
              <dd class="col-sm-10">22254341</dd>
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="http://dx.doi.org/10.1109/IEMBS.2011.6090058">http://dx.doi.org/10.1109/IEMBS.2011.6090058</a></dd>
            
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">Future multiscale and multiphysics models must use the power of high performance computing (HPC) systems to enable research into human disease, translational medical science, and treatment. Previously we showed that computationally efficient multiscale models will require the use of sophisticated hybrid programming models, mixing distributed message passing processes (e.g. the message passing interface (MPI)) with multithreading (e.g. OpenMP, POSIX pthreads). The objective of this work is to compare the performance of such hybrid programming models when applied to the simulation of a lightweight multiscale cardiac model. Our results show that the hybrid models do not perform favourably when compared to an implementation using only MPI which is in contrast to our results using complex physiological models. Thus, with regards to lightweight multiscale cardiac models, the user may not need to increase programming complexity by using a hybrid programming approach. However, considering that model complexity will increase as well as the HPC system size in both node count and number of cores per node, it is still foreseeable that we will achieve faster than real time multiscale cardiac simulations on these systems using hybrid programming models.</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>A Lightweight Interactive Debugger for Haskell</a>,  Haskell '07 Proceedings of the ACM SIGPLAN workshop on Haskell,  2007.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Simon Marlow, Jose Iborra, Bernard Pope, Andy Gill</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">Haskell '07 Proceedings of the ACM SIGPLAN workshop on Haskell</dd>
            
            
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2007</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1145/1291201.1291204</dd>
            
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="http://dx.doi.org/10.1145/1291201.1291204">http://dx.doi.org/10.1145/1291201.1291204</a></dd>
            
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">This paper describes the design and construction of a Haskell source-level debugger built into the GHCi interactive environment. We have taken a pragmatic approach: the debugger is based on the traditional stop-examine-continue model of online debugging, which is simple and intuitive, but has traditionally been shunned in the context of Haskell because it exposes the lazy evaluation order. We argue that this drawback is not as severe as it may seem, and in some cases is an advantage. The design focuses on availability: our debugger is intended to work on all programs that can be compiled with GHC, and without requiring the programmer to jump through additional hoops to debug their program. The debugger has a novel approach for reconstructing the type of runtime values in a polymorphic context. Our implementation is light on complexity, and was integrated into GHC without significant upheaval.
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>A Declarative Debugger for Haskell</a>,  2006.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Bernard Pope</dd>
            
            
            
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2006</dd>
            
            
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="assets/files/BerniePope.PhD.Thesis.pdf">assets/files/BerniePope.PhD.Thesis.pdf</a></dd>
            
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10"><p>This thesis is about the design and implementation of a debugging tool which helps Haskell programmers understand why their programs do not work as intended. The traditional debugging technique of examining the program execution step-by-step, popular with imperative languages, is less suitable for Haskell because its unorthodox evaluation strategy is difficult to relate to the structure of the original program source code. We build a debugger which focuses on the high-level logical meaning of a program rather than its evaluation order. This style of debugging is called declarative debugging, and it originated in logic programming languages. At the heart of the debugger is a tree which records information about the evaluation of the program in a manner which is easy to relate to the structure of the program. Links between nodes in the tree reflect logical relationships between entities in the source code. An error diagnosis algorithm is applied to the tree in a top-down fashion, searching for causes of bugs. The search is guided by an oracle, who knows how each part of the program should behave. The oracle is normally a human — typically the person who wrote the program — however, much of its behaviour can be encoded in software.</p><p>An interesting aspect of this work is that the debugger is implemented by means of a program transformation. That is, the program which is to be debugged is trans- formed into a new one, which when evaluated, behaves like the original program but also produces the evaluation tree as a side-effect. The transformed program is augmented with code to perform the error diagnosis on the tree. Running the trans- formed program constitutes the evaluation of the original program plus a debugging session. The use of program transformation allows the debugger to take advantage of existing compiler technology — a whole new compiler and runtime environment does not need to be written — which saves much work and enhances portability.</p><p>The technology described in this thesis is well-tested by an implementation in software. The result is a useful tool, called buddha, which is publicly available and supports all of the Haskell 98 standard.</p>
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Declarative Debugging with Buddha</a>,  Advanced Functional Programming. AFP 2004. Lecture Notes in Computer Science,  2004.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Bernard Pope</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">Advanced Functional Programming. AFP 2004. Lecture Notes in Computer Science</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">3622</dd>
            
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2004</dd>
            
            
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="https://link.springer.com/chapter/10.1007/11546382_7">https://link.springer.com/chapter/10.1007/11546382_7</a></dd>
            
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10"><p>Haskell is a very safe language, particularly because of its type system. However there will always be programs that do the wrong thing. Programmer fallibility, partial or incorrect specifications and typographic errors are but a few of the reasons that make bugs a fact of life. This paper is about the use and implementation of a debugger, called Buddha, which helps Haskell programmers understand why their programs misbehave. Traditional debugging tools that examine the program execution step-by-step are not suitable for Haskell because of its unorthodox evaluation strategy. Instead, a different approach is taken which abstracts away the evaluation order of the program and focuses on its high-level logical meaning.</p><p>This style of debugging is called Declarative Debugging, and it has its roots in the Logic Programming community. At the heart of the debugger is a tree which records information about the evaluation of the program in a manner which is easy to relate to the structure of the source code. It resembles a call graph annotated with the arguments and results of function applications, shown in their most evaluated form. Logical relationships between entities in the source are reflected in the links between nodes in the tree. An error diagnosis algorithm is applied to the tree in a top-down fashion in the search for causes of bugs.</p>
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>Practical Aspects of Declarative Debugging in Haskell 98</a>,  Proceedings of the 5th ACM SIGPLAN international conference on Principles and practice of declaritive programming,  2003.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Bernard Pope, Lee Naish</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">Proceedings of the 5th ACM SIGPLAN international conference on Principles and practice of declaritive programming</dd>
            
            
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2003</dd>
            
                          <dt class="col-sm-2">DOI</dt>
              <dd class="col-sm-10">10.1145/888251.888273</dd>
            
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="http://dx.doi.org/10.1145/888251.888273">http://dx.doi.org/10.1145/888251.888273</a></dd>
            
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">Non-strict purely functional languages pose many challenges to the designers of debugging tools. Declarative debugging has long been considered a suitable candidate for the task due to its abstraction over the evaluation order of the program, although the provision of practical implementations has been lagging. In this paper we discuss the solutions used in our declarative debugger for Haskell to tackle the problems of printing values, memory usage and I/O. The debugger is based on program transformation, although much leverage is gained by interfacing with the runtime environment of the language implementation through a foreign function interface.
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>


  <ul style="list-style-type:none">
    <li>
      <details>
        <summary>A program transformation for debugging Haskell 98</a>,  ACSC '03 Proceedings of the 26th Australasian computer science conference,  2003.</summary>
        <div class="card card-body my-3">
          <dl class="row">
                          <dt class="col-sm-2">Authors</dt>
              <dd class="col-sm-10">Bernard Pope, Lee Naish</dd>
            
                          <dt class="col-sm-2">Journal</dt>
              <dd class="col-sm-10">ACSC '03 Proceedings of the 26th Australasian computer science conference</dd>
            
                          <dt class="col-sm-2">Volume</dt>
              <dd class="col-sm-10">16</dd>
            
            
                          <dt class="col-sm-2">Year</dt>
              <dd class="col-sm-10">2003</dd>
            
            
            
                          <dt class="col-sm-2">URL</dt>
              <dd class="col-sm-10"><a href="https://dl.acm.org/citation.cfm?id=783132">https://dl.acm.org/citation.cfm?id=783132</a></dd>
            
            
                          <dt class="col-sm-2">Abstract</dt>
              <dd class="col-sm-10">We present a source-to-source transformation of Haskell 98 pro-grams for the purpose of debugging. The source code of a program is transformed into a new program which, when executed,computes the value of the original program and a high-level semantics for that computation. The semantics is given by a tree whose nodes represent function applications that were evaluated during execution. This tree is useful in situations where a high-level view of a computation is needed, such as declarative debugging. The main contribution of the paper is the treatment of higher-order functions, which have previously proven difficult to support in declarative debugging schemes.
</dd>
                      </dl>
        </div>
      </details>
    </li>
  </ul>

      </div> <!-- container -->

      <!-- jQuery first, then Popper.js, then Bootstrap JS -->
      <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.3/umd/popper.min.js" integrity="sha384-vFJXuSJphROIrBnz7yo7oB41mKfc8JzQZiCq4NCceLEaO4IHwicKwpJf9c9IpFgh" crossorigin="anonymous"></script>
      <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/js/bootstrap.min.js" integrity="sha384-alpBpkh1PFOepccYVYDB4do5UnbKysX5WZXm3XxPqe5iKTfUKjNkCk9SaVuEZflJ" crossorigin="anonymous"></script>
      <script src="assets/js/site.js"></script>
      <script src="assets/js/details-element-polyfill.js"></script>
  </body>
</html>
