<!doctype html>
<html lang="en">
  <head>
     
    <title>Bernie Pope: Software</title>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css"
          integrity="sha384-PsH8R72JQ3SOdhVi3uxftmaW6Vc51MKb0q5P2rRUpPvrszuE4W1povHYgTpBfshb" crossorigin="anonymous">
    <link rel="stylesheet" href="assets/css/website.css">
      </head>

  <body>
      <div class="container"> 
      <!-- <div class="container-fluid"">  -->
      
      <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
            <a class="navbar-brand" href="index.html">Bernie Pope</a>
            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
              <span class="navbar-toggler-icon"></span>
            </button>
            <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
              <div class="navbar-nav">

                                  <a class="nav-item nav-link"href="index.html">Home</a>
                                  <a class="nav-item nav-link"href="biography.html">Biography</a>
                                  <a class="nav-item nav-link"href="publications.html">Publications</a>
                                  <a class="nav-item nav-link"href="presentations.html">Presentations</a>
                                  <a class="nav-item nav-link active"href="software.html">Software</a>
                                  <a class="nav-item nav-link"href="funding.html">Funding</a>
                                  <a class="nav-item nav-link"href="teaching.html">Teaching</a>
                                  <a class="nav-item nav-link"href="students.html">Students</a>
                                  <a class="nav-item nav-link"href="notes.html">Notes</a>
                                  <a class="nav-item nav-link"href="fun.html">Fun</a>
                                  <a class="nav-item nav-link"href="contact.html">Contact</a>
                
              </div>
            </div>
          </nav>
      

<h1 class="display-4 my-5">Research Software</h1>

<p>GitHub: <a href="https://github.com/bjpop">github.com/bjpop</a>
<a href="https://github.com/bjpop?tab=followers"><img src="https://img.shields.io/github/followers/bjpop.svg?style=social&logo=github&label=Followers"></a>
</p>


  <div class="card my-5">
    <h4 class="card-header">Varlap</h4>
      <div class="card-body">
         <dl class="row">
	     <dt class="col-sm-2">Summary</dt>
             <dd class="col-sm-10">Varlap is a quality control tool for genetic variants arising from high throughput DNA sequencing, where the variants have been called by aligning DNA sequencing reads to a reference genome.</dd>

	     <dt class="col-sm-2">Repository</dt>
             <dd class="col-sm-10"><a href="https://github.com/bjpop/varlap">https://github.com/bjpop/varlap</a></dd>

	     <dt class="col-sm-2">Description</dt>
             <dd class="col-sm-10"><p>Varlap is primarily a quality control tool for genetic variants arising from high throughput DNA sequencing, where the variants have been called by aligning DNA sequencing reads to a reference genome. It takes as input a set of DNA variants and one or more BAM files. Varlap considers the genomic locus of each variant in each of the supplied BAM files and records information about the corresponding alignment context at that locus. For example, one of the metrics it calculates is the average edit distance of reads overlapping the variant locus. This can be a useful metric because regions with significantly higher average edit distance are more likely to contain erroneous variant calls. Varlap outputs a CSV file containing one row per input variant, with columns recording the various computed metrics about that variant. Subsequent analysis of this output (such as outlier detection) can be used to identify potentially problematic variants and samples.</p><p>Common use cases are to consider somatic variants in the context of tumour and normal alignments, or germline variants against normal alignments. However, varlap is quite flexible and allows the use of any number of BAM files as input.</p>
</dd>

	              </dl>
      </div>
  </div>

  <div class="card my-5">
    <h4 class="card-header">hatch</h4>
      <div class="card-body">
         <dl class="row">
	     <dt class="col-sm-2">Summary</dt>
             <dd class="col-sm-10">Hatch is a command line program for plotting and analysis of tabular data (CSV, TSV files).</dd>

	     <dt class="col-sm-2">Repository</dt>
             <dd class="col-sm-10"><a href="https://github.com/bjpop/hatch">https://github.com/bjpop/hatch</a></dd>

	     <dt class="col-sm-2">Description</dt>
             <dd class="col-sm-10"><p>Hatch provides a convenient yet featureful way to plot tabular data from the command line. It supports numerous plot types including: histograms, distribution plots, scatter plots, line plots, heatmaps, count plots (bar plots), principal component analysis (PCA) plots.</p>
</dd>

	              </dl>
      </div>
  </div>

  <div class="card my-5">
    <h4 class="card-header">Bionitio</h4>
      <div class="card-body">
         <dl class="row">
	     <dt class="col-sm-2">Summary</dt>
             <dd class="col-sm-10">Bionitio provides a template for command line bioinformatics tools in various programming languages, and automates the creation of new software repositories following best practices.</dd>

	     <dt class="col-sm-2">Repository</dt>
             <dd class="col-sm-10"><a href="https://github.com/bionitio-team/bionitio">https://github.com/bionitio-team/bionitio</a></dd>

	     <dt class="col-sm-2">Description</dt>
             <dd class="col-sm-10"><p>The purpose of Bionitio is to provide an easy-to-understand working example that is built on best-practice software engineering principles. It can be used as a basis for learning and as a solid foundation for starting new projects. We provide a script called bionitio-boot.sh for starting new projects from bionitio, which saves time and ensures good programming practices are adopted from the beginning</p>
</dd>

	     	     <dt class="col-sm-2">Publication</dt>
             <dd class="col-sm-10"><a href="https://doi.org/10.1093/gigascience/giz109">Bionitio: demonstrating and facilitating best practices for bioinformatics command-line software</a>, GigaScience.</dd>
	              </dl>
      </div>
  </div>

  <div class="card my-5">
    <h4 class="card-header">UNDR ROVER</h4>
      <div class="card-body">
         <dl class="row">
	     <dt class="col-sm-2">Summary</dt>
             <dd class="col-sm-10">A fast and accurate variant caller for targeted DNA sequencing.</dd>

	     <dt class="col-sm-2">Repository</dt>
             <dd class="col-sm-10"><a href="https://github.com/bjpop/undr_rover">https://github.com/bjpop/undr_rover</a></dd>

	     <dt class="col-sm-2">Description</dt>
             <dd class="col-sm-10"><p>UNDR ROVER is an improved version of our ROVER variant calling tool for targeted DNA sequencing. It enables users to quickly and accurately identify genetic variants from PCR-targeted, overlapping paired-end MPS datasets. It calls the same variants as the ROVER tool but at a significantly reduced runtime. It achieves its higher performance by avoiding read alignment before variant calling, and can be applied directly to input FASTQ files.</p>
</dd>

	     	     <dt class="col-sm-2">Publication</dt>
             <dd class="col-sm-10"><a href="http://www.biomedcentral.com/1471-2105/17/165">UNDR ROVER - a fast and accurate variant caller for targeted DNA sequencing</a>, BMC Bioinformatics.</dd>
	              </dl>
      </div>
  </div>

  <div class="card my-5">
    <h4 class="card-header">HiTIME</h4>
      <div class="card-body">
         <dl class="row">
	     <dt class="col-sm-2">Summary</dt>
             <dd class="col-sm-10">High-resolution Twin-Ion Metabolic Extraction.</dd>

	     <dt class="col-sm-2">Repository</dt>
             <dd class="col-sm-10"><a href="https://github.com/bjpop/HiTIME">https://github.com/bjpop/HiTIME</a></dd>

	     <dt class="col-sm-2">Description</dt>
             <dd class="col-sm-10"><p>HiTIME is a software tool for detecting twin ion signals in high resolution liquid chromatography mass spectrometry (LCMS) data.</p><p>This is a collaboration with Andrew Isaac, Michael Leeming, Richard O'Hair and William Alexander Donald.</p>
</dd>

	     	     <dt class="col-sm-2">Publication</dt>
             <dd class="col-sm-10"><ul><li><a href="http://pubs.acs.org/doi/abs/10.1021/ac504767d">High-Resolution Twin-Ion Metabolite Extraction (HiTIME) Mass Spectrometry: Nontargeted Detection of Unknown Drug Metabolites by Isotope Labeling, Liquid Chromatography Mass Spectrometry, and Automated High-Performance Computing</a>, Analytical Chemistry.</li><li><a href="https://doi.org/10.1016/j.softx.2020.100559">HiTIME: An efficient model-selection approach for the detection of unknown drug metabolites in LC-MS data</a>, SoftwareX.</li></ul>
</dd>
	              </dl>
      </div>
  </div>

  <div class="card my-5">
    <h4 class="card-header">SRST2</h4>
      <div class="card-body">
         <dl class="row">
	     <dt class="col-sm-2">Summary</dt>
             <dd class="col-sm-10">Short Read Sequence Typing for Bacterial Pathogens.</dd>

	     <dt class="col-sm-2">Repository</dt>
             <dd class="col-sm-10"><a href="http://katholt.github.io/srst2/">http://katholt.github.io/srst2/</a></dd>

	     <dt class="col-sm-2">Description</dt>
             <dd class="col-sm-10"><p>This takes Illumina sequence data, a MLST (Multi-Locus Sequence Type) database and/or a database of gene sequences (e.g. resistance genes, virulence genes, etc) and report the presence of STs and/or reference genes.</p><p>This is a collaboration with Kat Holt, Mike Inouye and others.</p>
</dd>

	     	     <dt class="col-sm-2">Publication</dt>
             <dd class="col-sm-10"><a href="http://genomemedicine.com/content/6/11/90">SRST2: Rapid genomic surveillance for public health and hospital microbiology labs</a>, Genome Medicine.</dd>
	              </dl>
      </div>
  </div>

  <div class="card my-5">
    <h4 class="card-header">Methpat</h4>
      <div class="card-body">
         <dl class="row">
	     <dt class="col-sm-2">Summary</dt>
             <dd class="col-sm-10">A program for summarising and visualising CpG methylation patterns.</dd>

	     <dt class="col-sm-2">Repository</dt>
             <dd class="col-sm-10"><a href="https://github.com/bjpop/methpat">https://github.com/bjpop/methpat</a></dd>

	     <dt class="col-sm-2">Description</dt>
             <dd class="col-sm-10"><p>Methpat summarises the resultant DNA methylation pattern data from the output of Bismark methylation extractor. Information of the DNA methylation positions for each amplicon, DNA methylation patterns observed within each amplicon and their abundance counts are summarised into a tab delimited text file amenable for further downstream statistical analysis and visualization.</p>
</dd>

	     	     <dt class="col-sm-2">Publication</dt>
             <dd class="col-sm-10"><a href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-0950-8">MethPat: a tool for the analysis and visualisation of complex methylation patterns obtained by massively parallel sequencing</a>, BMC Bioinformatics.</dd>
	              </dl>
      </div>
  </div>

  <div class="card my-5">
    <h4 class="card-header">Annokey</h4>
      <div class="card-body">
         <dl class="row">
	     <dt class="col-sm-2">Summary</dt>
             <dd class="col-sm-10">Gene-based search for key-terms in the NCBI gene database and associated PubMed abstracts.</dd>

	     <dt class="col-sm-2">Repository</dt>
             <dd class="col-sm-10"><a href="http://bjpop.github.io/annokey/">http://bjpop.github.io/annokey/</a></dd>

	     <dt class="col-sm-2">Description</dt>
             <dd class="col-sm-10"><p>Annokey is a command line tool for annotating gene lists with the results of a key-term search of the NCBI Gene database and linked PubMed article abstracts. Its purpose is to help users prioritise genes by relevance to a domain of interest, such as "breast cancer" or "DNA repair" etcetera. The user steers the search by specifying a ranked list of keywords and terms that are likely to be highly correlated with their domain of interest.</p>
</dd>

	     	     <dt class="col-sm-2">Publication</dt>
             <dd class="col-sm-10"><a href="http://scfbm.biomedcentral.com/articles/10.1186/1751-0473-9-15">Annokey: an annotation tool based on key term search of the NCBI Entrez Gene database</a>, Source Code for Biology and Medicine.</dd>
	              </dl>
      </div>
  </div>

  <div class="card my-5">
    <h4 class="card-header">ROVER</h4>
      <div class="card-body">
         <dl class="row">
	     <dt class="col-sm-2">Summary</dt>
             <dd class="col-sm-10">Read-pair overlap considerate variant-calling software for PCR-based massively parallel sequencing datasets.</dd>

	     <dt class="col-sm-2">Repository</dt>
             <dd class="col-sm-10"><a href="https://github.com/bjpop/rover">https://github.com/bjpop/rover</a></dd>

	     <dt class="col-sm-2">Description</dt>
             <dd class="col-sm-10"><p>ROVER-PCR Variant Caller enables users to quickly and accurately identify genetic variants from PCR-targeted, overlapping paired-end MPS datasets. The open-source availability of the software and threshold tailorability enables broad access for a range of PCR-MPS users.</p>
</dd>

	     	     <dt class="col-sm-2">Publication</dt>
             <dd class="col-sm-10"><a href="http://www.scfbm.org/content/9/1/3">ROVER variant caller: read-pair overlap considerate variant-calling software applied to PCR-based massively parallel sequencing datasets</a>, Source Code for Biology and Medicine.</dd>
	              </dl>
      </div>
  </div>

  <div class="card my-5">
    <h4 class="card-header">Blip</h4>
      <div class="card-body">
         <dl class="row">
	     <dt class="col-sm-2">Summary</dt>
             <dd class="col-sm-10">A bytecode compiler for Python 3.</dd>

	     <dt class="col-sm-2">Repository</dt>
             <dd class="col-sm-10"><a href="https://github.com/bjpop/blip">https://github.com/bjpop/blip</a></dd>

	     <dt class="col-sm-2">Description</dt>
             <dd class="col-sm-10"><p>Blip compiles Python 3 source files to bytecode. The output bytecode is compatible with the CPython interpreter.</p>
</dd>

	              </dl>
      </div>
  </div>

  <div class="card my-5">
    <h4 class="card-header">FAVR</h4>
      <div class="card-body">
         <dl class="row">
	     <dt class="col-sm-2">Summary</dt>
             <dd class="col-sm-10">Filtering and Annotation of Variants that are Rare.</dd>

	     <dt class="col-sm-2">Repository</dt>
             <dd class="col-sm-10"><a href="https://github.com/bjpop/favr">https://github.com/bjpop/favr</a></dd>

	     <dt class="col-sm-2">Description</dt>
             <dd class="col-sm-10"><p>Characterizing genetic diversity through the analysis of massively parallel sequence (MPS) data offers enormous potential in terms of our understanding of predisposition to complex human disease. Great challenges remain, however, regarding our ability to resolve those genetic variants that are genuinely associated with disease from the millions of "bystanders" and artefactual signals. FAVR is designed to assist in the resolution of some of these issues in the context of rare germline variants by facilitating "platform-steered" artefact filtering.</a>
</dd>

	     	     <dt class="col-sm-2">Publication</dt>
             <dd class="col-sm-10"><a href="http://www.biomedcentral.com/1471-2105/14/65/abstract">FAVR (Filtering and Annotation of Variants that are Rare): methods to facilitate the analysis of rare germline genetic variants from massively parallel sequencing datasets</a>, BMC Bioinformatics</dd>
	              </dl>
      </div>
  </div>

  <div class="card my-5">
    <h4 class="card-header">berp</h4>
      <div class="card-body">
         <dl class="row">
	     <dt class="col-sm-2">Summary</dt>
             <dd class="col-sm-10">A compiler and interpreter for Python 3.</dd>

	     <dt class="col-sm-2">Repository</dt>
             <dd class="col-sm-10"><a href="http://github.com/bjpop/berp">http://github.com/bjpop/berp</a></dd>

	     <dt class="col-sm-2">Description</dt>
             <dd class="col-sm-10"><p>Berp is an implementation of Python 3. At its heart is a translator, which takes Python code as input and generates Haskell code as output. The Haskell code is fed into a Haskell compiler (GHC) for compilation to machine code or interpretation as byte code.</p><p>Berp provides both a compiler and an interactive interpreter. For the most part it can be used in the same way as CPython (the main Python implementation).</p>
</dd>

	              </dl>
      </div>
  </div>

  <div class="card my-5">
    <h4 class="card-header">haskell-mpi</h4>
      <div class="card-body">
         <dl class="row">
	     <dt class="col-sm-2">Summary</dt>
             <dd class="col-sm-10">A Haskell interface to the MPI distributed parallel library.</dd>

	     <dt class="col-sm-2">Repository</dt>
             <dd class="col-sm-10"><a href="http://hackage.haskell.org/package/haskell-mpi">http://hackage.haskell.org/package/haskell-mpi</a></dd>

	     <dt class="col-sm-2">Description</dt>
             <dd class="col-sm-10"><p>MPI is defined by the Message-Passing Interface Standard, as specified by the Message Passing Interface Forum. The latest release of the standard is known as MPI-2. These Haskell bindings are designed to work with any standards compliant implementation of MPI-2.</p>
</dd>

	     	     <dt class="col-sm-2">Publication</dt>
             <dd class="col-sm-10"><a href="https://themonadreader.files.wordpress.com/2011/10/issue19.pdf">High Performance Haskell with MPI</a>, The Monad Reader</dd>
	              </dl>
      </div>
  </div>

  <div class="card my-5">
    <h4 class="card-header">language-python</h4>
      <div class="card-body">
         <dl class="row">
	     <dt class="col-sm-2">Summary</dt>
             <dd class="col-sm-10">A lexer, parser and pretty printer for Python programs, written in Haskell.</dd>

	     <dt class="col-sm-2">Repository</dt>
             <dd class="col-sm-10"><a href="http://hackage.haskell.org/package/language-python">http://hackage.haskell.org/package/language-python</a></dd>

	     <dt class="col-sm-2">Description</dt>
             <dd class="col-sm-10"><p>This package provides a parser (and lexer) for Python written in Haskell. It supports version 2 and 3 of Python. The parser is implemented using the happy parser generator, and the alex lexer generator. The package also provides a pretty printer, which makes it also suitable for generating Python code.</p>
</dd>

	              </dl>
      </div>
  </div>

  <div class="card my-5">
    <h4 class="card-header">ministg</h4>
      <div class="card-body">
         <dl class="row">
	     <dt class="col-sm-2">Summary</dt>
             <dd class="col-sm-10">An interpreter for the small-step operational semantics of the STG machine.</dd>

	     <dt class="col-sm-2">Repository</dt>
             <dd class="col-sm-10"><a href="http://www.haskell.org/haskellwiki/Ministg">http://www.haskell.org/haskellwiki/Ministg</a></dd>

	     <dt class="col-sm-2">Description</dt>
             <dd class="col-sm-10"><p>Ministg is an interpreter for a high-level, small-step, operational semantics for the STG machine. The STG machine is the abstract machine at the core of GHC. The operational semantics used in Ministg is taken from the paper "Making a fast curry: push/enter versus eval/apply for higher-order languages" by Simon Marlow and Simon Peyton Jones. Ministg implements both sets of evaluation rules from the paper.
</dd>

	     	     <dt class="col-sm-2">Publication</dt>
             <dd class="col-sm-10"><a href="http://www.haskell.org/haskellwiki/Ministg">Ministg</a>, Haskell Wiki</dd>
	              </dl>
      </div>
  </div>

      </div> <!-- container -->

      <!-- jQuery first, then Popper.js, then Bootstrap JS -->
      <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.3/umd/popper.min.js" integrity="sha384-vFJXuSJphROIrBnz7yo7oB41mKfc8JzQZiCq4NCceLEaO4IHwicKwpJf9c9IpFgh" crossorigin="anonymous"></script>
      <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/js/bootstrap.min.js" integrity="sha384-alpBpkh1PFOepccYVYDB4do5UnbKysX5WZXm3XxPqe5iKTfUKjNkCk9SaVuEZflJ" crossorigin="anonymous"></script>
      <script src="assets/js/site.js"></script>
      <script src="assets/js/details-element-polyfill.js"></script>
  </body>
</html>
